<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	18-05-03		URL: 	http://www.HGVS.org/mutnomen/checklist.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>HGVS recommendations: checklist </title>    <style type="text/css"><!--  h1 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}p { font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #000000;margin-left : 5%; }.norm { font-weight : normal; } --></style> </head>  <body style="color: black; background-color: white;" link="#0000FF" vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Checklist for the description of sequence              variants</h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified April 28, 2014</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i><font face="Times New Roman">Since references to          WWW-sites are not yet acknowledged as citations, please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den
            Dunnen JT and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when          referring to these pages.</font></i></p>    <hr>    <h2>Purpose</h2>    <p>Going through publications one can easily see where people tend to offend      the <i><b>"<a href="recs.html">Current recommendations for the            description of sequence variants</a>"</b></i>. The checklist below      covers the most problematic issues and should assist those preparing a      publication to describe sequence variants following the current      recommendations. </p>    <hr>    <h2>Checklist</h2>    <ol>      <li><strong>Reference Sequence</strong> - do you clearly mention the        reference sequence used for numbering (nucleotides/amino acids)?<br>        A publication should mention, preferably in the Materials &amp; Methods        section and/or Table legend, which sequence file was used as reference        sequence for numbering of the residues (DNA, RNA and protein) and        describing the variants; see <a href="recs.html#version"><i>Recommendations</i></a>,        <a href="refseq.html"><i>Discussion</i></a> and <a href="refseq.html#mtDNA"><i>mtDNA
            variants</i></a>.        <ul type="disc">          <li>do you mention a <em>GenBank (not Gen<b><u>e</u></b>Bank)              RefSeq-file accession number <b>with version number</b></em>?; do            not forget the underscore in the accession number (correct is NM<span              style="font-weight: bold; font-style: italic;">_</span>004006<b>.2</b>,            not NM004006.2).</li>          <li>a genomic reference sequence starts with nucleotide 1; a genomic            reference sequence can not have negative numbers</li>          <li>for a coding DNA reference sequence, do you clearly state that <em>nucleotide
              numbering uses the A of the ATG translation initiation start site              as nucleotide 1</em>?</li>          <li>when you <span style="font-weight: bold; font-style: italic;">do              not</span> use a coding DNA reference sequence and <span style="font-style: italic; font-weight: bold;">do
              not</span> start nucleotide numbering <strong><em>with 1 at the A                of the ATG</em></strong> translation initiation start you should            not use descriptions preceded by "<em><strong>c.</strong></em>".</li>          <li>if legacy numbering is used, this can only be done <em><strong>in                addition to</strong></em> the approved nomenclature.</li>          <li>does your reference sequence contain introns?;&nbsp; not that <em><strong>NM_</strong></em>            reference sequences cover mature transcripts and do not contain            intronic sequences. </li>        </ul>      </li>      <li><b>Intronic variants</b> - do you indicate where the <em>reference          intron sequence</em> can be found ?<br>        The recommendation is to describe <i><b>intronic variants</b></i> in        the format "<i><b>c.89-2A&gt;G</b></i>" and not like "<i>IVS4-2A&gt;G</i>"        (see <a href="disc.html#IVS"><i>Discussion</i></a>). When the format "<i>IVS4-2A&gt;G</i>"        is used, it is essential to give a clear reference for <em><strong>intron
            / exon numbering</strong> </em>and to mention the <em>reference          sequence used for the intron</em>.</li>      <li><strong>Tabular overview</strong> - do you provide a clear,        unequivocal overview of all changes reported?<br>        Preferably, a publication contains a <em>tabular overview</em> of all        variants reported. This overview contains columns describing the change        at <em>DNA-level (absolutely essential) </em>and, optional, at <em>RNA
          </em>and <em>protein level</em>. When data on RNA and/or protein        level are provided, it should be made clear whether the data were <strong><em>deduced
            or experimentally verified</em></strong> (i.e. state explicitly when        RNA was analysed, e.g. to study the consequences of a variant affecting        splicing).</li>      <li><strong>Insertions</strong>        <ul type="disc">          <li>are insertions reported in the format c.51_52insT?<br>            Since it is not clear whether one means <strong><em>insertion at </em></strong>or<strong><em>                insertion after</em></strong> position 52, insertions should not            be reported as c.52insT but in the format c.51_52insT (see <a href="disc.html#ins"><i>Discussion</i></a>).</li>          <li>do you give the inserted sequence?<br>            Describing a variant like c.5439_430ins6 is <em><strong>not                sufficient</strong></em>, the inserted sequence (ins6, e.g.            TGCCAT)<em><strong> should always be mentioned</strong></em>.</li>          <li>are the insertions reported really insertions or are they in fact            duplications?<br>            Duplicating insertions should be described as duplications, not as            insertions; for the change CCAGTAAC to CCAGT<span style="font-weight: bold; font-style: italic;">GT</span>AAC
            the description is c.4_5dup (or c.4_5dupGT) is correct, c.5_6insGT            is not correct (see <a href="disc.html#dupins"><i>Discussion</i></a>).&nbsp;</li>        </ul>      </li>      <li><b>Most 3' position</b> - do you correctly assign the change to the        most 3' (or C-terminal for protein variants) position possible?<br>        For deletions, duplications and insertions the <em><strong>most 3'            position possible is arbitrarily assigned</strong></em> to have been        changed (see <a href="recs.html#repeat"><i>Recommendations</i></a>);        important especially in single residue (nucleotide or amino acid)        stretches or tandem repeats. Example CCAGT<span style="font-weight: bold; font-style: italic;">GT</span>AAC
        to CCAGTAAC is described as c.6_7del (or c.6_7delGT, not as c.3_4del or        c.4_5del.</li>      <li><strong>Recessive diseases </strong>- do you clearly describe which        changes are found in which combination?<br>        a publication describing sequence changes found in patients suffering        from a recessive disease should for each patient explicitly mention        which <em><strong>combination of changes</strong></em> was identified        (see <a href="recs.html#2in"><i>Recommendations</i></a>). Example        c.[76C&gt;T];[87G&gt;A] or c.[76C&gt;T];[?].<br>        <span lang="EN-GB"><b><i>NOTE:</i></b> this description differs from          that describing <b><i>several changes in one allele,</i></b> which          has the format c.[76A&gt;C;113G&gt;C].<o:p> </o:p> </span></li>      <li><strong>Range</strong> - the sign used to indicate a range is "_"        (underscore) and not a "-" (minus)?<br>        To prevent confusion, the <span style="font-style: italic;">underscore</span>        should be used to indicate a <span style="font-style: italic;">range</span>        and not the minus sign. The <span style="font-style: italic;">minus          sign</span> should only be used to indicate <span style="font-style: italic;">negative
          numbers</span>. The correct description to indicate a deletion of the        coding DNA nucleotides 12 to 14 is c.12_14del. Not correct is        c.12-14del, this describes a deletion of nucleotide -14 in the intron        directly preceding cDNA nucleotide 12 (see <a href="disc.html#range"><i>Discussion</i></a>).</li>      <li><b>Deletion </b>- do you indicate the first and last residue involved        in a deletion?<br>        A deletion of more than one residue should mention the first and last        residue deleted, separated using a "_" (underscore), e.g. c.21_24del or        p.Ala13_Gln16del. Descriptions like c.21del3 should not be used.</li>      <li><b>Describe at DNA-level</b> - do you describe all changes reported at        DNA-level?<br>        All changes reported must be described at DNA-level        <ul>          <li>when descriptions at protein level are given in the text, upon            first appearance, use a format like "<b><i>c.78G&gt;C (p.(Trp26Cys),                RNA not analysed)</i></b>" or "<b><i>c.78G&gt;C (p.Trp26Cys, RNA                analysed)</i></b>"</li>          <li>description of "<i>silent variants</i>" in the format            "p.(Leu54Leu) (or <em><strong>p.(L54L))" should not be used</strong></em>            (<i><a href="disc.html#silent">see Discussion</a></i>). Descriptions            should be given at DNA level. Descriptions like p.(Leu54Leu) are            non-informative and not unequivocal (there are five possibilities at            DNA level); a correct description is <b><i>c.162C&gt;G&nbsp;</i></b></li>        </ul>      </li>      <li><b>RNA protein level descriptions</b><br>        Recommendations exist to describe alternative transcripts deriving from        one allele (see <i><a href="recs.html#RNA">Recommendations</a></i>).        Since these descriptions may be rather complex to explain, it is wise to        include a link to the HGVS recommendations in the publication.</li>      <li><strong>Protein level descriptions</strong>        <ul type="disc">          <li><i><b>protein reference sequence</b></i> - the protein reference            sequence should represent the<b><i> </i></b>primary translation            product, not a processed mature protein, and thus include any signal            peptide sequences (see <a href="recs.html#protein"><i>Recommendations</i></a>).</li>          <li><img src="new.gif" border="0" height="12" width="31"><em><strong>Ter&nbsp;
                / *</strong></em> - do you use Ter or * to indicate a            translation stop codon; the X is not allowed anymore (see <a href="versioning.html#important"><em>Important
                changes</em></a>)</li>          <li><b><i>one/three letter amino acid code</i> </b>- are the correct            amino acid codes used at protein level?;&nbsp; several amino acids            start with the same initial letter (Ala, Arg, Asn, Asp start with <i><b><u>A</u></b></i>,            Gln, Glu, Gly with <i><b><u>G</u></b></i>, Leu, Lys with <b><i><u>L</u></i></b>,            Phe, Pro with <u><b><i>P</i></b></u> and Thr, Tyr with <i><b><u>T</u></b></i>)            but that initial letter is used as one-letter-amino-acid-code for            only one of these (see <i><a href="disc.html#1or3">Discussion</a> </i>and<i>              <a href="codon.html#aalist">Codons and amino acids</a></i>)</li>          <li><strong><i>initiating methionine (Met1)</i></strong> - <i>p.Met1?</i>            denotes that amino acid Methionine-1 (translation initiation site)            is changed and that it is unclear what the consequence of this            change is. When experimental data show that no protein is made, the            description <i>p.0</i> should be used. The description p.Met1Val is            not allowed (see <a href="disc.html#Met"><i>Discussion</i></a>)</li>          <li><b><i>no-stop change</i></b> - recommendations have recently been            made to describe substitutions in the stop codon, so called <i><b>no-stop
                changes</b>&nbsp;</i>like p.*110Tyrext*16 (see <a href="recs-prot.html#extp"><i>Recommendations</i></a>)</li>        </ul>      </li>      <li><b>Polymorphisms</b><br>        Do not describe polymorphic variants as c.127A/G (or p.43I/V). A        description of a variant should be <i><b>neutral</b></i> and        polymorphisms and pathogenic changes should not be described differently        (see <a href="disc.html#polymorphism"><i>Discussion</i></a>). Correct        descriptions are c.127A&gt;G and p.(Ile43Val).</li>    </ol>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">MutNomen
        homepage</a> |<br>      | <b>Recommendations:</b>&nbsp; <a href="recs-DNA.html">DNA</a>,&nbsp; <a        href="recs-RNA.html">RNA</a>,&nbsp;
      <a href="recs-prot.html">protein</a>, <a href="uncertain.html">uncertain</a>      |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="standards.html"><font          face="Verdana, Arial, Helvetica, sans-serif">Symbols,
          codons, etc.</font></a> | <a href="history.html">History</a> |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef /        symbols</a>,&nbsp; <a href="examplesDNA.html">DNA</a>,&nbsp; <a href="examplesRNA.html">RNA</a>,&nbsp;
      <a href="examplesAA.html">protein</a> | </p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright
                  ©</a> </font><font size="1">HGVS 2007 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>