<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	13-04-2015		URL: 	comments001.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>Sequence variant descriptions - Community Concultation 001</title>    <style type="text/css"><!--  h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body style="         color: black; background-color: white;" link="#0000FF" vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center"><font size="6">Open for comments - SVD-WG001</font>            </h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Published: May 14, 2015<br>                  <span style="color: red;">Closes:</span>&nbsp; July 16, 2015<br>                </i></font></h4>          </td>        </tr>      </tbody>    </table>    <hr>    <h2>Proposal</h2>    <hr>    <p>SVD-WG001 suggests to extend the current HGVS recommendations for the      description of sequence variants with the proposal below. Comments can be      sent to the HGVS/HVP/HUGO Sequence Variant Description Working Group      (SVD-WG), addressed to "<span style="font-weight: bold; font-style: italic;">Varnomen
        @ variome.org</span>", <span style="font-weight: bold; font-style: italic;">Subject:
        SVD-WG001</span>. Comments need to be sent <span style="color: red; font-weight: bold; font-style: italic;">ultimately
        July 16, 2015</span>. </p>    <h3>No change</h3>    <p>When a variant screen was performed but no change detected this can be      described using the character "=" (equal to). The description should      include the position(s) screened. Of course, as always, it is mandatory to      mention the reference sequence used.</p>    <p><span style="font-weight: bold;"><span style="text-decoration: underline;">Examples;</span></span></p>    <ul>      <li>c.123G=<br>        indicates that a screen was performed to determine the identity of        nucleotide c.123 showing it was a "G" like in the coding DNA reference        sequence (the nucleotide was not changed).</li>      <ul>        <li>r.473u=<br>          indicates that a screen was performed to determine the identity of          nucleotide r.473 showing it was a "u" like in the RNA reference          sequence (the nucleotide was not changed)</li>        <li>p.(Arg41=)<br>          indicates the predicted consequence of a change identified at DNA          level (<span style="font-style: italic;">RNA not analysed</span>) is          that the amino acid residue "Arg41" does not change<br>          <span style="font-weight: bold; font-style: italic;">NOTE:</span>          depending on the amino acid, up to 4 different DNA changes may leave          the amino acid unchanged<br>        </li>      </ul>      <li>c.[123G=;456A=;789C=]&nbsp; -&nbsp; indicates that a screen was        performed to determine the identity of nucleotides c.123, c.456 and        c.789 (all on the same allele) showing they were all not changed (equal        to the coding DNA reference sequence).<br>        <br>      </li>      <li>NM_001849.3(COL6A2):c.1_*1=&nbsp; -&nbsp; indicates that a screen was        performed to determine the entire coding DNA sequence of the COL6A2 gene        (nucleotides c.1 to c.*1 in NM_001849.3, exons only), no changes were        detected.<br>      </li>      <li>LRG_476(COL6A2):g.4950_39800=&nbsp; -&nbsp; indicates that a screen        was performed to determine the entire genomic sequence of the COL6A2        gene (nucleotides g.4950 to g.39800 in LRG_476, exon ànd intron), no        changes were detected.</li>    </ul>    <hr>    <p align="center"><font face="Verdana, Arial, Helvetica, sans-serif">| <a href="index.html">Varnomen
          homepage</a> | <a href="comments.html">Comments homepage</a> | <a href="standards.html">Symbols,
          codons, etc.</a> |<br>      </font></p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright
                  ©</a> </font><font size="1">HGVS 2010 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>