<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	17-11-2002		URL: 	recs.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>Sequence variant descriptions - Community Concultation 003</title>    <style type="text/css"><!--  h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body style="           color: black; background-color: white;" link="#0000FF"    vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center"><font size="6">Open for comments - SVD-WG003</font>            </h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Published: May 14, 2015<br>                  <span style="color: red;">Closes:</span>&nbsp; July 16, 2015<br>                </i></font></h4>          </td>        </tr>      </tbody>    </table>    <hr>    <h2>Proposal</h2>    <hr>    <p>SVD-WG003 suggests to extend the current HGVS recommendations for the      description of sequence variants with the proposal below. Comments can be      sent to the HGVS/HVP/HUGO Sequence Variant Description Working Group      (SVD-WG), addressed to "<span style="font-weight: bold; font-style: italic;">Varnomen
        @ variome.org</span>", <span style="font-weight: bold; font-style: italic;">Subject:
        SVD-WG003</span>. Comments need to be sent <span style="color: red; font-weight: bold; font-style: italic;">ultimately
        July 16, 2015</span>). </p>    <h3>Exon del/dup descriptions</h3>    Deletions<span style="font-weight: bold;"> (</span>duplications) detected by    MLPA tests (testing all exons) should be described using the format    c.(649+1_650-1)_(1331+1_1332-1)del.<br>    <br>    <span style="font-weight: bold; font-style: italic;">NOTE:</span>&nbsp; the    format suggested originally to describe deletions/duplications detected by    MLPA tests (testing all exons) is like c.650-?_1331+?del, i.e. a deletion    starting in the intron preceding c.650 (first nucleotide of the first    deleted exon) and ending in the intron after c.1331 (last nucleotide of the    last deleted exon). This format <span style="font-weight: bold; font-style: italic;">conflicts
      with</span> the basic format to describe deletions/duplications where the    break point was not sequenced but where a region of uncertainty is present    (see <a href="uncertain.html#exondel"><span style="font-style: italic;">Uncertainties</span></a>).
    This format is (last-normal_first-changed)_(last-changed_first-normal). The    Committee considered the format c.(649_650)_(1331_1332)del which is shorter    but not specifically indicates that the expected break points of the change    are in an intron.<br>    <span style="font-weight: bold; font-style: italic;">NOTE:&nbsp;</span> to    be 100% accurate one should not use the position of the introns in the    description of the change but the location of the probes actually tested in    the assay. The current proposal bears in mind that: i) it is not    straightforward what probe position to use (see <a href="file:///Users/Boomheide/Files/WWW/HGVS/uncertain.html#exondel"><span        style="font-style: italic;">Uncertainties</span></a>),
    ii) the additional detail would hardly add useful information, and iii) we    should preferably not deviate too far from the original proposal (i.e. to    use flanking intron locations). When one wants to use probe locations (e.g.    with breakpoints in large exons) the suggestion is to use the central    position of the probes in variant descriptions.<br>    <p><span style="font-weight: bold;"><span style="text-decoration: underline;">Examples;</span></span></p>    <ul>      <li>c.(649+1_650-1)_(1331+1_1332-1)del&nbsp; describes a deletion of exons        8 to 11 (DMD gene) with breakpoints in the introns between nucleotides        c.649 and c.650 (intron 7) on one side and nucleotides c.1331 and c.1332        (intron 11) on the other side. The flanking exons (7 and 12) on either        side were tested and found not deleted.<br>        <span style="font-weight: bold; font-style: italic;">NOTE:</span>&nbsp;        c.(649+1_650-1)_(1331+1_1332-1)dup describes a duplication of the same        region.</li>      <li>c.(516+1_517-1)_(1970_6774)del&nbsp; describes a deletion in the BRCA2        gene with one break point in the intron between nucleotides c.516 and        c.517 (intron 6) and one breakpoint in exon 10 between nucleotides        c.1970 and c.6774. Flanking exons (5 and 11) on each side were tested        and found not deleted. Description of the break point in exon 10 was        derived from the two MLPA probes tested, taken the central nucleotide        position per probe.<br>        <span style="font-weight: bold; font-style: italic;">NOTE:</span>&nbsp;        c.(516+1_517-1)_(1970_6774)dup describes a duplication of the same        region.</li>    </ul>    <p><span style="font-weight: bold; font-style: italic;">NOTE:</span>&nbsp;      the description "<span style="font-weight: bold; font-style: italic;">dup</span>"      may <span style="font-weight: bold; font-style: italic;">by definition</span>      (see <a style="font-style: italic;" href="standards.html#define">Standards</a>)      only be used when the additional copy is located directly 3'-flanking the      original copy (a tandem duplication). In most cases there will be no      experimental proof, one simply detects the presence of an additional copy      that in theory can be anywhere in the genome (inserted / transposed).      Discussions are ongoing how to include this uncertainty best in the      description: a proposal will follow later.</p>    <ul>    </ul>    <hr>    <p align="center"><font face="Verdana, Arial, Helvetica, sans-serif">| <a href="index.html">Varnomen
          homepage</a> | <a href="comments.html">Comments homepage</a> | <a href="standards.html">Symbols,
          codons, etc.</a> |<br>      </font></p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright
                  ©</a> </font><font size="1">HGVS 2010 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>