<html>
<!--
	Author: 	Dr. Johan T. den Dunnen
	Date: 	17-11-02
	URL: 	http://www.dmd.nl/mutnomen_current.html
-->

<head>
<title>HGVS recommendations: examples DNA</title>
<style type="text/css">

<!--  

h1 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h2 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #ff9900;

}

h3 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h4 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h5 {
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #ffffff;

}

.norm {

 font-weight : normal;

 }
 -->
</style>
</head>

<body text="#000000" bgcolor="#FFFFFF" link="#0000FF" vlink="#9999ff">

<table width="100%" border="0" cellspacing="0" cellpadding="10" bgcolor="#CCFF99">
  <tr>
    <td width="25%" valign="top" bgcolor="#FFFFFF"><a href="http://www.HGVS.org"><img
    src="small.gif" width="155" height="94" border="0"></a></td>
    <td width="75%" valign="CENTER" bgcolor="#FFFFFF" bordercolor="#FFFFFF"><h3 align="center">Description
    of sequence changes:&nbsp;<br>
    examples DNA-level</h3>
    </td>
  </tr>
</table>

<table width="100%" border="0" cellspacing="0" cellpadding="10">
  <tr>
    <td width="100%" valign="BOTTOM" bgcolor="#0099cc"><h4 align="center"><font size="3"
    color="#FFFFFF"><i><br>
    Last modified July 16, 2013</i></font></h4>
    </td>
  </tr>
</table>

<p align="center"><i>Since references to WWW-sites are not yet acknowledged as citations,
please mention <a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den Dunnen JT
and Antonarakis SE (2000). Hum.Mutat. 15: 7-12</a> when referring to these pages.</i></p>

<hr>

<h2>Contents</h2>

<ul>
  <li><a HREF="#intro">Introduction</a> <ul>
      <li>Reference Sequence</li>
      <li>numbering</li>
    </ul>
  </li>
  <li><b>Examples&nbsp; -&nbsp; DNA changes</b> <ul>
      <li><a HREF="#general">general</a> </li>
      <li>seven elementary changes <ul>
          <li><a HREF="#sub">substitution</a></li>
          <li><a HREF="#del">deletion</a>&nbsp; <i>(incl. in single nucleotide stretches)</i> </li>
          <li><a HREF="#dup">duplication</a></li>
          <li><a HREF="#ins">insertion</a></li>
          <li><a href="#var">sequence repeat variability</a>&nbsp; <i>(CA, CAG, ...)</i> </li>
          <li><a HREF="#inv">inversion</a> </li>
          <li><a href="#con">gene conversion</a></li>
          <li><a HREF="#trans">translocation</a> </li>
          <li>transposition</li>
        </ul>
      </li>
      <li><a HREF="#complex">complex rearrangements</a> <ul>
          <li>deletion/insertion <i>(indel)</i> </li>
        </ul>
      </li>
      <li><a HREF="#misc">miscellaneous</a> <ul>
          <li>two changes in one allele </li>
          <li>two changes in one individual with alleles unknown</li>
          <li>recessive diseases - changes in different alleles (known and unknown) <ul>
              <li><img border="0" src="new.gif" width="31" height="12">probably homozygous</li>
              <li>several changes in two alleles</li>
              <li>changes in different genes</li>
            </ul>
          </li>
        </ul>
      </li>
      <li><a href="uncertain.html">uncertainties</a>&nbsp; <i>(exact position not known; Southern
        blot, PCR, arrayCGH, SNP-array, ...)</i></li>
    </ul>
  </li>
  <li><a HREF="examplesRNA.html"><b>Examples&nbsp; -&nbsp; RNA changes</b></a></li>
  <li><b><a HREF="examplesAA.html">Examples&nbsp; -&nbsp; protein changes</a></b></li>
</ul>

<hr>

<h2><a NAME="intro"></a>Introduction</h2>

<p>Within this page examples will be given for the description of sequence variations. The
examples will be given independently for descriptions at DNA, RNA and protein level. All
examples are described <i><b>relative to a Reference Sequence</b></i>, depending on the
level a genomic or coding DNA sequence (DNA-level), an mRNA sequence (RNA-level) or an
amino acid sequence (protein level).&nbsp;</p>

<hr>

<h3><a NAME="RefSeq"></a>Reference sequence DNA-level</h3>

<hr>

<p>Within this page examples will be given for the description of sequence variations in a
DNA sequence. For other examples go to those describing <a HREF="examplesRNA.html">changes
in RNA</a>. Examples for protein level are given at <a HREF="examplesAA.html">the protein
page</a>. All examples are described <i><b>relative to a Reference Sequence</b></i>, here
a coding DNA sequence.</p>

<table BORDER="1" WIDTH="688" align="center">
  <tr>
    <th WIDTH="224" COLSPAN="2"><b>Part of gene</b></th>
    <th WIDTH="134" ALIGN="center">nucleotide numbering<br>
    <i><b>genomic</b><br>
    </i>Reference Sequence&nbsp;</th>
    <th WIDTH="163" ALIGN="center">nucleotide numbering<br>
    <i><b>coding DNA</b></i><br>
    Reference Sequence</th>
    <th WIDTH="139" ALIGN="center">&nbsp;nucleotide numbering<br>
    <i><b>protein</b></i><br>
    Reference Sequence</th>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">5' gene flanking region</td>
    <td WIDTH="134" ALIGN="center">1 to 270</td>
    <td WIDTH="163" ALIGN="center">(-300 to -31)</td>
    <td WIDTH="139" ALIGN="center">&nbsp;-</td>
  </tr>
  <tr>
    <td WIDTH="70" ROWSPAN="2"><b>exon 1</b></td>
    <td WIDTH="148">5' UTR</td>
    <td WIDTH="134" ALIGN="center">271 to 300</td>
    <td WIDTH="163" ALIGN="center">-30 to -1</td>
    <td WIDTH="139" ALIGN="center">&nbsp;-</td>
  </tr>
  <tr>
    <td WIDTH="148"><b>coding region</b></td>
    <td WIDTH="134" ALIGN="center">301 to 312</td>
    <td WIDTH="163" ALIGN="center">1 to 12</td>
    <td WIDTH="139" ALIGN="center">1 to 4</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">intron 1</td>
    <td WIDTH="134" ALIGN="center">313 to 412</td>
    <td WIDTH="163" ALIGN="center"><p ALIGN="center"><i>12+1 ... 12+50,<br>
    13-50 ... 13-1</i> </td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2"><b>exon 2</b></td>
    <td WIDTH="134" ALIGN="center">413 to 488</td>
    <td WIDTH="163" ALIGN="center">13 to 88</td>
    <td WIDTH="139" ALIGN="center">5 to 29 (30)</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">intron 2</td>
    <td WIDTH="134" ALIGN="center">489 to 688</td>
    <td WIDTH="163" ALIGN="center"><i>88+1 ... 88+100,<br>
    89-100 ... 89-1&nbsp;</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2"><b>exon 3</b></td>
    <td WIDTH="134" ALIGN="center">689 to 723</td>
    <td WIDTH="163" ALIGN="center">89 to 123</td>
    <td WIDTH="139" ALIGN="center">30 to 41</td>
  </tr>
  <tr>
    <td WIDTH="70">intron 3</td>
    <td WIDTH="148">contains rare alternatively spliced exon from 800 to 859 (coding DNA
    123+77 to 123+136)</td>
    <td WIDTH="134" ALIGN="center">724 to 1023</td>
    <td WIDTH="163" ALIGN="center"><i>123+1 ... 123+150,<br>
    124-150 ... 124-1&nbsp;</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2"><b>exon 4</b></td>
    <td WIDTH="134" ALIGN="center">1024 to 1200</td>
    <td WIDTH="163" ALIGN="center">124 to 300</td>
    <td WIDTH="139" ALIGN="center">42 to 100</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">intron 4</td>
    <td WIDTH="134" ALIGN="center">1201 to 1600</td>
    <td WIDTH="163" ALIGN="center"><i>300+1 ... 300+200,<br>
    301-200 ... 301-1&nbsp;</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="70" ROWSPAN="2"><b>exon 5</b></td>
    <td WIDTH="148"><b>coding region</b></td>
    <td WIDTH="134" ALIGN="center">1601 to 1630</td>
    <td WIDTH="163" ALIGN="center">301 to 330</td>
    <td WIDTH="139" ALIGN="center">101 to 109</td>
  </tr>
  <tr>
    <td WIDTH="148">3' UTR, containing a (CA)<sub>7</sub>-stretch from nucleotides 1700 to
    1713 (coding DNA *70 to *83); poly-A addition site at 1825 (coding DNA *195) </td>
    <td ALIGN="center" WIDTH="134">1631 to 1850</td>
    <td WIDTH="163" ALIGN="center">*1 to *220</td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">3' gene flanking region</td>
    <td WIDTH="134" ALIGN="center">1851 to 2000</td>
    <td WIDTH="163" ALIGN="center">(*221 to *370)</td>
    <td WIDTH="139" ALIGN="center">-&nbsp;</td>
  </tr>
</table>

<p><i><b>NOTE:</b> nucleotides in introns in the 5' UTR are numbered like -23+1, -23+2,
..., -22-2, -22-1. Nucleotides in introns in the 3' UTR are numbered like *154+1, *154+2,
..., *155-2, *155-1.&nbsp;</i> </p>

<p><b><big>Legend:</big></b><br>
Reference sequence of imaginary gene used for the exaples given on this page. Nucleotide
+1 in the coding DNA reference sequence is the A of the ATG translation initiation codon.
Abbreviations used: nt = nucleotide, UTR = untranslated region of the mRNA. For a picture
of part of this hypothetical sequence <i><a href="refseq_figure.html">see Figure</a></i>.</p>

<hr>

<h3><a NAME="general"></a>General</h3>

<hr>

<p>Publications reporting changes in different sequences (genes) or which report linkage
or association studies should prevent any confusion regarding which variant resides in
which sequence. An easy way to achieve this is to include an unequivocal identifier to the
reference sequence used in the description, e.g. <i><b>NM_004006.2:c.3G&gt;T</b></i> or <em><strong>DMD:c.3G&gt;T</strong></em>
(<i><a href="disc.html#SNP">see Discussion</a></i>).</p>

<hr>

<h3><a NAME="sub"></a>Substitutions</h3>

<hr>

<p>Substitutions are designated by a <i><b>&quot;&gt;&quot;</b></i>-character after the
number of the affected nucleotide. 

<ul>
  <li><b>5' gene flanking region&nbsp; -&nbsp; </b>T to C substitution of nt 241 (located 30
    nucleotides upstream of the transcription initiation site, i.e. in the promoter region) <table
    WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.241T&gt;C </td>
        <td WIDTH="300" ALIGN="center">- </td>
      </tr>
    </table>
  </li>
  <li>&nbsp;<b>5' UTR&nbsp; -&nbsp; </b>G to A substitution of nt 289, 12 nucleotides upstream
    of the ATG translation initiation codon (coding DNA -12). For nucleotide numbering in a
    case where the ATG is not in exon 1 <a href="refseq.html#IVSin5"><i>see here</i></a>.&nbsp;
    <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.289G&gt;A </td>
        <td WIDTH="300" ALIGN="center">c.-12G&gt;A </td>
      </tr>
    </table>
  </li>
  <li><b>translation initiation codon&nbsp; -&nbsp; </b>A to C substitution of nt 301, i.e. nt
    1 of the coding region (coding DNA 1) <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.301A&gt;C </td>
        <td WIDTH="300" ALIGN="center">c.1A&gt;C </td>
      </tr>
    </table>
  </li>
  <li><b>coding region&nbsp; -&nbsp; </b>G to C substitution of nt 423 in exon 2, i.e. nt 23
    of the coding region (coding DNA 23) <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.423G&gt;C </td>
        <td WIDTH="300" ALIGN="center">c.23G&gt;C </td>
      </tr>
    </table>
  </li>
  <li><b>intron&nbsp;</b>(regarding the numbering of intronic nucleotides <a
    href="disc.html#IVS"><i>see Discussion</i></a>) <ul>
      <li><i><u>5' part intron</u>&nbsp; - </i>T to G substitution of the second nt in the intron
        (88+2) positioned between coding DNA nucleotides 88 and 89 (intron 2) <table WIDTH="612">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="297" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.490T&gt;G </td>
            <td WIDTH="297" ALIGN="center">c.88+2T&gt;G </td>
          </tr>
        </table>
      </li>
      <li><i><u>3' part intron</u>&nbsp; - </i>G to T substitution of the last nt of the
        intron&nbsp; (89-1) positioned between coding DNA nucleotides 88 and 89 (intron 2) <table
        WIDTH="611">
          <tr>
            <th WIDTH="302" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="295" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="302" ALIGN="center">g.688G&gt;T </td>
            <td WIDTH="295" ALIGN="center">c.89-1G&gt;T </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><b>alternatively spliced exon&nbsp; -&nbsp; </b>G to C substitution of intronic nt 812
    (coding DNA 123+89) <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.812C&gt;T </td>
        <td WIDTH="300" ALIGN="center">c.123+89C&gt;T </td>
      </tr>
    </table>
  </li>
  <li><b>translation termination codon&nbsp; -&nbsp; </b>G to C substitution of nt 1629, i.e.
    nt 329 of the coding region (coding DNA 329) <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.1629G&gt;C </td>
        <td WIDTH="300" ALIGN="center">c.329G&gt;C </td>
      </tr>
    </table>
  </li>
  <li><b>3' UTR&nbsp; -&nbsp; </b>T to A substitution of nt 1700 (coding DNA 70), located in
    the 3' UTR (70 nucleotides downstream of the termination codon) <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.1700T&gt;A </td>
        <td WIDTH="300" ALIGN="center">c.*70T&gt;A </td>
      </tr>
    </table>
  </li>
  <li><b>3' gene flanking region&nbsp; -&nbsp; </b>C to A substitution of nt 1923 (located 293
    nucleotides downstream of the gene, i.e. the polyA-addition site) <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.1923C&gt;A </td>
        <td WIDTH="300" ALIGN="center">c.*293C&gt;A</td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<h3><a NAME="del"></a>Deletion</h3>

<hr>

<p>Deletions are designated by <i><b>&quot;del&quot;</b></i> after a description of the
deleted segment, i.e. the first (and last) nucleotide(s) deleted (see also <i><a
href="disc.html#largedel">Discussion</a></i>). To describe deletions with unknown
breakpoints, e.g. based on Southern blotting, PCR, arrayCGH, SNP array data, etc. <i><a
href="uncertain.html">see Uncertainties</a></i>. 

<ul>
  <li><b>single nucleotide deletion&nbsp; -&nbsp; </b>deletion of nt 13 of the coding region <table
    WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.413del&nbsp;&nbsp; (g.413delG) </td>
        <td WIDTH="300" ALIGN="center">c.13del&nbsp;&nbsp; (c.13delG) </td>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.304del&nbsp;&nbsp; (g.304delG)<br>
        <i>(not g.303del / g.303delG)</i> </td>
        <td WIDTH="300" ALIGN="center">c.4del&nbsp;&nbsp; (c.4delG)<br>
        <i>(not c.3del / c.3delG)</i> </td>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.1598delG<br>
        <i>(not g.1596del / g.1596delG)</i> </td>
        <td WIDTH="300" ALIGN="center">c.301-3del&nbsp;&nbsp; (c.301-3delT)<br>
        <i>(not c.301-5del or&nbsp; c.301-5delT)</i> </td>
      </tr>
    </table>
  </li>
  <li>&nbsp;<b>deletion of more then 1 nucleotide&nbsp;</b> <ul>
      <li>deletion of nucleotides -11 to -4 in the 5' UTR<table WIDTH="612">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="297" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.290_297del</td>
            <td WIDTH="297" ALIGN="center">c.-11_-4del&nbsp;&nbsp;&nbsp;</td>
          </tr>
        </table>
      </li>
      <li>deletion of nucleotides 92 to 94 (GAC) of the coding region <table WIDTH="612">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="297" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.692_694del<br>
            (g.692_694delGAC)&nbsp; </td>
            <td WIDTH="297" ALIGN="center">c.92_94del&nbsp;&nbsp;&nbsp;<br>
            (c.92_94delGAC)&nbsp; </td>
          </tr>
        </table>
      </li>
      <li>deletion of nucleotides *8 to *21 in the 3' UTR<table WIDTH="612">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="297" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.1638_1651del</td>
            <td WIDTH="297" ALIGN="center">c.*8_*21del&nbsp;&nbsp;&nbsp;</td>
          </tr>
        </table>
        <p>deletion across the exon 3 / intron 3 border, nucleotides 120 to 123 of the coding
        region (exon 3) and the first 48 nucleotides of intron 3 (nucleotides 123+1 to 123+48) </p>
        <table WIDTH="611">
          <tr>
            <th WIDTH="302" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="295" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="302" ALIGN="center">g.720_771del</td>
            <td WIDTH="295" ALIGN="center">c.120_123+48del</td>
          </tr>
        </table>
      </li>
      <li>deletion across the intron 3 / exon 4 border, the last 12 nucleotides of intron 3
        (nucleotides 124-12 to 124-1) and nucleotides 124 to 129 of the coding region (exon
        4)&nbsp; <table WIDTH="611">
          <tr>
            <th WIDTH="302" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="295" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="302" ALIGN="center">g.1012_1029del</td>
            <td WIDTH="295" ALIGN="center">c.124-12_129del </td>
          </tr>
        </table>
      </li>
      <li>deletion of a TG dinucleotide in the sequence ATGTTG<u><strong>TG</strong></u>CC to
        ATGTTG<b>_</b>CC <table WIDTH="611">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="296" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.307_308del&nbsp;<br>
            (g.307_308delTG)<br>
            <b><i>NOT</i></b> g.305_306del </td>
            <td WIDTH="296" ALIGN="center">c.7_8del&nbsp;<br>
            &nbsp;(c.7_8delTG)<br>
            <i><b>NOT</b></i> c.5_6del)&nbsp; </td>
          </tr>
        </table>
      </li>
      <li>deletion of an A nucleotide in the sequence CA<b><u>A</u></b>gt... / ..agAAG to CAgt...
        / ..agAAG <table WIDTH="611">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="296" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.723del<br>
            (g.723delA) </td>
            <td WIDTH="296" ALIGN="center">c.123del&nbsp;<br>
            &nbsp;(c.123delA)<br>
            <i><b>NOT</b></i> c.125delA </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><b>variability in short sequence repeat - </b><i><a href="#var">see below</a></i> </li>
  <li><b>(multi) exon deletion</b> <ul>
      <li><u><i>breakpoint not sequenced</i></u> <ul>
          <li>deletion of exons 2 to 4 (e.g. detected on Southern blot, <i><a HREF="disc.html#del?">see
            Discussion</a>)</i> </li>
        </ul>
        <table WIDTH="608">
          <tr>
            <th WIDTH="300" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="294" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="300" ALIGN="center">- </td>
            <td WIDTH="294" ALIGN="center">c.13-?_300+?del &nbsp; </td>
          </tr>
        </table>
        <ul>
          <li>deletion of the entire gene; coding DNA reference sequence runs from -30 (cap site) to
            *220 (polyA-addition site); <i><a href="recs.html#uncertain">see Recommendations</a></i> </li>
        </ul>
        <table WIDTH="608">
          <tr>
            <th WIDTH="300" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="294" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="300" ALIGN="center">- </td>
            <td WIDTH="294" ALIGN="center">c.(?_-30)_(*220_?)del&nbsp;&nbsp;&nbsp; </td>
          </tr>
        </table>
      </li>
      <li><u><i>breakpoints sequenced</i></u> - deletion of exons 2 to 4; the sequences of the
        deletion breakpoints need to be submitted to a sequence database (Genbank, EMBL, DDJB) and
        the accession number should be given <table WIDTH="606">
          <tr>
            <th WIDTH="299" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="293" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="299" ALIGN="center">g.390_1458del&nbsp;&nbsp; (g.390_1458del1069) </td>
            <td WIDTH="293" ALIGN="center">c.13-23_301-143del&nbsp;&nbsp; (c.13-23_301-143del1069) </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><b>large deletion fusing two genes</b><i>&nbsp; (<a HREF="disc.html#fusion">see
    Discussion</a>)</i> <ul>
      <li><i><u>genes with identical transcriptional orientation</u></i> - a large deletion,
        starting in intron 4, removes the entire 3' end of the gene and fuses it to position 457
        (intron position 233+17) of the gene XYZ <table WIDTH="606">
          <tr>
            <th WIDTH="297" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="295" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="297" ALIGN="center">g.1458_XYZ:457del </td>
            <td WIDTH="295" ALIGN="center">c.301-143_XYZ:233+17del </td>
          </tr>
        </table>
      </li>
      <li><i><u>genes with opposite transcriptional orientation</u></i> - a large deletion,
        starting in intron 4, removes the entire 3' end of the gene and fuses it to position 457
        (intron position 233+17) of the XYZ-gene, which has an opposite transcriptional
        orientation (<i>indicated by the &quot;o&quot;</i>) <table WIDTH="607">
          <tr>
            <th WIDTH="298" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="295" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="298" ALIGN="center">g.1458_oXYZ:457del </td>
            <td WIDTH="295" ALIGN="center">c.301-143_oXYZ:233+17del </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="dup"></a>Duplication</h3>

<hr>

<p>Duplications are designated by <i><b>&quot;dup&quot;</b></i> after a description of the
duplicated segment, i.e. the first (and last) nucleotide(s) duplicated (even when a
mono-nucleotide is duplicated, <i><a href="recs-DNA.html#dup">see Recommendations</a></i>).
To describe duplications with unknown breakpoints, e.g. based on Southern blotting, PCR,
arrayCGH, SNP array data, etc. see also <i><a href="uncertain.html">Uncertainties</a></i>.

<ul>
  <li><b>single nucleotide duplication&nbsp; -&nbsp; </b>duplication of nt 13 of the coding
    region <table WIDTH="654">
      <tr>
        <th WIDTH="340" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="300" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="340" ALIGN="center">g.413dup&nbsp;&nbsp; (g.413dupG) </td>
        <td WIDTH="300" ALIGN="center">c.13dup&nbsp;&nbsp; (c.13dupG) </td>
      </tr>
    </table>
  </li>
  <li>&nbsp;<b>several nucleotide duplication&nbsp;</b> <ul>
      <li>duplication of nucleotides 92 and 94 (GAC) of the coding region <table WIDTH="612">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="297" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.692_694dup<br>
            (g.692_694dupGAC)&nbsp; </td>
            <td WIDTH="297" ALIGN="center">c.92_94dup&nbsp;&nbsp;&nbsp;<br>
            (c.92_94dupGAC)&nbsp; </td>
          </tr>
        </table>
      </li>
      <li>duplication across the exon 3 / intron 3 border, nucleotides 120 to 123 of the coding
        region (exon 3) and the first 48 nucleotides of intron 3 (nucleotides 123+1 to 123+48) <table
        WIDTH="611">
          <tr>
            <th WIDTH="302" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="295" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="302" ALIGN="center">g.720_771dup</td>
            <td WIDTH="295" ALIGN="center">c.120_123+48dup&nbsp; </td>
          </tr>
        </table>
      </li>
      <li>duplication of a TG dinucleotide in the sequence ATGTTGTGCC to ATGTTGTG<b><u>TG</u></b>CC
        <table WIDTH="611">
          <tr>
            <th WIDTH="301" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="296" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="301" ALIGN="center">g.307_308dup&nbsp;<br>
            (g.307_308dupTG)<br>
            <b><i>NOT</i></b> g.305_306dup </td>
            <td WIDTH="296" ALIGN="center">c.7_8dup&nbsp;<br>
            (c.7_8dupTG)<br>
            <i><b>NOT</b></i> c.5_6dup)&nbsp; </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><b>variability in short sequence repeat - </b><i><a href="#var">see below</a></i> </li>
  <li><b>(multi) exon duplication<br>
    </b>note that a duplication <strong><em>by definition</em></strong> (<em><a
    href="standards.html#define">see Standards</a></em>) should only be used when the
    additional copy is located directly 3'-flanking the original copy (a tandem duplication).
    In most cases there will be no experimental proof of this, the additional copy may reside
    anywhere in the gene/genome (i.e. inserted/transposed). It seems thus better to describe
    exonic duplications using the format for sequence repeat variability, i.e. <i><b>c.88-?_301+?[2]</b></i>
    (<em><a href="recs-DNA.html#var">see Recommendations</a></em> and <em><a href="#var">below</a></em>).<ul>
      <li><u><i>breakpoints not sequenced</i></u> - duplication of exons 2 to 4 (e.g. detected on
        Southern blot, <i><a HREF="disc.html#del?">see Discussion</a>)</i> <table WIDTH="608">
          <tr>
            <th WIDTH="300" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="294" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="300" ALIGN="center">- </td>
            <td WIDTH="294" ALIGN="center">c.13-?_300+?[2] <br>
            or c.13-?_300+?dup </td>
          </tr>
        </table>
      </li>
      <li><u><i>breakpoints not sequenced</i></u> - duplication of the entire gene; coding DNA
        reference sequence runs from -30 (cap site) to *220 (polyA-addition site); <i><a
        href="recs.html#uncertain">see Recommendations</a></i> <table WIDTH="608">
          <tr>
            <th WIDTH="300" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="294" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="300" ALIGN="center">- </td>
            <td WIDTH="294" ALIGN="center">c.(?_-30)_(*220_?)[2] <br>
            or c.(?_-30)_(*220_?)dup </td>
          </tr>
        </table>
      </li>
      <li><u><i>breakpoints sequenced</i></u> - duplication of exons 2 to 4<table WIDTH="606">
          <tr>
            <th WIDTH="299" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="293" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="299" ALIGN="center">g.390_1458dup</td>
            <td WIDTH="293" ALIGN="center">c.13-23_301-143dup</td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="ins"></a>Insertion</h3>

<hr>

<p>Insertions are designated by <i><b>&quot;ins&quot;</b></i> after the nucleotides
flanking the insertion. <i><b>NOTE:</b></i> duplicating insertions (incl. duplication of a
mono-nucleotide) should be described as duplications (<a HREF="#dup"><i>see above</i></a>).

<ul>
  <li><b>single nucleotide insertion&nbsp; -&nbsp; </b>insertion of a T between nucleotides 51
    and 52 of the coding region <table WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.451_452insT </td>
        <td WIDTH="305" ALIGN="center">c.51_52insT </td>
      </tr>
    </table>
  </li>
  <li>&nbsp;<b>several nucleotide insertion</b> <ul>
      <li>insertion of a GAGA-sequence between nucleotides 51 and 52 of the coding region <table
        WIDTH="604">
          <tr>
            <th WIDTH="296" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="294" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="296" ALIGN="center">g.451_452insGAGA </td>
            <td WIDTH="294" ALIGN="center">c.51_52insGAGA </td>
          </tr>
        </table>
      </li>
      <li>insertion of a TG dinucleotide in the sequence ATGTTGTGCC to ATGTTGTG<u><b>TG</b></u>CC;
        is a duplication (<a HREF="#dup"><i>see above</i></a>)</li>
      <li>variability in short sequence repeat<b> </b>(<i><a href="recs-DNA.html#var">see
        Recommendations</a></i>)</li>
    </ul>
  </li>
  <li><b>variability in short sequence repeat - </b><i><a href="#var">see below</a></i> </li>
  <li><b>large insertion</b>&nbsp; -&nbsp; insertion of a 345 nucleotide sequence in intron 3
    from AB012345.1; note that to be able to describe it the sequence of the insertion should
    be submitted to a sequence database (Genbank, EMBL, DDJB) and the accession.version number
    should be given. <table WIDTH="609">
      <tr>
        <th WIDTH="290" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="290" ALIGN="center">g.777_778insAB012345.1</td>
        <td WIDTH="305" ALIGN="center">c.123+54_123+55insAB012345.1</td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<h3><a NAME="var"></a>Sequence repeat variability</h3>

<hr>

<p>For the recommendations how to describe sequence repeat variability <i><a
href="recs-DNA.html#var">see Recommendations</a></i> 

<ul>
  <li><b>polymorphic CA-repeat&nbsp;</b> -&nbsp; the 3' UTR of the gene contains a (CA)<sub>7</sub>-stretch
    from nucleotides 1700 to 1713 (coding DNA *70 to *83) which in the population varies in
    length between 6 and 13 CA's <table WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.1700_1701(6_13)<br>
        or g.1700CA(6_13)<br>
        <i><b>NOT</b></i> g.1712_1713(6_13)</td>
        <td WIDTH="305" ALIGN="center">c.*70_*71(6_13)<br>
        or c.*70CA(6_13)<br>
        <i><b>NOT</b></i> c.*82_*83(6_13) </td>
      </tr>
    </table>
  </li>
  <li><b>polymorphic CA-repeat</b>&nbsp; -&nbsp; a person carries a CA di-nucleotide repeat of
    length 6 on one allele (the reference sequence has a repeat length of 7) <table
    WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.1700_1701[6]<br>
        or g.1700CA[6]<br>
        <i><b>NOT</b></i> g.1712_1713del </td>
        <td WIDTH="305" ALIGN="center">c.*70_71[6]<br>
        or c.*70CA[6]<br>
        <i><b>NOT</b></i> c.*82_83del </td>
      </tr>
    </table>
  </li>
  <li><b>polymorphic CA-repeat&nbsp;</b> -&nbsp; a person carries a CA di-nucleotide repeat of
    length 8 on one allele (the reference sequence has a repeat length of 7) <table
    WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.1700_1701[8]<br>
        or g.1700CA[8]<br>
        <i><b>NOT</b></i> g.1712_1713dup </td>
        <td WIDTH="305" ALIGN="center">c.*70_71[8]<br>
        or c.*70CA[8]<br>
        <i><b>NOT</b></i> c.*82_83dup </td>
      </tr>
    </table>
  </li>
  <li><b>polymorphic CA-repeat</b>&nbsp; -&nbsp; a person carries a CA di-nucleotide repeat of
    length 6 on one allele and of length 11 on the other allele <table WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.1700_1701[6];[11] <br>
        or g.1700CA[6];[11]</td>
        <td WIDTH="305" ALIGN="center">c.*70_71[6];[11] <br>
        or c.*70CA[6];[11]</td>
      </tr>
    </table>
  </li>
  <li><b>FMR1 GGC-repeat</b> - in literature the Fragile-X tri-nucleotide repeat is known as
    the CGG-repeat, but based on the coding DNA reference sequence (GenBank <a
    href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002024.4">NM_002024.4</a>) and the rule
    that for all descriptions the most 3' position possible should be arbitrarily assigned the
    repeat has to be described as a GGC-repeat.<ul>
      <li>c.-128_-126[79] (<em>alternatively c.-128GGC[79]</em>) describes the presence of an
        extended GGC-repeat of <i><b>exactly 79 units</b></i></li>
      <li>c.-128_-126(1000) (<em>alternatively c.-128GGC(1000)</em>) describes the presence of an
        extended GGC-repeat of <i><b>about 1000 units</b></i><br>
        <b><i>NOTE:</i></b> &quot;()&quot; is used to indicate uncertainties (<a
        href="uncertain.html"><i>see Uncertainties</i></a>). </li>
    </ul>
  </li>
  <li><b>HD AGC-repeat</b> - based on the HTT (huntingtin) coding DNA reference sequence
    (GenBank <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002111.6">NM_002111.6</a>) the
    Huntington's Disease tri-nucleotide repeat has to be described as an AGC (not CAG) repeat.
    The reference sequence represents an allele of 21 AGC repeats, described as c.53AGC[21].
    On protein level the reference allele contains 23 Gln's, described as p.Gln[23] (<em>alternatively
    p.Q[23]</em>). The difference derives from the fact that the AGC repeat is interrupted by
    a AAC-repeat (&quot;CAA&quot;) at position 22.</li>
</ul>

<hr>

<h3><a NAME="inv"></a>Inversion</h3>

<hr>

<p>Inversions are designated by <i><b>&quot;inv&quot;</b></i> after the nt number of the
nucleotides inverted. 

<ul>
  <li><b>short inversion&nbsp; -&nbsp; </b>inversion of nucleotides 177 to 180 of the coding
    region, changing -CTAG- to -GATC-<table WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.1077_1080inv&nbsp;<br>
        (g.1077_1080invCTAG) </td>
        <td WIDTH="305" ALIGN="center">c.77_80inv&nbsp;<br>
        (c.77_80invCTAG) </td>
      </tr>
    </table>
  </li>
  <li><b>large inversion</b>&nbsp; -&nbsp; a large inversion (212,434 nucleotides in length),
    starting in intron 4, inverts the entire 3' end of the gene and fuses it to position
    233+17 (intron) of the XYZ-gene, having an opposite transcriptional orientation (<i>indicated
    by the &quot;o&quot;</i>) <table WIDTH="609">
      <tr>
        <th WIDTH="290" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="290" ALIGN="center">g.1458_oXYZ:457inv</td>
        <td WIDTH="305" ALIGN="center">c.301-143_oXYZ:233+17inv</td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<h3><a NAME="con"></a>Gene conversion</h3>

<hr>

<p>Gene conversions are designated by <b><i>&quot;con&quot;</i></b> after the nt number of
the nucleotides converted, followed by a description of the origin on the new sequence; <i><b>&quot;region_changed&quot;
con &quot;region of origin&quot;</b> (<a HREF="disc.html#con">see Discussion</a>).</i> 

<ul>
  <li>a gene conversion replacing a segment of the coding region of a gene for a segment
    derived from elsewhere in the genome (as described in another GenBank file; AC096506.5 and
    NM_004006.1 resp.) <table WIDTH="609">
      <tr>
        <th WIDTH="290" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="290" ALIGN="center">g.415_1655conAC096506.5:g.409_1683</td>
        <td WIDTH="305" ALIGN="center">c.15_355conNM_004006.1:c.15_355 </td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<h3><a NAME="trans"></a>Translocation</h3>

<hr>

<p>Translocations are designated in the format&nbsp; <i><b>&quot;t(X;4)(p21.2;q34)&quot;</b></i>,
followed by the usual description, placed <i><b>between brackets</b></i>, indicating the
exact translocation breakpoint. The sequences of the translocation breakpoints need to be
submitted to a sequence database (Genbank, EMBL, DDJB) and the accession numbers should be
given <i>(<a HREF="disc.html#translocation">see Discussion</a>).</i> 

<ul>
  <li>a translocation breakpoint in the 3' half of intron 4, between nucleotides 1453 and 1454
    (coding DNA 301-148 and 301-147), joining chromosome bands Xp21.2 and 4q35 <table
    WIDTH="609">
      <tr>
        <th WIDTH="290" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="290" ALIGN="center">t(X;4)(p21.2;q35)(g.1453_1454) </td>
        <td WIDTH="305" ALIGN="center">t(X;4)(p21.2;q35)(c.301-148_301-147) &nbsp;
        [t(X;4)(p21.2;q35)(c.IVS4)] </td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<h3><b><u><a NAME="complex"></a></u></b>Complex</h3>

<hr>

<p>Complex rearrangements are rearrangements which consist of several different types of
the <i><b>six elementary content changes</b> substitution, deletion, duplication,
insertion, inversion and translocation</i>. Such rearrangements can be very complex and
difficult to describe. Specific recommendations to describe such changes have not made.
Complex rearrangements can be best described as a combination of the elementary
changes.&nbsp;</p>

<p><b><u><a NAME="insdel"></a>Deletion / insertions</u></b> (<i>&quot;indels&quot;</i>)
are described as a deletion (<i><b>&quot;del&quot;</b></i>), followed by an insertion (<i><b>&quot;ins&quot;</b></i>)
after a description of the deleted segment, i.e. the first (and last) nucleotide(s)
deleted<em> </em><i>(<a HREF="disc.html#indel">see Discussion</a>).</i> 

<ul>
  <li>deletions of nucleotides 712 to 717 of the genomic region (coding DNA 112 to 117) and an
    TG-insertion at the same site <table WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.712_717delinsTG<br>
        (g.712_717delAGGGCAinsTG) </td>
        <td WIDTH="305" ALIGN="center">c.112_117delinsTG<br>
        (c.112_117delAGGGCAinsTG) </td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<h3><a NAME="misc"></a>Miscellaneous</h3>

<hr>

<ul>
  <li>&nbsp;<b><i>Two variants in one allele</i></b> are described as &quot;[first change;
    second change]&quot; (<i><a HREF="disc.html#2in1">see Discussion</a></i>) <ul>
      <li>one allele (chromosome) containing an C to T change at nt 476 (coding DNA nt 76) and a G
        to C change at nt 483 (coding DNA 83) <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[476C&gt;T; 483G&gt;C]&nbsp; </td>
            <td WIDTH="305" ALIGN="center">c.[76C&gt;T; 83G&gt;C]&nbsp; </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><b><i>Two variants in one individual with alleles unknown</i></b> are described as
    &quot;[first change (;) second change]&quot; (<i><a HREF="disc.html#2unknwon">see
    Discussion</a></i>) <ul>
      <li>one individual containing an C to T change at nt 476 (coding DNA nt 76) and a G to C
        change at nt 1083 (coding DNA 183) while it is unknown whether these changes are on the
        same or different alleles <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[476C&gt;T(;)1083G&gt;C]&nbsp; </td>
            <td WIDTH="305" ALIGN="center">c.[76C&gt;T(;)183G&gt;C]&nbsp; </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><b><i>Recessive disease - (changes in different alleles) </i></b>are described as
    &quot;[change allele 1];[change allele 2]&quot; (<i><a HREF="disc.html#2in2">see
    Discussion</a></i>) <ul>
      <li><i><u>both variants identified</u></i>&nbsp; -&nbsp; a homozygous C to T change at nt 76
        of the coding region <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[476C&gt;T];[476C&gt;T] </td>
            <td WIDTH="305" ALIGN="center">c.[76C&gt;T];[76C&gt;T] </td>
          </tr>
        </table>
      </li>
      <li><i><u>one variant not yet identified</u></i>&nbsp; -&nbsp; one allele containing a C to
        T change at nt 76 of the coding region, the other allele containing an unknown change <table
        WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[476C&gt;T];[?] </td>
            <td WIDTH="305" ALIGN="center">c.[76C&gt;T];[?] </td>
          </tr>
        </table>
      </li>
      <li><img border="0" src="new.gif" width="31" height="12"><i><u>probably homozygous variant</u></i>&nbsp;
        -&nbsp; a homozygous C to T change at nt 76 of the coding region, not confirmed by
        analysis of both parents leaving the possibility of non-amplification of the second allele
        due to a primer mismatch or a deletion <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[476C&gt;T];[(476C&gt;T)] </td>
            <td WIDTH="305" ALIGN="center">c.[76C&gt;T];[(76C&gt;T)] </td>
          </tr>
        </table>
      </li>
      <li><i><u>one variant allele and one normal allele</u></i>&nbsp; -&nbsp; one allele
        containing a C to T change at nt 76 of the coding region, the other allele having the
        normal sequence (wild type) <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[476C&gt;T];[=] </td>
            <td WIDTH="305" ALIGN="center">c.[76C&gt;T];[=] </td>
          </tr>
        </table>
      </li>
      <li><i><u>several variants in both alleles</u></i>&nbsp; -&nbsp; two alleles containing a C
        to G change at nt -5, one allele containing a C to T change at nt 76, and two alleles with
        a G to C change at nt 183 all in relation to the coding region <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.[266C&gt;G;476C&gt;T; 1083G&gt;C];<br>
            g.[266C&gt;G;=; 1083G&gt;C] </td>
            <td WIDTH="305" ALIGN="center">c.[-5C&gt;G; 76C&gt;T; 183G&gt;C];<br>
            c.[-5C&gt;G; =; 183G&gt;C] </td>
          </tr>
        </table>
      </li>
      <li><i><u>two variants in different genes</u></i>&nbsp; -&nbsp; the GJB2 gene contains a
        deletion of a G, the GJB6 gene a T insertion <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center"></td>
            <td WIDTH="305" ALIGN="center">NM_004004.2:c.[35delG] ; NM_006783.1:c.[689T&gt;C]<br>
            <br>
            (GJB2:c.[35delG]; GJB6:c.[689T&gt;C]) </td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><img border="0" src="new.gif" width="31" height="12"><em><strong>Mosaicism - two
    different nucleotides in one position caused by somatic mosaicims</strong></em><i> are
    described as &quot;[=/nucleotide 2]&quot; (<a HREF="FAQ.html#mosaic">see FAQ</a>)</i><ul>
      <li>in a tumor some cells contain at position 476 (coding DNA nt 76) the reference sequence
        while other cells contain a C to T change <table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.=/476C&gt;T</td>
            <td WIDTH="305" ALIGN="center">c.=/76C&gt;T</td>
          </tr>
        </table>
      </li>
      <li>in a tumor at position 476 (coding DNA nt 76) both the reference sequence and a C to T
        change are found but it is not known whether the is variant is somatic or germline
        (inherited)<table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.(=/)476C&gt;T</td>
            <td WIDTH="305" ALIGN="center">c.(=/)76C&gt;T</td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li><img border="0" src="new.gif" width="31" height="12"><em><strong>Chimerism - two
    different nucleotides in one position caused by chimerism</strong></em><i> are described
    as &quot;[=//nucleotide 2]&quot;</i><ul>
      <li>a chimeric individual some cells contain at position 476 (coding DNA nt 76) the
        reference sequence while other cells contain a C to T change<table WIDTH="611">
          <tr>
            <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
            <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
          </tr>
          <tr>
            <td WIDTH="292" ALIGN="center">g.=//476C&gt;T</td>
            <td WIDTH="305" ALIGN="center">c.=//76C&gt;T</td>
          </tr>
        </table>
      </li>
    </ul>
  </li>
  <li>one allele containing a TG di-nucleotide repeat of length 4, the other allele containing
    a repeat of length 5 (<i><a href="recs-DNA.html#var">see Recommendation</a></i>) <table
    WIDTH="611">
      <tr>
        <th WIDTH="292" ALIGN="center"><b><i>genomic </i>Reference Sequence</b></th>
        <th WIDTH="305" ALIGN="center"><b><i>coding DNA</i> Reference Sequence</b></th>
      </tr>
      <tr>
        <td WIDTH="292" ALIGN="center">g.983TG[4];[5] </td>
        <td WIDTH="305" ALIGN="center">c.88+495TG[4];[5] </td>
      </tr>
    </table>
  </li>
</ul>

<hr>

<p align="center">| <a HREF="#top">Top of page</a> | <a HREF="index.html">MutNomen
homepage</a> | <a href="checklist.html">Check-list</a> |<br>
| <b>Recommendations:</b>&nbsp; <a href="recs-DNA.html">DNA</a>,&nbsp; <a
href="recs-RNA.html">RNA</a>,&nbsp; <a href="recs-prot.html">protein</a>, <a
href="uncertain.html">uncertain</a> |<br>
| <a HREF="disc.html">Discussions</a> | <a HREF="FAQ.html">FAQ's</a> | <a
href="standards.html">Symbols, codons, etc.</a> | <a href="history.html">History</a> |<br>
| <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef / symbols</a>,&nbsp;
<a HREF="examplesRNA.html">RNA</a>,&nbsp; <a HREF="examplesAA.html">protein</a> | </p>

<table width="100%" border="0" cellspacing="0" cellpadding="15" bgcolor="#ff9900">
  <tr>
    <td><p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"
    size="-5"><a href="http://www.HGVS.org/copyright.html">Copyright </a> </font><font
    size="1">HGVS 2007 All Rights Reserved<br>
    Website Created by Rania Horaitis, Nomenclature by J.T. Den Dunnen - <a
    href="http://www.HGVS.org/disclaimer.html">Disclaimer</a> </font></td>
  </tr>
</table>
</body>
</html>
