<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	17-11-02	URL: 	http://www.dmd.nl/mutnomen_current.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>HGVS recommendations: examples DNA</title>    <style type="text/css"><!--  h1 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body bgcolor="#FFFFFF" link="#0000FF" text="#000000" vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Description of sequence changes:&nbsp;<br>              examples DNA-level</h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified May 20, 2015</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i>Since references to WWW-sites are not yet acknowledged        as citations, please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den
          Dunnen JT and Antonarakis SE (2000). Hum.Mutat. 15: 7-12</a> when        referring to these pages.</i></p>    <hr>    <h2>Contents</h2>    <ul>      <li><a href="#intro">Introduction</a>        <ul>          <li>Reference Sequence</li>          <li>numbering</li>        </ul>      </li>      <li><b>Examples&nbsp; -&nbsp; DNA changes</b>        <ul>          <li><a href="#general">general</a> </li>          <li>seven elementary changes            <ul>              <li><a href="#sub">substitution</a></li>              <li><a href="#del">deletion</a>&nbsp; <i>(incl. in single                  nucleotide stretches)</i> </li>              <li><a href="#dup">duplication</a></li>              <li><a href="#ins">insertion</a></li>              <li><a href="#var">sequence repeat variability</a>&nbsp; <i>(CA,                  CAG, ...)</i> </li>              <li><a href="#inv">inversion</a> </li>              <li><a href="#con">gene conversion</a></li>              <li><a href="#trans">translocation</a> </li>              <li>transposition</li>            </ul>          </li>          <li><a href="#complex">complex rearrangements</a>            <ul>              <li>deletion/insertion <i>(indel)</i> </li>            </ul>          </li>          <li><a href="#misc">miscellaneous</a>            <ul>              <li>two changes in a gene on one chromosome </li>              <li>two changes in one individual, chromosome unknown</li>              <li>recessive diseases - changes in different chromosomes (known                and unknown)                <ul>                  <li><img src="new.gif" border="0" height="12" width="31">probably
                    homozygous</li>                  <li>several changes in two chromosomes</li>                  <li>changes in different genes</li>                </ul>              </li>            </ul>          </li>          <li><a href="uncertain.html">uncertainties</a>&nbsp; <i>(exact              position not known; Southern blot, PCR, arrayCGH, SNP-array, ...)</i></li>        </ul>      </li>      <li><a href="examplesRNA.html"><b>Examples&nbsp; -&nbsp; RNA changes</b></a></li>      <li><b><a href="examplesAA.html">Examples&nbsp; -&nbsp; protein changes</a></b></li>    </ul>    <hr>    <h2><a name="intro"></a>Introduction</h2>    <p>Within this page examples will be given for the description of sequence      variations. The examples will be given independently for descriptions at      DNA, RNA and protein level. All examples are described <i><b>relative to          a Reference Sequence</b></i>, depending on the level a genomic or      coding DNA sequence (DNA-level), an mRNA sequence (RNA-level) or an amino      acid sequence (protein level).&nbsp;</p>    <hr>    <h3><a name="RefSeq"></a>Reference sequence DNA-level</h3>    <hr>    <p>Within this page examples will be given for the description of sequence      variations in a DNA sequence. For other examples go to those describing <a        href="examplesRNA.html">changes
in
        RNA</a>. Examples for protein level are given at <a href="examplesAA.html">the
        protein page</a>. All examples are described <i><b>relative to a          Reference Sequence</b></i>, here a coding DNA sequence.</p>    <table align="center" border="1" width="688">      <tbody>        <tr>          <th colspan="2" width="224"><b>Part of gene</b></th>          <th align="center" width="134">nucleotide numbering<br>            <i><b>genomic</b><br>            </i>Reference Sequence&nbsp;</th>          <th align="center" width="163">nucleotide numbering<br>            <i><b>coding DNA</b></i><br>            Reference Sequence</th>          <th align="center" width="139">&nbsp;nucleotide numbering<br>            <i><b>protein</b></i><br>            Reference Sequence</th>        </tr>        <tr>          <td colspan="2" width="224">5' gene flanking region</td>          <td align="center" width="134">1 to 270</td>          <td align="center" width="163">(-300 to -31)</td>          <td align="center" width="139">&nbsp;-</td>        </tr>        <tr>          <td rowspan="2" width="70"><b>exon 1</b></td>          <td width="148">5' UTR</td>          <td align="center" width="134">271 to 300</td>          <td align="center" width="163">-30 to -1</td>          <td align="center" width="139">&nbsp;-</td>        </tr>        <tr>          <td width="148"><b>coding region</b></td>          <td align="center" width="134">301 to 312</td>          <td align="center" width="163">1 to 12</td>          <td align="center" width="139">1 to 4</td>        </tr>        <tr>          <td colspan="2" width="224">intron 1</td>          <td align="center" width="134">313 to 412</td>          <td align="center" width="163">            <p align="center"><i>12+1 ... 12+50,<br>                13-50 ... 13-1</i> </p>          </td>          <td align="center" width="139">-</td>        </tr>        <tr>          <td colspan="2" width="224"><b>exon 2</b></td>          <td align="center" width="134">413 to 488</td>          <td align="center" width="163">13 to 88</td>          <td align="center" width="139">5 to 29 (30)</td>        </tr>        <tr>          <td colspan="2" width="224">intron 2</td>          <td align="center" width="134">489 to 688</td>          <td align="center" width="163"><i>88+1 ... 88+100,<br>              89-100 ... 89-1&nbsp;</i></td>          <td align="center" width="139">-</td>        </tr>        <tr>          <td colspan="2" width="224"><b>exon 3</b></td>          <td align="center" width="134">689 to 723</td>          <td align="center" width="163">89 to 123</td>          <td align="center" width="139">30 to 41</td>        </tr>        <tr>          <td width="70">intron 3</td>          <td width="148">contains rare alternatively spliced exon from 800 to            859 (coding DNA 123+77 to 123+136)</td>          <td align="center" width="134">724 to 1023</td>          <td align="center" width="163"><i>123+1 ... 123+150,<br>              124-150 ... 124-1&nbsp;</i></td>          <td align="center" width="139">-</td>        </tr>        <tr>          <td colspan="2" width="224"><b>exon 4</b></td>          <td align="center" width="134">1024 to 1200</td>          <td align="center" width="163">124 to 300</td>          <td align="center" width="139">42 to 100</td>        </tr>        <tr>          <td colspan="2" width="224">intron 4</td>          <td align="center" width="134">1201 to 1600</td>          <td align="center" width="163"><i>300+1 ... 300+200,<br>              301-200 ... 301-1&nbsp;</i></td>          <td align="center" width="139">-</td>        </tr>        <tr>          <td rowspan="2" width="70"><b>exon 5</b></td>          <td width="148"><b>coding region</b></td>          <td align="center" width="134">1601 to 1630</td>          <td align="center" width="163">301 to 330</td>          <td align="center" width="139">101 to 109</td>        </tr>        <tr>          <td width="148">3' UTR, containing a (CA)<sub>7</sub>-stretch from            nucleotides 1700 to 1713 (coding DNA *70 to *83); poly-A addition            site at 1825 (coding DNA *195) </td>          <td align="center" width="134">1631 to 1850</td>          <td align="center" width="163">*1 to *220</td>          <td align="center" width="139">-</td>        </tr>        <tr>          <td colspan="2" width="224">3' gene flanking region</td>          <td align="center" width="134">1851 to 2000</td>          <td align="center" width="163">(*221 to *370)</td>          <td align="center" width="139">-&nbsp;</td>        </tr>      </tbody>    </table>    <p><i><b>NOTE:</b> nucleotides in introns in the 5' UTR are numbered like        -23+1, -23+2, ..., -22-2, -22-1. Nucleotides in introns in the 3' UTR        are numbered like *154+1, *154+2, ..., *155-2, *155-1.&nbsp;</i> </p>    <p><b><big>Legend:</big></b><br>      Reference sequence of imaginary gene used for the exaples given on this      page. Nucleotide +1 in the coding DNA reference sequence is the A of the      ATG translation initiation codon. Abbreviations used: nt = nucleotide, UTR      = untranslated region of the mRNA. For a picture of part of this      hypothetical sequence <i><a href="refseq_figure.html">see Figure</a></i>.</p>    <hr>    <h3><a name="general"></a>General</h3>    <hr>    <p>Publications reporting changes in different sequences (genes) or which      report linkage or association studies should prevent any confusion      regarding which variant resides in which sequence. An easy way to achieve      this is to include an unequivocal identifier to the reference sequence      used in the description, e.g. <i><b>NM_004006.2:c.3G&gt;T</b></i> or <em><strong>DMD:c.3G&gt;T</strong></em>      (<i><a href="disc.html#SNP">see Discussion</a></i>).</p>    <hr>    <h3><a name="sub"></a>Substitutions</h3>    <hr>    <p>Substitutions are designated by a <i><b>"&gt;"</b></i>-character after      the number of the affected nucleotide. </p>    <ul>      <li><b>5' gene flanking region&nbsp; -&nbsp; </b>T to C substitution of        nt 241 (located 30 nucleotides upstream of the transcription initiation        site, i.e. in the promoter region)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.241T&gt;C </td>              <td align="center" width="300">- </td>            </tr>          </tbody>        </table>      </li>      <li>&nbsp;<b>5' UTR&nbsp; -&nbsp; </b>G to A substitution of nt 289, 12        nucleotides upstream of the ATG translation initiation codon (coding DNA        -12). For nucleotide numbering in a case where the ATG is not in exon 1        <a href="refseq.html#IVSin5"><i>see here</i></a>.&nbsp;        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.289G&gt;A </td>              <td align="center" width="300">c.-12G&gt;A </td>            </tr>          </tbody>        </table>      </li>      <li><b>translation initiation codon&nbsp; -&nbsp; </b>A to C substitution        of nt 301, i.e. nt 1 of the coding region (coding DNA 1)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.301A&gt;C </td>              <td align="center" width="300">c.1A&gt;C </td>            </tr>          </tbody>        </table>      </li>      <li><b>coding region&nbsp; -&nbsp; </b>G to C substitution of nt 423 in        exon 2, i.e. nt 23 of the coding region (coding DNA 23)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.423G&gt;C </td>              <td align="center" width="300">c.23G&gt;C </td>            </tr>          </tbody>        </table>      </li>      <li><b>intron&nbsp;</b>(regarding the numbering of intronic nucleotides <a          href="disc.html#IVS"><i>see
            Discussion</i></a>)        <ul>          <li><i><u>5' part intron</u>&nbsp; - </i>T to G substitution of the            second nt in the intron (88+2) positioned between coding DNA            nucleotides 88 and 89 (intron 2)            <table width="612">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="297"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.490T&gt;G </td>                  <td align="center" width="297">c.88+2T&gt;G </td>                </tr>              </tbody>            </table>          </li>          <li><i><u>3' part intron</u>&nbsp; - </i>G to T substitution of the            last nt of the intron&nbsp; (89-1) positioned between coding DNA            nucleotides 88 and 89 (intron 2)            <table width="611">              <tbody>                <tr>                  <th align="center" width="302"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="295"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="302">g.688G&gt;T </td>                  <td align="center" width="295">c.89-1G&gt;T </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><b>alternatively spliced exon&nbsp; -&nbsp; </b>G to C substitution        of intronic nt 812 (coding DNA 123+89)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.812C&gt;T </td>              <td align="center" width="300">c.123+89C&gt;T </td>            </tr>          </tbody>        </table>      </li>      <li><b>translation termination codon&nbsp; -&nbsp; </b>G to C        substitution of nt 1629, i.e. nt 329 of the coding region (coding DNA        329)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.1629G&gt;C </td>              <td align="center" width="300">c.329G&gt;C </td>            </tr>          </tbody>        </table>      </li>      <li><b>3' UTR&nbsp; -&nbsp; </b>T to A substitution of nt 1700 (coding        DNA 70), located in the 3' UTR (70 nucleotides downstream of the        termination codon)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.1700T&gt;A </td>              <td align="center" width="300">c.*70T&gt;A </td>            </tr>          </tbody>        </table>      </li>      <li><b>3' gene flanking region&nbsp; -&nbsp; </b>C to A substitution of        nt 1923 (located 293 nucleotides downstream of the gene, i.e. the        polyA-addition site)        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.1923C&gt;A </td>              <td align="center" width="300">c.*293C&gt;A</td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <h3><a name="del"></a>Deletion</h3>    <hr>    <p>Deletions are designated by <i><b>"del"</b></i> after a description of      the deleted segment, i.e. the first (and last) nucleotide(s) deleted (see      also <i><a href="disc.html#largedel">Discussion</a></i>). To describe      deletions with unknown breakpoints, e.g. based on Southern blotting, PCR,      arrayCGH, SNP array data, etc. <i><a href="uncertain.html">see          Uncertainties</a></i>. </p>    <ul>      <li><b>single nucleotide deletion&nbsp; -&nbsp; </b>deletion of nt 13 of        the coding region        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.413del&nbsp;&nbsp; (g.413delG) </td>              <td align="center" width="300">c.13del&nbsp;&nbsp; (c.13delG) </td>            </tr>            <tr>              <td align="center" width="340">g.304del&nbsp;&nbsp; (g.304delG)<br>                <i>(not g.303del / g.303delG)</i> </td>              <td align="center" width="300">c.4del&nbsp;&nbsp; (c.4delG)<br>                <i>(not c.3del / c.3delG)</i> </td>            </tr>            <tr>              <td align="center" width="340">g.1598delG<br>                <i>(not g.1596del / g.1596delG)</i> </td>              <td align="center" width="300">c.301-3del&nbsp;&nbsp;                (c.301-3delT)<br>                <i>(not c.301-5del or&nbsp; c.301-5delT)</i> </td>            </tr>          </tbody>        </table>      </li>      <li>&nbsp;<b>deletion of more then 1 nucleotide&nbsp;</b>        <ul>          <li>deletion of nucleotides -11 to -4 in the 5' UTR            <table width="612">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="297"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.290_297del</td>                  <td align="center" width="297">c.-11_-4del&nbsp;&nbsp;&nbsp;</td>                </tr>              </tbody>            </table>          </li>          <li>deletion of nucleotides 92 to 94 (GAC) of the coding region            <table width="612">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="297"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.692_694del<br>                    (g.692_694delGAC)&nbsp; </td>                  <td align="center" width="297">c.92_94del&nbsp;&nbsp;&nbsp;<br>                    (c.92_94delGAC)&nbsp; </td>                </tr>              </tbody>            </table>          </li>          <li>deletion of nucleotides *8 to *21 in the 3' UTR            <table width="612">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="297"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.1638_1651del</td>                  <td align="center" width="297">c.*8_*21del&nbsp;&nbsp;&nbsp;</td>                </tr>              </tbody>            </table>            <p>deletion across the exon 3 / intron 3 border, nucleotides 120 to              123 of the coding region (exon 3) and the first 48 nucleotides of              intron 3 (nucleotides 123+1 to 123+48) </p>            <table width="611">              <tbody>                <tr>                  <th align="center" width="302"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="295"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="302">g.720_771del</td>                  <td align="center" width="295">c.120_123+48del</td>                </tr>              </tbody>            </table>          </li>          <li>deletion across the intron 3 / exon 4 border, the last 12            nucleotides of intron 3 (nucleotides 124-12 to 124-1) and            nucleotides 124 to 129 of the coding region (exon 4)&nbsp;            <table width="611">              <tbody>                <tr>                  <th align="center" width="302"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="295"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="302">g.1012_1029del</td>                  <td align="center" width="295">c.124-12_129del </td>                </tr>              </tbody>            </table>          </li>          <li>deletion of a TG dinucleotide in the sequence ATGTTG<u><strong>TG</strong></u>CC
            to ATGTTG<b>_</b>CC            <table width="611">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="296"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.307_308del&nbsp;<br>                    (g.307_308delTG)<br>                    <b><i>NOT</i></b> g.305_306del </td>                  <td align="center" width="296">c.7_8del&nbsp;<br>                    &nbsp;(c.7_8delTG)<br>                    <i><b>NOT</b></i> c.5_6del)&nbsp; </td>                </tr>              </tbody>            </table>          </li>          <li>deletion of an A nucleotide in the sequence CA<b><u>A</u></b>gt...            / ..agAAG to CAgt... / ..agAAG            <table width="611">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="296"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.723del<br>                    (g.723delA) </td>                  <td align="center" width="296">c.123del&nbsp;<br>                    &nbsp;(c.123delA)<br>                    <i><b>NOT</b></i> c.125delA </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><b>variability in short sequence repeat - </b><i><a href="#var">see            below</a></i> </li>      <li><b>(multi) exon deletion</b>        <ul>          <li><u><i>breakpoint not sequenced</i></u>            <ul>              <li>deletion of exons 2 to 4 (e.g. detected on Southern blot, <i><a                    href="disc.html#del?">see
                    Discussion</a>)</i> </li>            </ul>            <table width="608">              <tbody>                <tr>                  <th align="center" width="300"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="294"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="300">- </td>                  <td align="center" width="294">c.13-?_300+?del &nbsp; </td>                </tr>              </tbody>            </table>            <ul>              <li>deletion of the entire gene; coding DNA reference sequence                runs from -30 (cap site) to *220 (polyA-addition site); <i><a href="recs.html#uncertain">see
                    Recommendations</a></i> </li>            </ul>            <table width="608">              <tbody>                <tr>                  <th align="center" width="300"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="294"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="300">- </td>                  <td align="center" width="294">c.(?_-30)_(*220_?)del&nbsp;&nbsp;&nbsp;
                    </td>                </tr>              </tbody>            </table>          </li>          <li><u><i>breakpoints sequenced</i></u> - deletion of exons 2 to 4;            the sequences of the deletion breakpoints need to be submitted to a            sequence database (Genbank, EMBL, DDJB) and the accession number            should be given            <table width="606">              <tbody>                <tr>                  <th align="center" width="299"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="293"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="299">g.390_1458del&nbsp;&nbsp;                    (g.390_1458del1069) </td>                  <td align="center" width="293">c.13-23_301-143del&nbsp;&nbsp;                    (c.13-23_301-143del1069) </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><b>large deletion fusing two genes</b><i>&nbsp; (<a href="disc.html#fusion">see
            Discussion</a>)</i>        <ul>          <li><i><u>genes with identical transcriptional orientation</u></i> - a            large deletion, starting in intron 4, removes the entire 3' end of            the gene and fuses it to position 457 (intron position 233+17) of            the gene XYZ            <table width="606">              <tbody>                <tr>                  <th align="center" width="297"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="295"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="297">g.1458_XYZ:457del </td>                  <td align="center" width="295">c.301-143_XYZ:233+17del </td>                </tr>              </tbody>            </table>          </li>          <li><i><u>genes with opposite transcriptional orientation</u></i> - a            large deletion, starting in intron 4, removes the entire 3' end of            the gene and fuses it to position 457 (intron position 233+17) of            the XYZ-gene, which has an opposite transcriptional orientation (<i>indicated
              by the "o"</i>)            <table width="607">              <tbody>                <tr>                  <th align="center" width="298"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="295"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="298">g.1458_oXYZ:457del </td>                  <td align="center" width="295">c.301-143_oXYZ:233+17del </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>    </ul>    <hr>    <h3><a name="dup"></a>Duplication</h3>    <hr>    <p>Duplications are designated by <i><b>"dup"</b></i> after a description      of the duplicated segment, i.e. the first (and last) nucleotide(s)      duplicated (even when a mono-nucleotide is duplicated, <i><a href="recs-DNA.html#dup">see
          Recommendations</a></i>). To describe duplications with unknown      breakpoints, e.g. based on Southern blotting, PCR, arrayCGH, SNP array      data, etc. see also <i><a href="uncertain.html">Uncertainties</a></i>. </p>    <ul>      <li><b>single nucleotide duplication&nbsp; -&nbsp; </b>duplication of nt        13 of the coding region        <table width="654">          <tbody>            <tr>              <th align="center" width="340"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="300"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="340">g.413dup&nbsp;&nbsp; (g.413dupG) </td>              <td align="center" width="300">c.13dup&nbsp;&nbsp; (c.13dupG) </td>            </tr>          </tbody>        </table>      </li>      <li>&nbsp;<b>several nucleotide duplication&nbsp;</b>        <ul>          <li>duplication of nucleotides 92 and 94 (GAC) of the coding region            <table width="612">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="297"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.692_694dup<br>                    (g.692_694dupGAC)&nbsp; </td>                  <td align="center" width="297">c.92_94dup&nbsp;&nbsp;&nbsp;<br>                    (c.92_94dupGAC)&nbsp; </td>                </tr>              </tbody>            </table>          </li>          <li>duplication across the exon 3 / intron 3 border, nucleotides 120            to 123 of the coding region (exon 3) and the first 48 nucleotides of            intron 3 (nucleotides 123+1 to 123+48)            <table width="611">              <tbody>                <tr>                  <th align="center" width="302"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="295"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="302">g.720_771dup</td>                  <td align="center" width="295">c.120_123+48dup&nbsp; </td>                </tr>              </tbody>            </table>          </li>          <li>duplication of a TG dinucleotide in the sequence ATGTTGTGCC to            ATGTTGTG<b><u>TG</u></b>CC            <table width="611">              <tbody>                <tr>                  <th align="center" width="301"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="296"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="301">g.307_308dup&nbsp;<br>                    (g.307_308dupTG)<br>                    <b><i>NOT</i></b> g.305_306dup </td>                  <td align="center" width="296">c.7_8dup&nbsp;<br>                    (c.7_8dupTG)<br>                    <i><b>NOT</b></i> c.5_6dup)&nbsp; </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><b>variability in short sequence repeat - </b><i><a href="#var">see            below</a></i> </li>      <li><b>(multi) exon duplication<br>        </b>note that a duplication <strong><em>by definition</em></strong> (<em><a            href="standards.html#define">see
            Standards</a></em>) should only be used when the additional copy is        located directly 3'-flanking the original copy (a tandem duplication).        In most cases there will be no experimental proof of this, the        additional copy may reside anywhere in the gene/genome (i.e.        inserted/transposed). It seems thus better to describe exonic        duplications using the format for sequence repeat variability, i.e. <i><b>c.88-?_301+?[2]</b></i>        (<em><a href="recs-DNA.html#var">see Recommendations</a></em> and <em><a            href="#var">below</a></em>).
        <ul>          <li><u><i>breakpoints not sequenced</i></u> - duplication of exons 2            to 4 (e.g. detected on Southern blot, <i><a href="disc.html#del?">see
                Discussion</a>)</i>            <table width="608">              <tbody>                <tr>                  <th align="center" width="300"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="294"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="300">- </td>                  <td align="center" width="294">c.13-?_300+?[2] <br>                    or c.13-?_300+?dup </td>                </tr>              </tbody>            </table>          </li>          <li><u><i>breakpoints not sequenced</i></u> - duplication of the            entire gene; coding DNA reference sequence runs from -30 (cap site)            to *220 (polyA-addition site); <i><a href="recs.html#uncertain">see                Recommendations</a></i>            <table width="608">              <tbody>                <tr>                  <th align="center" width="300"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="294"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="300">- </td>                  <td align="center" width="294">c.(?_-30)_(*220_?)[2] <br>                    or c.(?_-30)_(*220_?)dup </td>                </tr>              </tbody>            </table>          </li>          <li><u><i>breakpoints sequenced</i></u> - duplication of exons 2 to 4            <table width="606">              <tbody>                <tr>                  <th align="center" width="299"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="293"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="299">g.390_1458dup</td>                  <td align="center" width="293">c.13-23_301-143dup</td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>    </ul>    <hr>    <h3><a name="ins"></a>Insertion</h3>    <hr>    <p>Insertions are designated by <i><b>"ins"</b></i> after the nucleotides      flanking the insertion. <i><b>NOTE:</b></i> duplicating insertions (incl.      duplication of a mono-nucleotide) should be described as duplications (<a        href="#dup"><i>see
          above</i></a>). </p>    <ul>      <li><b>single nucleotide insertion&nbsp; -&nbsp; </b>insertion of a T        between nucleotides 51 and 52 of the coding region        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.451_452insT </td>              <td align="center" width="305">c.51_52insT </td>            </tr>          </tbody>        </table>      </li>      <li>&nbsp;<b>several nucleotide insertion</b>        <ul>          <li>insertion of a GAGA-sequence between nucleotides 51 and 52 of the            coding region            <table width="604">              <tbody>                <tr>                  <th align="center" width="296"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="294"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="296">g.451_452insGAGA </td>                  <td align="center" width="294">c.51_52insGAGA </td>                </tr>              </tbody>            </table>          </li>          <li>insertion of a TG dinucleotide in the sequence ATGTTGTGCC to            ATGTTGTG<u><b>TG</b></u>CC; is a duplication (<a href="#dup"><i>see                above</i></a>)</li>          <li>variability in short sequence repeat<b> </b>(<i><a href="recs-DNA.html#var">see
                Recommendations</a></i>)</li>        </ul>      </li>      <li><b>variability in short sequence repeat - </b><i><a href="#var">see            below</a></i> </li>      <li><b>large insertion</b>&nbsp; -&nbsp; insertion of a 345 nucleotide        sequence in intron 3 from AB012345.1; note that to be able to describe        it the sequence of the insertion should be submitted to a sequence        database (Genbank, EMBL, DDJB) and the accession.version number should        be given.        <table width="609">          <tbody>            <tr>              <th align="center" width="290"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="290">g.777_778insAB012345.1</td>              <td align="center" width="305">c.123+54_123+55insAB012345.1</td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <h3><a name="var"></a>Sequence repeat variability</h3>    <hr>    <p>For the recommendations how to describe sequence repeat variability <i><a          href="recs-DNA.html#var">see
          Recommendations</a></i> </p>    <ul>      <li><b>polymorphic CA-repeat&nbsp;</b> -&nbsp; the 3' UTR of the gene        contains a (CA)<sub>7</sub>-stretch from nucleotides 1700 to 1713        (coding DNA *70 to *83) which in the population varies in length between        6 and 13 CA's        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.1700_1701[(6_13)]<br>                or g.1700CA[(6_13)]<br>                <i><b>NOT</b></i> g.1712_1713[(6_13)]</td>              <td align="center" width="305">c.*70_*71[(6_13)]<br>                or c.*70CA[(6_13)]<br>                <i><b>NOT</b></i> c.*82_*83[(6_13)] </td>            </tr>          </tbody>        </table>      </li>      <li><b>polymorphic CA-repeat</b>&nbsp; -&nbsp; a person carries a CA        di-nucleotide repeat of length 6 on one chromosome (the reference        sequence has a repeat length of 7)        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.1700_1701[6]<br>                or g.1700CA[6]<br>                <i><b>NOT</b></i> g.1712_1713del </td>              <td align="center" width="305">c.*70_71[6]<br>                or c.*70CA[6]<br>                <i><b>NOT</b></i> c.*82_83del </td>            </tr>          </tbody>        </table>      </li>      <li><b>polymorphic CA-repeat&nbsp;</b> -&nbsp; a person carries a CA        di-nucleotide repeat of length 8 on one chromosome (the reference        sequence has a repeat length of 7)        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.1700_1701[8]<br>                or g.1700CA[8]<br>                <i><b>NOT</b></i> g.1712_1713dup </td>              <td align="center" width="305">c.*70_71[8]<br>                or c.*70CA[8]<br>                <i><b>NOT</b></i> c.*82_83dup </td>            </tr>          </tbody>        </table>      </li>      <li><b>polymorphic CA-repeat</b>&nbsp; -&nbsp; a person carries a CA        di-nucleotide repeat of length 6 on one chromosome and of length 11 on        the other<br>        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.1700_1701[6];[11] <br>                or g.1700CA[6];[11]</td>              <td align="center" width="305">c.*70_71[6];[11] <br>                or c.*70CA[6];[11]</td>            </tr>          </tbody>        </table>      </li>      <li><b>FMR1 GGC-repeat</b> - in literature the Fragile-X tri-nucleotide        repeat is known as the CGG-repeat, but based on the coding DNA reference        sequence (GenBank <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002024.5">NM_002024.5</a>)        and the rule that for all descriptions the most 3' position possible        should be arbitrarily assigned the repeat has to be described as a        GGC-repeat (<a href="file:///Users/Boomheide/Files/WWW/HGVS/recs-DNA.html#var"><span            style="font-style: italic;">see            Recommendations</span></a>).        <ul>          <li>c.-128_-126[79] describes the presence of an extended repeat of <i><b>exactly                79 units</b></i></li>          <li>c.-128_-126[(600_800)] describes the presence of an extended epeat            with an <span style="font-weight: bold; font-style: italic;">estimated
              length between 600 and 800 copies</span><br>            <b><i>NOTE:</i></b> "()" is used to indicate uncertainties (<a href="uncertain.html"><i>see
                Uncertainties</i></a>), a description like c.-128GGC[(...)] can            not be used since the GGC-repeat is probably interrupted by one or            more GGA-triplets (<a href="recs-DNA.html#var"><span style="font-style: italic;">see                Recommendations</span></a>)</li>        </ul>      </li>      <li><b>HD AGC-repeat</b> - based on the HTT (huntingtin) coding DNA        reference sequence (GenBank <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002111.6">NM_002111.6</a>)        the Huntington's Disease tri-nucleotide repeat has to be described as an        AGC (not CAG) repeat. The reference sequence represents an allele of 21        AGC repeats, described as c.53AGC[21]. On protein level the reference        allele contains 23 Gln's, described as p.Gln[23] (<em>alternatively          p.Q[23]</em>). The difference derives from the fact that the AGC        repeat is interrupted by a AAC-triplet ("CAA" coding) at position 22.</li>    </ul>    <hr>    <h3><a name="inv"></a>Inversion</h3>    <hr>    <p>Inversions are designated by <i><b>"inv"</b></i> after the nt number of      the nucleotides inverted. </p>    <ul>      <li><b>short inversion&nbsp; -&nbsp; </b>inversion of nucleotides 177 to        180 of the coding region, changing -CTGA- to -TCAG-        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.1077_1080inv&nbsp;<br>                (g.1077_1080invCTGA) </td>              <td align="center" width="305">c.77_80inv&nbsp;<br>                (c.77_80invCTGA) </td>            </tr>          </tbody>        </table>      </li>      <li><b>large inversion</b>&nbsp; -&nbsp; a large inversion (212,434        nucleotides in length), starting in intron 4, inverts the entire 3' end        of the gene and fuses it to position 233+17 (intron) of the XYZ-gene,        having an opposite transcriptional orientation (<i>indicated by the "o"</i>)        <table width="609">          <tbody>            <tr>              <th align="center" width="290"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="290">g.1458_oXYZ:457inv</td>              <td align="center" width="305">c.301-143_oXYZ:233+17inv</td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <h3><a name="con"></a>Gene conversion</h3>    <hr>    <p>Gene conversions are designated by <b><i>"con"</i></b> after the nt      number of the nucleotides converted, followed by a description of the      origin on the new sequence; <i><b>"region_changed" con "region of origin"</b>        (<a href="disc.html#con">see Discussion</a>).</i> </p>    <ul>      <li>a gene conversion replacing a segment of the coding region of a gene        for a segment derived from elsewhere in the genome (as described in        another GenBank file; AC096506.5 and NM_004006.1 resp.)        <table width="609">          <tbody>            <tr>              <th align="center" width="290"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="290">g.415_1655conAC096506.5:g.409_1683</td>              <td align="center" width="305">c.15_355conNM_004006.1:c.15_355 </td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <h3><a name="trans"></a>Translocation</h3>    <hr>    <p>Translocations are designated in the format&nbsp; <i><b>"t(X;4)(p21.2;q34)"</b></i>,      followed by the usual description, placed <i><b>between brackets</b></i>,      indicating the exact translocation breakpoint. The sequences of the      translocation breakpoints need to be submitted to a sequence database      (Genbank, EMBL, DDJB) and the accession numbers should be given <i>(<a href="disc.html#translocation">see
          Discussion</a>).</i> </p>    <ul>      <li>a translocation breakpoint in the 3' half of intron 4, between        nucleotides 1453 and 1454 (coding DNA 301-148 and 301-147), joining        chromosome bands Xp21.2 and 4q35        <table width="609">          <tbody>            <tr>              <th align="center" width="290"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="290">t(X;4)(p21.2;q35)(g.1453_1454) </td>              <td align="center" width="305">t(X;4)(p21.2;q35)(c.301-148_301-147)
                &nbsp; [t(X;4)(p21.2;q35)(c.IVS4)] </td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <h3><b><u><a name="complex"></a></u></b>Complex</h3>    <hr>    <p>Complex rearrangements are rearrangements which consist of several      different types of the <i><b>six elementary content changes</b>        substitution, deletion, duplication, insertion, inversion and        translocation</i>. Such rearrangements can be very complex and difficult      to describe. Specific recommendations to describe such changes have not      made. Complex rearrangements can be best described as a combination of the      elementary changes.&nbsp;</p>    <p><b><u><a name="insdel"></a>Deletion / insertions</u></b> (<i>"indels"</i>)      are described as a deletion (<i><b>"del"</b></i>), followed by an      insertion (<i><b>"ins"</b></i>) after a description of the deleted      segment, i.e. the first (and last) nucleotide(s) deleted<em> </em><i>(<a          href="disc.html#indel">see
          Discussion</a>).</i> </p>    <ul>      <li>deletions of nucleotides 712 to 717 of the genomic region (coding DNA        112 to 117) and an TG-insertion at the same site        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.712_717delinsTG<br>                (g.712_717delAGGGCAinsTG) </td>              <td align="center" width="305">c.112_117delinsTG<br>                (c.112_117delAGGGCAinsTG) </td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <h3><a name="misc"></a>Miscellaneous</h3>    <hr>    <ul>      <li>&nbsp;<b><i>Two variants in a gene on one chromosome of an individual</i></b>        are described as "[first change; second change]" (<i><a href="disc.html#2in1">see
            Discussion</a></i>)        <ul>          <li>a gene on one chromosome containing an C to T change at nt 476            (coding DNA nt 76) and a G to C change at nt 483 (coding DNA 83)            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[476C&gt;T; 483G&gt;C]&nbsp;                  </td>                  <td align="center" width="305">c.[76C&gt;T; 83G&gt;C]&nbsp; </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><b><i>Two variants in a gene in one individual with chromosome unknown</i></b>        are described as "[first change (;) second change]" (<i><a href="disc.html#2unknwon">see
            Discussion</a></i>)        <ul>          <li>one individual containing an C to T change at nt 476 (coding DNA            nt 76) and a G to C change at nt 1083 (coding DNA 183) while it is            unknown whether these changes are on the same or different            chromosomes<br>            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[476C&gt;T(;)1083G&gt;C]&nbsp;
                    </td>                  <td align="center" width="305">c.[76C&gt;T(;)183G&gt;C]&nbsp;                  </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><b><i>Recessive disease - changes in a gene on different chromosomes </i></b>are
        described as "[change gene 1];[change gene 2]" (<i><a href="disc.html#2in2">see
            Discussion</a></i>)        <ul>          <li><i><u>both variants identified</u></i>&nbsp; -&nbsp; a homozygous            C to T change at nt 76 of the coding region            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[476C&gt;T];[476C&gt;T] </td>                  <td align="center" width="305">c.[76C&gt;T];[76C&gt;T] </td>                </tr>              </tbody>            </table>          </li>          <li><i><u>one variant not yet identified</u></i>&nbsp; -&nbsp; one            chromosome containing a C to T change at nt 76 of the coding region,            the other chromosome containing an unknown change            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[476C&gt;T];[?] </td>                  <td align="center" width="305">c.[76C&gt;T];[?] </td>                </tr>              </tbody>            </table>          </li>          <li><img src="new.gif" border="0" height="12" width="31"><i><u>probably
                homozygous variant</u></i>&nbsp; -&nbsp; a homozygous C to T            change at nt 76 of the coding region, not confirmed by analysis of            both parents leaving the possibility of non-amplification of the            second chromosome due to a primer mismatch or a deletion            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[476C&gt;T];[(476C&gt;T)] </td>                  <td align="center" width="305">c.[76C&gt;T];[(76C&gt;T)] </td>                </tr>              </tbody>            </table>          </li>          <li><i><u>one variant and a normal chromosome</u></i>&nbsp; -&nbsp;            one chromosome containing a C to T change at nt 76 of the coding            region, the other having a reference sequence (wild type)            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[476C&gt;T];[=] </td>                  <td align="center" width="305">c.[76C&gt;T];[=] </td>                </tr>              </tbody>            </table>          </li>          <li><i><u>several variants in a gene on both chromosomes</u></i>&nbsp;            -&nbsp; a gene containing a C to G change at nt -5 and a G to C            change at nt 183 and one copy containing a C to T change at nt 76            all in relation to the coding region            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.[266C&gt;G;476C&gt;T;                    1083G&gt;C];<br>                    g.[266C&gt;G; 1083G&gt;C] </td>                  <td align="center" width="305">c.[-5C&gt;G; 76C&gt;T;                    183G&gt;C];<br>                    c.[-5C&gt;G; 183G&gt;C] </td>                </tr>              </tbody>            </table>          </li>          <li><i><u>two variants in different genes</u></i>&nbsp; -&nbsp; the            GJB2 gene contains a deletion of a G, the GJB6 gene a T insertion            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292"><br>                  </td>                  <td align="center" width="305">NM_004004.2:c.[35delG]                    NM_006783.1:c.[689T&gt;C]<br>                    (GJB2:c.[35delG] GJB6:c.[689T&gt;C]) </td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><img src="new.gif" border="0" height="12" width="31"><em><strong>Mosaicism
            - two different nucleotides in one position caused by somatic            mosaicims</strong></em><i> are described as "[=/nucleotide 2]" (<a href="FAQ.html#mosaic">see
            FAQ</a>)</i>        <ul>          <li>in a tumor some cells contain at position 476 (coding DNA nt 76)            the reference sequence while other cells contain a C to T change            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.=/476C&gt;T</td>                  <td align="center" width="305">c.=/76C&gt;T</td>                </tr>              </tbody>            </table>          </li>          <li>in a tumor at position 476 (coding DNA nt 76) both the reference            sequence and a C to T change are found but it is not known whether            the is variant is somatic or germline (inherited)            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.(=/)476C&gt;T</td>                  <td align="center" width="305">c.(=/)76C&gt;T</td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li><img src="new.gif" border="0" height="12" width="31"><em><strong>Chimerism
            - two different nucleotides in one position caused by chimerism</strong></em><i>          are described as "[=//nucleotide 2]"</i>        <ul>          <li>a chimeric individual some cells contain at position 476 (coding            DNA nt 76) the reference sequence while other cells contain a C to T            change            <table width="611">              <tbody>                <tr>                  <th align="center" width="292"><b><i>genomic </i>Reference                      Sequence</b></th>                  <th align="center" width="305"><b><i>coding DNA</i> Reference                      Sequence</b></th>                </tr>                <tr>                  <td align="center" width="292">g.=//476C&gt;T</td>                  <td align="center" width="305">c.=//76C&gt;T</td>                </tr>              </tbody>            </table>          </li>        </ul>      </li>      <li>one chromosome containing a TG di-nucleotide repeat of length 4, the        other chromosome containing a repeat of length 5 (<i><a href="recs-DNA.html#var">see
            Recommendation</a></i>)        <table width="611">          <tbody>            <tr>              <th align="center" width="292"><b><i>genomic </i>Reference                  Sequence</b></th>              <th align="center" width="305"><b><i>coding DNA</i> Reference                  Sequence</b></th>            </tr>            <tr>              <td align="center" width="292">g.983TG[4];[5] </td>              <td align="center" width="305">c.88+495TG[4];[5] </td>            </tr>          </tbody>        </table>      </li>    </ul>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">MutNomen
        homepage</a> | <a href="checklist.html">Check-list</a> |<br>      | <b>Recommendations:</b>&nbsp; <a href="recs-DNA.html">DNA</a>,&nbsp; <a        href="recs-RNA.html">RNA</a>,&nbsp;
      <a href="recs-prot.html">protein</a>, <a href="uncertain.html">uncertain</a>      |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="standards.html">Symbols,
        codons, etc.</a> | <a href="history.html">History</a> |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef /        symbols</a>,&nbsp; <a href="examplesRNA.html">RNA</a>,&nbsp; <a href="examplesAA.html">protein</a>      | </p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright
                  </a> </font><font size="1">HGVS 2007 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>