<html>
<!--
	Author: 	Dr. Johan T. den Dunnen
	Date: 	17-11-02
	URL: 	http://www.dmd.nl/mutnomen_current.html
-->

<head>
<title>HGVS recommendations: examples RNA</title>
<style type="text/css">

<!--  

h1 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h2 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #ff9900;

}

h3 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h4 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h5 {
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #ffffff;

}

p { 
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #000000;
margin-left : 5%;

 }

.norm {

 font-weight : normal;

 }
 -->
</style>
</head>

<body text="#000000" bgcolor="#FFFFFF" link="#0000FF" vlink="#9999ff">

<table width="100%" border="0" cellspacing="0" cellpadding="10" bgcolor="#CCFF99">
  <tr>
    <td width="25%" valign="top" bgcolor="#FFFFFF"><a href="http://www.HGVS.org"><img
    src="small.gif" width="155" height="94" border="0"></a></td>
    <td width="75%" valign="CENTER" bgcolor="#FFFFFF" bordercolor="#FFFFFF"><h3 align="center">Description
    of sequence changes:&nbsp;<br>
    examples RNA-level</h3>
    </td>
  </tr>
</table>

<table width="100%" border="0" cellspacing="0" cellpadding="10">
  <tr>
    <td width="100%" valign="BOTTOM" bgcolor="#0099cc"><h4 align="center"><font size="3"
    color="#FFFFFF"><i><br>
    Last modified January 26, 2013</i></font></h4>
    </td>
  </tr>
</table>

<p align="center"><i><font face="Times New Roman">Since references to WWW-sites are not
yet acknowledged as citations, please mention <a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den Dunnen JT
and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when referring to these pages.</font></i></p>

<hr>

<h2>Contents</h2>

<ul>
  <li><a HREF="#intro"><b><font face="Verdana, Arial, Helvetica, sans-serif">Introduction</font></b></a>
    <ul>
      <li><font face="Verdana, Arial, Helvetica, sans-serif">Reference Sequence</font></li>
    </ul>
  </li>
  <li><font face="Verdana, Arial, Helvetica, sans-serif"><b>Examples&nbsp; -&nbsp; RNA changes</b>
    </font><ul>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><a HREF="#general">general</a> </font></li>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><b>six elementary content changes;</b><i>
        substitution, deletion, duplication, insertion, inversion and translocation </i>see <a
        HREF="examplesDNA.html">DNA-level</a> </font></li>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><a HREF="#prod">RNA processing /
        initiation</a>&nbsp; <i>(promoter, transcription initiation, polyA-addition)</i> </font></li>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><a HREF="#splice">splicing</a></font></li>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><a HREF="#more">one change, more RNA
        molecules</a> </font></li>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><a HREF="#del">large deletions</a>&nbsp;
        <i>(promoter region, 3'-terminal exon, fused transcripts)</i> </font></li>
      <li><font face="Verdana, Arial, Helvetica, sans-serif"><a
        href="file:///G:/WWW/HGVS/uncertain.html">uncertainties</a>&nbsp; <i>(exact position not
        known; Southern blot, PCR, arrayCGH, SNP-array, ...)</i></font></li>
    </ul>
  </li>
  <li><font face="Verdana, Arial, Helvetica, sans-serif"><a HREF="examplesDNA.html"><b>Examples&nbsp;
    -&nbsp; DNA changes</b></a></font></li>
  <li><font face="Verdana, Arial, Helvetica, sans-serif"><b><a HREF="examplesAA.html">Examples&nbsp;
    -&nbsp; protein changes</a></b></font></li>
</ul>

<hr>

<h2><a NAME="intro"></a>Introduction</h2>

<hr>

<p>Within this page examples will be given for the description of sequence variations
which are <i><b>unique for RNA</b></i>. For other examples go to those describing <a
HREF="examplesDNA.html">changes in DNA</a>. Examples for protein level are given at <a
HREF="examplesAA.html">the protein page</a>. All examples are described <i><b>relative to
a Reference Sequence</b></i>, here a coding DNA sequence.</p>

<h3><a NAME="RefSeq"></a>Reference sequence RNA-level</h3>

<table BORDER="1" WIDTH="688" align="center">
  <tr>
    <th WIDTH="224" COLSPAN="2"><b>Part of gene</b></th>
    <th WIDTH="134" ALIGN="center">nucleotide numbering<br>
    <i>genomic<br>
    </i>Reference Sequence&nbsp;</th>
    <th WIDTH="163" ALIGN="center">nucleotide numbering<br>
    <i>coding DNA</i><br>
    Reference Sequence</th>
    <th WIDTH="139" ALIGN="center">&nbsp;nucleotide numbering<br>
    <i>protein</i><br>
    Reference Sequence</th>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">5' gene flanking region</td>
    <td WIDTH="134" ALIGN="center">1 to 270</td>
    <td WIDTH="163" ALIGN="center">(-300 to -31)</td>
    <td WIDTH="139" ALIGN="center">&nbsp;-</td>
  </tr>
  <tr>
    <td WIDTH="70" ROWSPAN="2"><b>exon 1</b></td>
    <td WIDTH="148">5' UTR</td>
    <td WIDTH="134" ALIGN="center">271 to 300</td>
    <td WIDTH="163" ALIGN="center">-30 to -1</td>
    <td WIDTH="139" ALIGN="center">&nbsp;-</td>
  </tr>
  <tr>
    <td WIDTH="148"><b>coding region</b></td>
    <td WIDTH="134" ALIGN="center">301 to 312</td>
    <td WIDTH="163" ALIGN="center">1 to 12</td>
    <td WIDTH="139" ALIGN="center">1 to 4</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">intron 1</td>
    <td WIDTH="134" ALIGN="center">313 to 412</td>
    <td WIDTH="163" ALIGN="center"><i>12+1 ... 12+50,<br>
    13-50 ... 13-1</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2"><b>exon 2</b></td>
    <td WIDTH="134" ALIGN="center">413 to 488</td>
    <td WIDTH="163" ALIGN="center">13 to 88</td>
    <td WIDTH="139" ALIGN="center">5 to 29 (30)</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">intron 2</td>
    <td WIDTH="134" ALIGN="center">489 to 689</td>
    <td WIDTH="163" ALIGN="center"><i>88+1 ... 88+100,<br>
    89-100 ... 89-1&nbsp;</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2"><b>exon 3</b></td>
    <td WIDTH="134" ALIGN="center">689 to 723</td>
    <td WIDTH="163" ALIGN="center">89 to 123</td>
    <td WIDTH="139" ALIGN="center">30 to 41</td>
  </tr>
  <tr>
    <td WIDTH="70">intron 3</td>
    <td WIDTH="148">contains rare alternatively spliced exon from 800 to 859 (coding DNA
    123+77 to 123+136)</td>
    <td WIDTH="134" ALIGN="center">724 to 1023</td>
    <td WIDTH="163" ALIGN="center"><i>123+1 ... 123+150,<br>
    124-150 ... 124-1&nbsp;</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2"><b>exon 4</b></td>
    <td WIDTH="134" ALIGN="center">1024 to 1200</td>
    <td WIDTH="163" ALIGN="center">124 to 300</td>
    <td WIDTH="139" ALIGN="center">42 to 100</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">intron 4</td>
    <td WIDTH="134" ALIGN="center">1201 to 1600</td>
    <td WIDTH="163" ALIGN="center"><i>300+1 ... 300+200,<br>
    301-200 ... 301-1&nbsp;</i></td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="70" ROWSPAN="2"><b>exon 5</b></td>
    <td WIDTH="148"><b>coding region</b></td>
    <td WIDTH="134" ALIGN="center">1601 to 1630</td>
    <td WIDTH="163" ALIGN="center">301 to 330</td>
    <td WIDTH="139" ALIGN="center">101 to 109</td>
  </tr>
  <tr>
    <td WIDTH="148">3' UTR, containing a (CA)<sub>7</sub>-stretch from nts 1700 to 1713
    (coding DNA *71 to *83); poly-A addition site at 1825 (coding DNA *195) </td>
    <td ALIGN="center" WIDTH="134">1631 to 1850</td>
    <td WIDTH="163" ALIGN="center">*1 to *220</td>
    <td WIDTH="139" ALIGN="center">-</td>
  </tr>
  <tr>
    <td WIDTH="224" COLSPAN="2">3' gene flanking region</td>
    <td WIDTH="134" ALIGN="center">1851 to 2000</td>
    <td WIDTH="163" ALIGN="center">(*221 to *370)</td>
    <td WIDTH="139" ALIGN="center">-&nbsp;</td>
  </tr>
</table>

<p><b><big>Legend:</big></b><br>
Reference sequence of imaginary gene used for the exaples given on this page. Nucleotide
+1 in the coding DNA reference sequence is the A of the ATG translation initiation codon.
Abbreviations used: nt = nucleotide, nts = nucleotides, UTR = untranslated region of the
mRNA. For a picture of part of this hypothetical sequence <i><a href="refseq_figure.html">see
Figure</a></i>.</p>

<hr>

<h3><a NAME="general"></a>General</h3>

<p>It should be noted that the descriptions at RNA level, like those at protein level, are
often <b><i>deduced</i></b> and not based on experimental evidence. Publications
describing changes at RNA level should make it clear whether experimental proof was
available or not. In fact, changes at RNA level should not be provided unless RNA was
actually analysed, i.e. experimental proof is available. When changes are reported for
which experimental proof is not available one should consider to list them between
brackets. </p>

<p>Sequence changes at RNA level are basically described like those at the <a
HREF="examplesDNA.html">DNA level</a>, with a few modifications; 

<ul>
  <li>an &#147;r.&#148; is used to indicate that a change is described at RNA-level</li>
  <li>description starts with the nucleotide number, followed by the nucleotide (in lower
    case); a (adenine), c (cytosine), g (guanine) and u (uracil)<ul>
      <li>78u&gt;a denotes that at nt 78 of the mRNA a U has been substituted for an A</li>
      <li><i>r.(?)</i>&nbsp;&nbsp;&nbsp; - RNA was not analysed but the effect on RNA is expected
        to be directly copied from the change at DNA level</li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="prod"></a>RNA processing / initiation</h3>

<p>Several changes at DNA level can influence the <i><b>amount of RNA</b></i> transcribed
(e.g. changes altering the promoter sequence) or <i><b>RNA processing</b></i> (e.g.
transcription initiation site, polyA-addition signal, polyA-addition site; for changes
affecting <i><b>RNA splicing</b> <a HREF="#splice">see below</a></i>). Unless experimental
proof is available, the effect on RNA level should be reported as &quot;?&quot; (i.e.
unknown or not analysed).&nbsp; However, for many changes an effect on RNA will be very
likely (e.g. when the respective sites are directly changed). In such cases, although RNA
has not been analysed, one is tempted to list these changes and indicate the deduced
effect. Suggested descriptions are&nbsp;(<a HREF="recs-RNA.html#RNA"><i>see
Recommendations</i></a>); 

<ul>
  <li><b>no effect on the RNA</b> <ul>
      <li><i>r.=&nbsp;</i>&nbsp;&nbsp; - RNA sequence equals the reference sequence (wild type)</li>
    </ul>
  </li>
  <li><b>no RNA produced</b> <ul>
      <li><i>r.0</i>&nbsp;&nbsp;&nbsp; - no RNA can be detected (e.g. due to a change which
        deletes the promoter and transcription start site)</li>
    </ul>
  </li>
  <li><b>unknown effect on RNA</b> <ul>
      <li><i>r.0?&nbsp;&nbsp; - </i>the effect on RNA is unknown but expected to result in no RNA
        (e.g. when a change affects the promoter)</li>
      <li><i>r.?</i>&nbsp;&nbsp;&nbsp;&nbsp; - RNA was not analysed but an effect is expected
        (e.g. when the transcription start site, the polyA-addition signal or the polyA-addition
        site is changed)</li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="splice"></a>Splicing</h3>

<p>changes affecting splicing result on RNA-level either in a deletion (shift of a splice
site to within an exon or skipping of a complete exon) or an insertion (shift of a splice
site to within an intron or inclusion of an intronic sequence as a new exon). Such changes
can be described following the rules to describe deletions and insertions (see <a
HREF="examplesDNA.html">DNA level</a>). However, some additional changes might occur. 

<ul>
  <li><b>unknown effect</b> <ul>
      <li><i>r.spl?&nbsp; </i>- RNA was not analysed but the change is expected to affect
        splicing, e.g. when the splice donor or splice acceptor site is changed</li>
      <li><i>r.(spl?)&nbsp; </i>- RNA was not analysed but the change might affect splicing, e.g.
        changes close to the splice donor or splice acceptor site or in the first or last
        nucleotide of an exon</li>
    </ul>
  </li>
  <li><b>change destroys a splice site</b> (in the examples of exon 3) <ul>
      <li><u>splice acceptor site</u> (at DNA c.89-1G&gt;T) <ul>
          <li><i>r.89_123del</i> - the entire exon is skipped (deleted from the RNA)</li>
          <li><i>r.89_100del</i>&nbsp;-&nbsp;a new (cryptic) splice site in the exon, between coding
            DNA nts 100 and 101 is activated, resulting in the deletion of the beginning of the exon</li>
          <li><i>r.[89-12_89-1ins; 89-1g&gt;u]</i>&nbsp; -&nbsp; a new (cryptic) splice site in the
            intron, between nts 89-13 and 89-12, is activated, resulting in the insertion of the 3'
            end of the intron<i><br>
            <b>NOTE:</b> </i>due to the change a substituted nt is present in the mRNA, requiring the
            use of the rule to describe two changes in one allele (<i><a HREF="recs-DNA.html#2in">see
            Recommendations</a></i>)</li>
        </ul>
      </li>
      <li><u>splice donor site</u> (at DNA c.123+1G&gt;T) <ul>
          <li><i>r.89_123del</i>&nbsp; -&nbsp; skipping the entire exon</li>
          <li><i>r.112_123del</i>&nbsp; -&nbsp; a new (cryptic) splice site in the exon, between
            coding DNA nts 111 and 112 is activated, resulting in the deletion of the 3' end of the
            exon</li>
          <li><i>r.[123+1_123+35ins; 123+1g&gt;u]</i>&nbsp; -&nbsp; a new (cryptic) splice site in the
            intron, between nts 123+35 and 123+36, is activated, resulting in the insertion of the
            beginning of the intron <i><br>
            <b>NOTE:</b> </i>due to the change a substituted nt is present in the mRNA, requiring the
            use of the rule to describe two changes in one allele (<i><a HREF="recs-DNA.html#2in">see
            Recommendations</a></i>)</li>
        </ul>
      </li>
    </ul>
  </li>
  <li><b>change activates intronic exon</b> (in the examples from intron 2) <ul>
      <li>r.88_89ins88+73_89-189&nbsp; -&nbsp; a change in the intron results in the activation of
        an intronic sequence, from nt 88+73 to 89-189, which is now inserted in the RNA as an exon</li>
    </ul>
    <p><i><b>NOTE:</b> </i>at protein level, when RNA was not analysed the protein description
    is <em>p.?</em>. When RNA was analysed, the change is described depending on the effect on
    the reading frame;<ul>
      <li><u>in frame;</u>&nbsp; describe as an insertion (e.g. <em>p.Arg29_Lys30insHisGluGlyHisGln</em>)
        or deletion/insertion (e.g. <em>p.Cys172delins16</em>)</li>
      <li><u>out of frame</u> (or in frame with stop codon)<u>;</u>&nbsp; describe as a frame
        shift (e.g. <em>p.Arg29Lysfs*23</em>)</li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="more"></a>One change, more RNA molecules</h3>

<p>When a change affects RNA-processing, yielding two or more transcripts, these are
described between square brackets, separated by a&nbsp; &quot;,&quot;-character (<a
HREF="recs-RNA.html#2RNA"><i>see Recommendations</i></a>)&nbsp; 

<ul>
  <li>r.[=, 73_88del] denotes the nucleotide change c.76A&gt;C causing the appearance of two
    RNA molecules, one normal transcript and one containing a deletion of nucleotides 73 to 88
    (shift of the splice donor site to within the exon)</li>
  <li>r.[76a&gt;c, 73_88del] denotes the nucleotide change c.76A&gt;C causing the appearance
    of two RNA molecules, one carrying this variation only and one containing in addition a
    deletion of nucleotides 73 to 88 (shift of the splice donor site to within the exon)</li>
  <li>r.[88g&gt;a; 88_89ins88+1_88+45] denotes the nucleotide change c.88G&gt;A causing an
    insertion of the intronic nucleotides 88+1 to 88+45 (shift of the splice donor site to an
    intronic position)</li>
  <li>r.[=, 88_89ins88+1_88+10; 88+2t&gt;c] denotes the intronic variant c.88+2T&gt;C causing
    the appearance of two RNA molecules, one normal (r.=) and one containing an insertion of
    the intronic nucleotides 88+1 to 88+10 with the nucleotide change 88+2t&gt;c</li>
</ul>

<hr>

<h3><a NAME="del"></a>Large deletions</h3>

<p>Large deletions involving the <b><i>promoter region</i></b> of the gene or the <i><b>gene's
3'-end</b></i> usually have an unpredictable effect on transcript level. In general, when
the promoter is deleted no transcript will be produced (change described as
&quot;r.0&quot;), unless another promoter is activated. The latter frequently occurs when
a deletion <i><b>fuses two genes</b></i> which are directed in the same transcriptional
orientation. When a deletion removes the gene's 3'-end, the transcript usually finds a new
3'-terminal exon. In addition to the changes which will affect the <i>content of the
transcript</i> (sequences transcribed), most large deletions will also affect the <i>transcript
levels</i>. 

<ul>
  <li><b>promoter deletion</b> <ul>
      <li>new promoter activated <ul>
          <li><u>intra-genic</u> (downstream of the normal promoter); deletion of promoter, exon 1 and
            exon 2 and activation of a promoter located in intron 3 which generates a new transcript
            inserting the intronic sequence from 123+98 to 124-112 (coding DNA Reference Sequence)
            directly upstream of exon 4 -&nbsp; <i><b>r.-30_123delins123+98_124-112</b></i></li>
          <li><u>extra-genic</u> (upstream of the normal promoter); deletion of promoter and exon 1
            and activation of a promoter upstream of the gene (sequence present in GenBank file
            ABC01234) which generates a new transcript inserting the genomic sequence from 1345 to
            1476 directly upstream of exon 2 -&nbsp; <i><b>r.-30_12delinsABC01234:g.1345_1476</b></i></li>
          <li><u>fusion with upstream gene</u>; the deletion fuses the transcript with that of the
            upstream XYZ-gene (GenBank file AB001235.1). The fusion transcript contains exons 1 to 5
            of the XYZ-gene (coding DNA positions -124 to 1289) directly upstream of exon 2 -&nbsp; <i><b>r.-30_12delinsAB001235.1:r.-124_1298</b>&nbsp;<font
            color="#FF0000"> (<a HREF="disc.html#fusion">see Discussion</a>)</font></i></li>
        </ul>
      </li>
    </ul>
  </li>
  <li><b>deletion 3'-terminal exon</b><i>&nbsp; (new 3'-terminal exon transcribed)</i> <ul>
      <li><u>intra-genic</u> (upstream of the normal 3'-terminal exon); deletion of exons 4 to 5
        and activation of a new polyA-addition site in intron 3 which generates a new transcript
        inserting the intronic sequence from 123+1 to 122+138 (coding DNA Reference Sequence)
        directly downstream of exon 3 -&nbsp; <i><b>r.124_500delins123+1_123+138</b></i></li>
      <li><u>extra-genic</u> (downstream of the normal promoter); deletion of exons 4 to 5 and
        activation of a new polyA-addition site downstream of the gene (sequence present in
        GenBank file AB001234.1) which generates a new transcript inserting the genomic sequence
        from 1345 to 1476 directly downstream of exon 3 -&nbsp; <i><b>r.124_500delinsAB001234.1:g.1345_1476</b></i></li>
      <li><u>fusion with downstream gene</u>; the deletion of exons 4 to 5 activates a new
        polyA-addition site located in the downstream ZYX-gene (GenBank file AB053210.2), which is
        transcribed in an opposite transcriptional orientation. The fusion transcript contains
        exons 1 to 3, directly upstream of an intron 5 sequence of the ZYX-gene (complementary
        strand, coding DNA positions 1289-365 to 1289-73)&nbsp; -&nbsp; <i><b>r.124_500delinsoAB053210.2:r.1289-365_1289-73</b>&nbsp;<font
        color="#FF0000"> (<a HREF="disc.html#fusion">see Discussion</a>)</font></i></li>
    </ul>
  </li>
</ul>

<hr>

<p align="center">| <a HREF="#top">Top of page</a> | <a HREF="index.html">MutNomen
homepage</a> | <a href="checklist.html">Check-list</a> |<br>
| <b>Recommendations:</b>&nbsp; <a href="recs-DNA.html">DNA</a>,&nbsp; <a
href="recs-RNA.html">RNA</a>,&nbsp; <a href="recs-prot.html">protein</a>, <a
href="uncertain.html">uncertain</a> |<br>
| <a HREF="disc.html">Discussions</a> | <a HREF="FAQ.html">FAQ's</a> | <a
href="standards.html"><font face="Verdana, Arial, Helvetica, sans-serif">Symbols, codons,
etc.</font></a> | <a href="history.html">History</a> |<br>
| <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef / symbols</a>,&nbsp;
<a HREF="examplesDNA.html">DNA</a>,&nbsp; <a HREF="examplesAA.html">protein</a> | </p>

<table width="100%" border="0" cellspacing="0" cellpadding="15" bgcolor="#ff9900">
  <tr>
    <td><p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"
    size="-5"><a href="http://www.HGVS.org/copyright.html">Copyright ©</a> </font><font
    size="1">HGVS 2007 All Rights Reserved<br>
    Website Created by Rania Horaitis, Nomenclature by J.T. Den Dunnen - <a
    href="http://www.HGVS.org/disclaimer.html">Disclaimer</a> </font></td>
  </tr>
</table>
</body>
</html>
