<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	17-11-02	URL: 	http://www.dmd.nl/mutnomen_current.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>HGVS recommendations: AA examples</title>    <style type="text/css"><!--  h1 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style>  </head>  <body bgcolor="#FFFFFF" link="#0000FF" text="#000000" vlink="#9999ff">    <table cellpadding="10" cellspacing="0" bgcolor="#CCFF99" border="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Description of sequence changes:&nbsp;<br>              examples protein-level</h3>          </td>        </tr>      </tbody>    </table>    <table cellpadding="10" cellspacing="0" border="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified September 1, 2015</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i>Since references to WWW-sites are not yet acknowledged        as citations,        please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den          Dunnen JT          and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when referring to        these pages.</i></p>    <hr>    <h2>Contents</h2>    <ul>      <li><a href="#intro"><b>Introduction</b></a>        <ul>          <li>reference sequence</li>        </ul>      </li>      <li><b>Examples&nbsp; -&nbsp; protein changes</b>        <ul>          <li><a href="#general">general</a> </li>          <li><b>basic changes</b>            <ul>              <li><a href="#silent">silent</a></li>              <li><a href="#sub">substitutions</a>&nbsp; <i>(missense,                  nonsense, no-stop)</i> </li>              <li><a href="#del">deletion</a> </li>              <li><a href="#dup">duplication</a>&nbsp; <i>(variability of short                  sequence repeats)</i> </li>              <li><a href="#ins">insertion</a>&nbsp;</li>              <li>inversion&nbsp; <i>(description not used at protein level)</i></li>              <li><a href="#trans">translocation</a>&nbsp; <i>(fusion proteins)</i></li>            </ul>          </li>          <li><a href="#complex">complex rearrangements</a>            <ul>              <li>deletion/insertion <i>(indel, frame shift)</i> </li>            </ul>          </li>          <li><a href="#misc">miscellaneous</a>            <ul>              <li><a href="#2in1">two changes in one allele</a> </li>              <li><a href="#2unknown">two change in one individual - allele                  unknown</a></li>              <li><a href="#2in2">changes in different alleles</a><i>&nbsp;                  (recessive diseases)</i> </li>            </ul>          </li>          <li><a href="file:///G:/WWW/HGVS/uncertain.html">uncertainties</a>&nbsp;            <i>(exact position not known; Southern blot, PCR, arrayCGH,              SNP-array, ...)</i></li>        </ul>      </li>      <li><b><a href="examplesDNA.html">Examples&nbsp; -&nbsp; DNA changes</a></b></li>      <li><a href="examplesRNA.html"><b>Examples&nbsp; -&nbsp; RNA changes</b></a></li>    </ul>    <hr>    <h2><a name="intro"></a>Introduction</h2>    <hr>    <p>Within this page examples will be given for the description of sequence      variants on <i><b>protein          level</b></i>, examples to describe changes at <a href="examplesDNA.html">DNA</a>      and <a href="examplesRNA.html">RNA level</a> are given at other pages.      All examples are described      <i><b>relative to a reference sequence</b></i>, here the amino acid      (protein)      sequence.&nbsp;</p>    <h3><a name="RefSeq"></a>Reference sequence</h3>    <table align="center" border="1" width="688">      <tbody>        <tr>          <th colspan="2" width="224"><b>Part of gene</b></th>          <th align="center" width="139">&nbsp;nucleotide numbering<br>            <i>protein</i><br>            Reference Sequence</th>          <th align="center" width="163">nucleotide numbering<br>            <i>coding DNA</i><br>            Reference Sequence</th>          <th align="center" width="134">nucleotide numbering<br>            <i>genomic<br>            </i>Reference Sequence&nbsp;</th>        </tr>        <tr>          <td colspan="2" width="224">5' gene flanking region</td>          <td align="center" width="139">&nbsp;-</td>          <td align="center" width="163">(-300 to -31)</td>          <td align="center" width="134">1 to 270</td>        </tr>        <tr>          <td rowspan="2" width="70"><b>exon 1</b></td>          <td width="148">5' UTR</td>          <td align="center" width="139">&nbsp;-</td>          <td align="center" width="163">-30 to -1</td>          <td align="center" width="134">271 to 300</td>        </tr>        <tr>          <td width="148"><b>coding region</b></td>          <td align="center" width="139">1 to 4</td>          <td align="center" width="163">1 to 12</td>          <td align="center" width="134">301 to 312</td>        </tr>        <tr>          <td colspan="2" width="224">intron 1</td>          <td align="center" width="139">-</td>          <td align="center" width="163"><i>12+1 ... 12+50,<br>              13-50 ... 13-1</i></td>          <td align="center" width="134">313 to 412</td>        </tr>        <tr>          <td colspan="2" width="224"><b>exon 2</b></td>          <td align="center" width="139">5 to 29 (30)</td>          <td align="center" width="163">13 to 88</td>          <td align="center" width="134">413 to 488</td>        </tr>        <tr>          <td colspan="2" width="224">intron 2</td>          <td align="center" width="139">-</td>          <td align="center" width="163"><i>88+1 ... 88+100,<br>              89-100 ... 89-1&nbsp;</i></td>          <td align="center" width="134">489 to 689</td>        </tr>        <tr>          <td colspan="2" width="224"><b>exon 3</b></td>          <td align="center" width="139">30 to 41</td>          <td align="center" width="163">89 to 123</td>          <td align="center" width="134">689 to 723</td>        </tr>        <tr>          <td width="70">intron 3</td>          <td width="148">contains rare alternatively spliced exon from 800 to            859 (coding DNA 123+77 to 123+136)</td>          <td align="center" width="139">-</td>          <td align="center" width="163"><i>123+1 ... 123+150,<br>              124-150 ... 124-1&nbsp;</i></td>          <td align="center" width="134">724 to 1023</td>        </tr>        <tr>          <td colspan="2" width="224"><b>exon 4</b></td>          <td align="center" width="139">42 to 100</td>          <td align="center" width="163">124 to 300</td>          <td align="center" width="134">1024 to 1200</td>        </tr>        <tr>          <td colspan="2" width="224">intron 4</td>          <td align="center" width="139">-</td>          <td align="center" width="163"><i>300+1 ... 300+200,<br>              301-200 ... 301-1&nbsp;</i></td>          <td align="center" width="134">1201 to 1600</td>        </tr>        <tr>          <td rowspan="2" width="70"><b>exon 5</b></td>          <td width="148"><b>coding region</b></td>          <td align="center" width="139">101 to 109</td>          <td align="center" width="163">301 to 330</td>          <td align="center" width="134">1601 to 1630</td>        </tr>        <tr>          <td width="148">3' UTR, containing a (CA)<sub>7</sub>-stretch from nts            1700 to 1713 (coding DNA *71 to *83); poly-A addition site at 1825            (coding DNA *195) </td>          <td align="center" width="139">-</td>          <td align="center" width="163">*1 to *220</td>          <td align="center" width="134">1631 to 1850</td>        </tr>        <tr>          <td colspan="2" width="224">3' gene flanking region</td>          <td align="center" width="139">-&nbsp;</td>          <td align="center" width="163">(*221 to *370)</td>          <td align="center" width="134">1851 to 2000</td>        </tr>      </tbody>    </table>    <p><b><big>Legend:</big></b><br>      Reference sequence of imaginary gene used for the exaples given on this      page. Nucleotide      +1 in the coding DNA reference sequence is the A of the ATG translation      initiation codon.      Abbreviations used: nt = nucleotide, nts = nucleotides, UTR = untranslated      region of the      mRNA. For a picture of part of this hypothetical sequence <i><a href="refseq_figure.html">see          Figure</a></i>.</p>    <hr>    <h3><a name="general"></a>General</h3>    <p>It should be noted that the descriptions at protein level, even more than      those at RNA      level, are mostly <b><i>deduced</i></b> and not based on experimental      evidence.      Publications describing changes at protein level should make it clear      whether experimental      proof was available or not. In fact, when changes are reported for which      experimental      proof is not available one should consider to list them between brackets.    </p>    <p>Sequence changes at protein level are basically described like those at      the <a href="examplesDNA.html">DNA level</a>, with a few modifications; </p>    <ul>      <li>the <a href="codon.html">three-letter amino acid code</a> is prefered        (see Discussion), with <em><strong>"Ter"</strong></em> or <em><strong>"*"</strong></em>        designating a translation termination codon; for clarity this page        describes changes using the three-letter amino acid </li>      <li><b>amino acid numbering</b>        <ul>          <li>descriptions start with the amino acid, followed by its number            (like p.<i>Cys24 or p.C24</i>) </li>          <li>the translation initiator Methionine is numbered as +1 </li>          <li>amino acids after the stop codon (here after *110) are numbered            relative to the stop codon as Gly*110+1, Trp*110+2, etc.&nbsp; </li>        </ul>      </li>      <li><b>"silent" changes</b><br>        description of so called "silent" changes in the format p.Leu54Leu (or        p.L54L) is not allowed; descriptions should be given at <b><i>DNA level</i></b>,        it is <i><b>non-informative</b></i> and <b><i>not unequivocal</i></b>        (there are five possibilities at DNA level which may underlie        p.Leu54Leu);&nbsp; correct description has the format c.162C&gt;G.</li>    </ul>    <hr>    <h3><a name="silent"></a><b>Silent changes</b></h3>    <p>Description of so called "silent" changes in the format p.(Leu54Leu) (or      p.(L54L)) should not be used. When desired such changes can be described      using <strong><em>p.(=)</em></strong>.      Descriptions should always be given at <b><i>DNA level</i></b>&nbsp;(<a href="disc.html#silent"><i>see          Discussion</i></a>).</p>    <hr>    <h3><a name="sub"></a>Substitutions</h3>    <p>Substitutions should be described without using the specific <b><i>"&gt;"</i></b>-character
which      is used on DNA and RNA level (i.e. p.Trp26Cys, not p.Trp26&gt;Cys). </p>    <ul>      <li><a name="ini"></a><b>translation initiation codon </b><i>(Methionine          codon, <a href="disc.html#Met">see Discussion</a>)</i><br>        description depends on the consequences of the change on the translation        product (protein);        <ul>          <li><b><i>no protein produced</i>&nbsp; </b>-&nbsp; p.0<br>            e.g. as a consequence of a variant deleting the promoter/first exon            or a change in the translation initiation codon (experimental data            should be available)</li>          <li><i><b>effect unknown (most cases)</b></i>&nbsp; -&nbsp; p.0?&nbsp;            (alternatively p.Met1?)<br>            when experimental data show that no protein is made, the description            <i>p.0</i> should be used.</li>          <li><i><b>new translation initiation site</b></i>            <ul>              <li><i><u>upstream</u></i>&nbsp; -&nbsp; p.Met1ValextMet-12                (alternatively p.M1VextM-12)<i><b><br>                  </b></i>denotes an extension of 12 amino acids (Met-12 to                Thr-1) of the protein combined with an amino acid change                (Met1Val)</li>              <li><i><u>downstream</u></i>&nbsp; -&nbsp; p.Phe2_Met46del                (alternatively p.F2_M46del)<br>                denotes inactivation of the normal and activation of a                downstream translation initiation site (MET) resulting in                deletion&nbsp;of the first 45 amino acids (Met-1 to Lys-45) of                the protein.<br>                <span style="font-weight: bold; font-style: italic;">NOTE:</span>                for the description of the change the 3' rule applies so                deletion of Phe2 to Met46.</li>            </ul>          </li>        </ul>      </li>      <li><b>one amino acid to</b>        <ul>          <li><i>another amino acid</i><b>&nbsp;<i>(missense change)</i></b><br>            Tryptophan 26 to a Cysteine; p.Trp26Cys (alternatively p.W26C)</li>          <li><i>a stop codon <b>(nonsense change)</b></i><b><br>            </b>Tryptophan 26 to a stop codon (*); p.Trp26* (alternatively            p.W26*)<br>            <em><strong>NOTE:</strong></em> this change is <em><strong>not</strong></em>            described as a deletion of the C-terminal end of the protein (e.g.            p.Trp26_Arg1623del or p.Trp26_Arg1623del)</li>        </ul>      </li>      <li><b>translation termination codon </b><i>(stop codon, <b>no-stop            change</b>)&nbsp;</i>        <ul>          <li>p.*110Glnext*17 (alternatively p.*110Qext*17)<br>            the stop codon (*) at position 110 is changed to a codon for            Glutamine (Gln, Q), adding a tail of 17 new amino acids (incl.            Gln110) to the protein's C-terminus after which a new stop codon is            reached (<i>'codon *127'</i>)</li>          <li>p.*321Argext*? (alternatively p.*321Rext*?) describes a variant in            the stop codon (*) at position 321, changing it to a codon for            Arginine (Arg, R) and adding a tail of new amino acids of unknown            length since the shifted frame does not contain a new stop codon.</li>        </ul>      </li>    </ul>    <p><em><b>NOTE:</b></em> <strong><em>polymorphic variants</em></strong> are      sometimes      described as p.36Leu/Ile (p.36L/I) or p.36Leu/Leu (p.36L/L) but this is      not correct (<a href="recs-prot.html#silent"><i>see Protein level          recommendations</i></a>).</p>    <hr>    <h3><a name="del"></a>Deletions</h3>    <hr>    <p>Deletions are designated by <i><b>"del"</b></i> after a description of      the      deleted segment, i.e. the first (and last) amino acid(s) deleted. </p>    <ul>      <li>M<u><strong>K</strong></u>LGHQQQCC to M<b>_</b>LGHQQQCC is described        as p.Lys2del (alternatively p.K2del)</li>      <li>MKL<b><u>GHQ</u></b>QQCC to MKL<b>___</b>QQCC&nbsp; is described as        p.Gly4_Gln6del (alternatively p.G4_Q6del)</li>      <li>MKLGHQQQCC to MKLGHQQCC is described as p.Gln8del (p.Q8del)<br>        <i><b>NOTE:</b></i> for deletions in single amino acid stretches or        tandem repeats, the most 3' residue is arbitrarily assigned to have been        deleted</li>      <li>if a deletion creates a new amino acid at the deletion junction the        change is described as a <strong>insertion/deletion</strong> (<a href="#indel">see          indels</a>)</li>      <li><strong>initiating methionine change (Met1) causing a N-terminal          deletion<em>&nbsp;</em></strong> (<i><a href="disc.html#Met">see            Discussion</a>,&nbsp; <a href="examplesAA.html#sub">see Examples</a></i>)        <br>        <em><strong>NOTE:</strong></em>&nbsp; changes extending the N-terminal        protein sequence are described as an <a href="#extp">extension</a>        <ul>          <li><i>p.0</i>&nbsp; -&nbsp; no protein is produced (experimental data            should be available)<br>            <em><strong>NOTE:</strong></em> this change is not described as <em>p.Met1_Leu833del</em>,            i.e. as a deletion removing the entire protein coding sequence</li>          <li><i>p.Met1_Lys45del&nbsp;</i> -&nbsp; a new translation initiation            site is activated (at Met46)</li>          <li><i>p.Met1?</i> -&nbsp; denotes that amino acid Methionine-1            (translation initiation site) is changed and that it is unclear what            the consequence of this change is</li>        </ul>      </li>      <li><strong>nonsense variant<br>        </strong>a nonsense variants is a <em><strong>special type of amino            acid deletion</strong></em> removing the entire C-terminal part of a        protein starting at the site of the variant. A nonsense change is        described using the format <i>p.Trp26Ter</i> (alternatively <i>p.Trp26*</i>).
        The description does not include the deletion at protein level from the        site of the change to the C-terminal end of the protein (stop codon)        like <em>p.Trp26_Leu833del</em> (the deletion of amino acid residue        Trp26 to the last amino acid of the protein Leu833).</li>      <li>p.(Trp26Ter) indicates RNA nor protein was analysed but amino acid        Tryptophan26 (Trp, W) is predicted to change to a stop codon (Ter) (<i>alternatively          p.(W26*) or p.(Trp26*)</i>)</li>    </ul>    <p><i><b>NOTE:</b></i> for all descriptions the <b>most C-terminal position</b>      possible      is arbitrarily assigned to have been changed</p>    <hr>    <h3><a name="dup"></a>Duplications</h3>    <hr>    <p>Duplications are designated by <i><b>"dup"</b></i> after a description      of the      duplicated segment, i.e. the first (and last) amino acid(s) duplicated. </p>    <ul>      <li>MKLGHQQQCC to MKLGHQ<strong><u>GHQ</u></strong>QQCC is described as        p.Gly4_Gln6dup (alternatively p.G4_Q6dup)</li>      <li>MKLGHQQQCC to MKLGHQQQ<b><u>Q</u></b>CC is described as p.Gln8dup        (alternatively p.Q8dup)<br>        <i><b>NOTE:</b></i> for duplications in single amino acid stretches or        tandem repeats, the most 3' residue is arbitrarily assigned to have been        duplicated</li>      <li>MKLGHQQQCC to MKLGHQ<u><b>HQ</b></u>QQCC is described as        p.His5_Gln6dup (alternatively p.H5_Q6dup)<br>        <b><i>NOTE:</i></b> duplicating insertions in single amino acid        stretches (or short tandem repeats) should be described as a duplication        and not as an insertion - so in the example shown p.Gln6_Gln7insHisGln        (alternatively p.Q6_C7insHQ) is not correct</li>      <li><strong>variability of short sequence repeats</strong> are designated        as p.Gln6(3_6) (alternatively p.Q6(3_6)) describing that the Glutamine        (Gln, Q) stretch starting at position 6 in MKLGHQQQCC is found repeated        3 to 6 times in the population</li>    </ul>    <hr>    <h3><a name="ins"></a>Insertions</h3>    <hr>    <p>Insertions are designated by <b><i>"ins" </i></b>after a description of      the      amino acids flanking the insertion site, followed by a description of the      inserted amino      acids. When the insertion is large it may be described by its length (e.g.      p.Lys2_Leu3ins34). However, it should be possible to derive the inserted      sequence from the      description at DNA level. Duplicating insertions should be described as      duplications (<a href="disc.html#dupins">see Discussion</a>). </p>    <ul>      <li>p.Lys2_Leu3insGlnSer (alternatively p.K2_L3insQS) describes the cahnge        from MKLGHQQQCC to MK<u><strong>QS</strong></u>LGHQQQCC</li>      <li>p.Arg78_Gly79ins23 describes the in-frame insertion of a 23 amino acid        sequence between residues Arg-78 and Gly79. Such an insertion can e.g.        derive from the inclusion of intronic sequences resulting from a change        affecting RNA splicing (see <a href="examplesRNA.html#splice">Examples          RNA-level</a>).<br>        <em><strong>NOTE:</strong></em> the inserted sequence at DNA/RNA level        should be specified, where necessary using a sequence database        submission (Genbank, EMBL, DDJB) and listing of the accession number.</li>    </ul>    <p>When an insertion creates a new amino acid at the insertion junction the      change is      described as an <strong>insertion/deletion</strong> (<a href="#indel">see        indels</a>) </p>    <hr>    <h3><a name="trans"></a>Translocations</h3>    <hr>    <p>Translocations at protein level occur when a translocation at DNA level      leads to the      production of a fusion protein, joining the N-terminal end of the protein      on one      chromosome to the C-terminal end of the protein on the other chromosome      (and vice versa).      No recommendations have been made sofar to describe protein      translocations. </p>    <ul>      <li>t(X;17)(DMD:p.Met1_Val1506; SGCA:p.Val250_*387) describes a fusion        protein resulting from a translocation between the chromosomes X and 17;        the fusion protein contains an N-terminal segment of DMD (dystrophin,        amino acids Methionine-1 to Valine-1506), and a C-terminal segment of        SGCA (alpha-sarcoglycan, amino acids Valine-250 to the stop codon at        387)</li>    </ul>    <hr>    <h3><a name="complex"></a>Complex rearrangements</h3>    <hr>    <p>Complex rearrangements are rearrangements which consist of several      different types of      the <i><b>six elementary content changes</b> substitution, deletion,        duplication,        insertion, inversion and translocation</i>. Such rearrangements can be      very complex and      difficult to describe. Specific recommendations to describe such changes      have not made.      Complex rearrangements can be best described as a combination of the      elementary      changes.&nbsp;</p>    <p><a name="indel"></a><u><b>Deletion/insertions</b></u> (indels) are      described as a <i>deletion</i>      followed by an <i>insertion</i> (<a href="disc.html#indel">see Discussion</a>)    </p>    <ul>      <li>p.(Cys28_Lys29delinsTrp) (RNA not analysed, alternatively        p.(C28_K29delinsW)) denotes a 3 bp deletion affecting the codons for        Cysteine-28 and Lysine-29, substituting them for a codon for Tryptophan</li>      <li>p.Cys28delinsTrpVal (alternatively p.C28delinsWV) denotes a 3 bp        insertion in the codon for Cysteine-28, generating codons for Tryptophan        (W) and Valine (V)</li>      <li><u><strong>frame shift changes</strong></u> (<a href="disc.html#frameshift">see
          Discussion</a>) cause, from a specific point onwards, the replacement        of the normal C-terminal end of a protein for a new segment, encoded by        the shifted reading frame. Frame shift changes can thus be best        considered as deletion/insertions (indels). Frame shifts are designated        by <b><i>"fs"</i></b> after the amino acid(s) affected by the change.        Descriptions either use a <i><b>short </b></i>("fs" only) or <b><i>long</i></b>        ("fs*#") notation; the long description should include the change        occurring at the site of the frame shift (<a href="disc.html#frameshift">see          Discussion</a>). In "fs*#", "*#" indicates at which codon position the        new reading frame ends in a stop codon (*). The position of the stop in        the new reading frame is calculated starting at the first amino acid        that is changed by the frame shift, and ending at the first stop codon        (*#). Thus, effectively, on DNA/RNA level the change might be one or        more coding triplets up- or downstream.<br>        <b><i>NOTE:</i></b> the shifted reading frame is thus open for '#-1'        amino acids.        <ul>          <li>p.(Arg97Profs*23) (RNA not analysed, short p.(Arg97fs)) denotes a            frame shifting change with Arginine-97 as the first affected amino            acid, changing into a Proline, and the new reading frame ending in a            stop at position 23</li>          <li>p.Arg97Hisfs*5 (short p.Arg97fs) denotes a frame shifting change            with Arginine-97 as the first affected amino acid, changing into a            Histidine, and the new reading frame ending in a stop at position 5</li>          <li>p.(Leu30Serfs*3) (RNA not analysed, short p.(Leu30fs)) denotes a            frame shifting change that deletes amino acids Leucine-30 to            Cysteine-42 (exon 3 of the gene), substituting these for a Serine at            the deletion junction and ending in the new reading frame in a stop            at position 3</li>          <li>p.(Ile327Argfs*?) (RNA not analysed, alternatively p.(Ile327fs))            describes the consequences of a frame shifting change (e.g. a            1-nucleotide insertion) with Isoleucine-327 as the first affected            amino acid, replacing it for an Arginine and creating a new reading            frame which does not encounter a new stop codon.</li>        </ul>      </li>    </ul>    <hr>    <h3><a name="misc"></a>Miscellaneous</h3>    <hr>    <ul>      <li><i><b><a name="2in1"></a>Two changes in one allele</b></i> are        described as "[first change; second change]" (<i><a href="disc.html#2in1">see            Discussion</a></i>)        <ul>          <li>p.[Trp13*; Pro43Ala]&nbsp;(alternatively p.[W13*; P43A]) describes            a Tryptophan-13 to stop codon and Proline-43 to Alanine change in            one allele (on one chromosome) </li>          <li>p.[(Ala25Thr; Gly28Val)] indicates two predicted changes in one            allele (RNA nor protein was analysed); amino acid Alanine25 to            Threonine and Glycine-28 to Valine</li>        </ul>      </li>      <li><em><strong>One change in one allele yielding two transcripts/two            encoded proteins</strong></em> (e.g. deriving from DNA change that        generates 2 different transcripts) are described as "[first change,        second change]" (<a href="disc.html#moreRNAs"><i>see Discussion</i></a>).&nbsp;
        <ul>          <li>p.[Asn26His, Ala25_Gly29del] denotes two protein changes deriving            from a change in one allele at DNA level (c.76A&gt;C) resulting in            two transcripts (r.[76a&gt;c, 73_88del] ) yielding two proteins, one            where amino acid Asparagine25 changes to Histidine and one with a            deletion of amino acids Asparagine25 to Glycine29</li>        </ul>      </li>      <li><i><b><a name="2unknown"></a>Two changes in one individual, unknown in            which allele </b></i>are described as "[first change (;) second        change]" (<i><a href="disc.html#2unknown">see Discussion</a></i>)        <ul>          <li>p.[Trp13*(;)Glu61Gln]&nbsp;(alternatively p.[W13*(;)E61Q])            describes a Tryptophan-13 to stop codon and Glutamine-61 to Glutamic            acid change in one individual while it is unknown whether these            changes are on the same or different alleles </li>        </ul>      </li>      <li><i><b><a name="2in2"></a>Recessive disease (one change in each allele)</b></i>        are are described as "[change allele 1];[change allele 2]" (<i><a href="disc.html#2in2">see
            Discussion</a></i>)        <ul>          <li>p.[Trp13*];[Cys28Arg]&nbsp;(alternatively p.[W13*];[C28R])            describes a Tryptophan-13 to stop codon change in one allele            (chromosome) and a Cysteine-28 to Arginine change in the other            allele (chromosome) </li>          <li>p.[Trp13*];[?]&nbsp;(alternatively p.[W13*];[?]) describes a            Tryptophan-13 to stop codon change in one allele (chromosome) and an            unknown change in the other allele (chromosome) </li>          <li>p.[Trp13*];[=]&nbsp;(alternatively p.[W13*];[=]) describes a            Tryptophan-13 to stop codon change in one allele (chromosome) and a            normal sequence in the other allele (chromosome) </li>        </ul>      </li>      <li><strong><i>Two changes in different genes</i>&nbsp;</strong> are        described as SGCA:p.[Arg175*]; SGCB:p.[Cys305Ser] - the SGCA gene        contains a nonsense change, the SGCB gene a Cys-to-Ser substitution </li>      <li><em><b>Mosaicism</b></em> is described using "/"        <ul>          <li><i>p.[=/Arg83Ser] describes a mosaic organism or somatic tissue              where the allele in some cells contain the normal sequence (Arg83              described as '='), while other cells contain a Ser at this              position</i></li>        </ul>      </li>      <li><em><b>Chimerism</b></em> is described using "//"        <ul>          <li><i>p.[=//Arg83Ser] describes a chimeric organism where the allele              in some cells contain the normal sequence (Arg83 described as              '='), while other cells contain a Ser at this position</i></li>        </ul>      </li>    </ul>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">Homepage</a>      | <a href="checklist.html">Check-list</a> |<br>      | <b>Recommendations:</b>&nbsp; <a href="recs-DNA.html">DNA</a>,&nbsp; <a        href="recs-RNA.html">RNA</a>,&nbsp;      <a href="recs-prot.html">protein</a>, <a href="uncertain.html">uncertain</a>      |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="standards.html">Symbols,        codons, etc.</a> | <a href="history.html">History</a> |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef /        symbols</a>,&nbsp;      <a href="examplesDNA.html">DNA</a>,&nbsp; <a href="examplesRNA.html">RNA</a>      | </p>    <table cellpadding="15" cellspacing="0" bgcolor="#ff9900" border="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright                  ©</a> </font><font size="1">HGVS 2007 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>