<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	17-11-02	URL: 	http://www.dmd.nl/mutnomen_current.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>HGVS recommendations; Frequently Asked Questions (FAQ) </title>    <style type="text/css"><!--  h1 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body bgcolor="#FFFFFF" link="#0000FF" text="#000000" vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Frequently asked questions regarding the              description of sequence variants</h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified November 16, 2015</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i>Since references to WWW-sites are not yet acknowledged        as citations, please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den
          Dunnen JT and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when        referring to these pages.</i></p>    <hr>    <h2>Contents</h2>    <ul>      <li><a href="#intro"><b>Introduction</b></a></li>      <li><a href="refseq.html"><strong>Reference sequences</strong></a>        <ul>          <li><a href="refseq.html#mtDNA">changes in mitochondrial DNA</a> </li>          <li><img src="new.gif" border="0" height="12" width="31"><em><a href="refseq.html#ncRNA">changes
                in non-coding RNA genes</a> </em></li>        </ul>      </li>      <li><a href="#DNA"><strong>DNA level changes</strong></a>        <ul>          <li><a href="#repeat">location breakpoint in repetitive sequence ?</a></li>          <li><a href="#exdel">description exon deletions ?</a></li>          <li><img src="new.gif" border="0" height="12" width="31"><a href="#sub2">dinucleotide
              change or insertion/deletion ?</a></li>          <li><a href="#dupins">duplication or insertion</a></li>          <li><a href="#pdup">imperfect duplications</a></li>          <li><img src="new.gif" border="0" height="12" width="31"><a href="#most3">3'
              position arbitrarily assigned changed</a> - VNTR's</li>          <li><img src="new.gif" border="0" height="12" width="31"><a href="#insname">large
              insertions</a>&nbsp; <i>(add origin like L1 or Alu ?)</i></li>          <li><a href="#2alleles">changes in different alleles</a></li>          <li><a href="#2unknown">two changes in one individual, alleles unknown</a></li>          <li><a href="#2in2">recessive disease - changes in different genes</a></li>          <li><img src="new.gif" border="0" height="12" width="31"><a href="#fe-male">X-chromosome
              alleles in males and females</a></li>          <li><img src="new.gif" border="0" height="12" width="31"><a href="#mosaic">mosaic
              cases</a></li>          <li><b><i><a href="#promSNP">SNP's</a></i></b> <a href="#promSNP">in              the promoter region</a></li>        </ul>      </li>      <li><a href="#RNA"><strong>RNA level changes</strong></a> </li>      <li><a href="#protein"><strong>Protein level changes</strong></a>        <ul>          <li>change in 5'UTR, new initiation codon</li>          <li><a href="disc.html#1or3">should I use the one- or three-letter              amino acid code</a></li>          <li><a href="#protDNA">should I incorporate knowledge of the change at              DNA-level</a></li>          <li><a href="#frameshift">frameshifts</a></li>          <li><i><img src="new.gif" border="0" height="12" width="31"></i><a href="#nostop">no
              stop codon encountered</a></li>        </ul>      </li>    </ul>    <hr>    <h3><a name="intro"></a>Introduction</h3>    <p>This page gives an overview of the questions we have received regarding      the description of sequence variations based on the <a href="recs.html">existing
        recommendations</a> (published in by den Dunnen and Antonarakis (2000, <em><b>Mutation
          nomenclature extensions and suggestions to describe complex mutations:          a discussion</b>. </em>Human Mutation 15: 7-12, <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART"><em>copy
in
          PDF-format</em></a>). </p>    <p><strong>For reactions:</strong> <em>E-mail</em> (<strong>to:</strong>      HGVSmn @ JohanDenDunnen.nl) or use the <em><a href="http://dmz-web03.lumc.nl/new.hgvs.org/?q=forum/2">HGVS
          variant description forum</a></em>. </p>    <hr>    <h3><a name="DNA"></a>DNA level changes</h3>    <p><em><a name="repeat"></a><strong>Question&nbsp;&nbsp; </strong>(Marco        Montagna, Padua, Italia) <br>        Recently, I have been involved in the molecular characterization of        BRCA1 gene rearrangements that are becoming more and more frequent in        breast/ovarian cancer families. Most often these rearrangements are        mediated by Alu sequences with a very high homology that reaches 100% in        the breakpoint region. I looked at the reference papers on mutation        nomenclature, but I still have some doubts on how to define such kind of        mutations. In particular, if a genomic deletion is mediated by <b>Alu          sequences that are identical over a large nucleotide stretch          containing the breakpoint, what nucleotide should be indicated?</b>.        Could I indicate the most 3' one (considering the "sense" strand),        similarly to the rule for deletions in repeated sequences?. Moreover, if        more than one genomic sequence is present in GenBank, which one should        be considered ?. For instance, for a rearrangement that deletes a        genomic region of 20kb containing exon 1 and the upstream sequence, with        a breakpoint occurring over a stretch of nucleotides that are identical        in the two recombining sequences, and a genomic reference sequence of        the antisense strand, I would suggest the following definition:&nbsp;        nt.X (the most 5' in the identity region of the "antisense"        reference&nbsp; sequence, i.e. the most 3' in the "sense" strand) --        nt.Y del 20kb (exon &gt; 1). Would be that fine ?.</em></p>    <p><strong>Answer</strong><br>      You touch on two subjects; <i><b>location of the breakpoint</b></i> and <i><b>reference
          sequence</b></i>.<br>      &nbsp;&nbsp;&nbsp;&nbsp; <u><i><b>Breakpoint;</b></i></u> indeed, like      you suggest, when breakpoints occur in stretches of identical sequences      the most 3' position (considering the sense strand) is used to describe      the position of the breakpoint (<i><a href="recs.html#repeat">see          Recommendations</a></i>).<br>      &nbsp;&nbsp;&nbsp;&nbsp; <b><u><i>Reference sequence;</i></u></b> any      reference sequence would be OK at least when you specify the one you use      (database accession.version number, <i><a href="refseq.html">see          Reference sequence discussion</a></i>). When present, it would be best      to use the genomic Reference Sequence from the <a href="http://www.ncbi.nlm.nih.gov/RefSeq/">RefSeq
        database</a>. When such a sequence is not present you should make,      annotate and submit one (<a href="refseq.html#IVS"><i>see Discussion</i></a>).<br>      &nbsp;&nbsp;&nbsp;&nbsp; Depending on whether a genomic or a coding DNA      reference sequence is used the final description should have the format;      g.1234_7234del (alternative g.1234_7246del6012) or c.123+45_955-234del      (alternative c.123+45_955-234del6012).</p>    <hr>    <p><a name="exdel"></a><em><strong>Question&nbsp;&nbsp; </strong>(Erik-Jan        Kamsteeg, Nijmegen, Nederland) <br>        The recommendations to describe unknown breakpoints are not exactly        clear to me. For example, PCR analysis of a gene on the X-chromosome        shows products for exons 1-3 and no product is detected for exons 4-14        (exon 14 is the last exon of the gene). Since PCR does not work with one        primer, we are not sure whether exon 4 and 14 are completely absent, or        only partially. Therefore, using the first base of exon 4 and the '-?' (<a          href="recs-DNA.html#del">see
          Recommendations</a>) could be wrong, as could be the last base of exon        14 with a '+?'. Therefore, I would like to use the last base of exon 3        with '+?' and the last base of exon 13 with a '+?'. What are your        recommendations? </em></p>    <p><b>Answer<br>      </b>Literally speaking you are right and it is best to set the borders as      precise as possible. So when exon 3 is present in fact the location of the      reverse primer can be used to set the most 5' border (and the same for the      exon 14 primer). Consequently the description could be <b><i>something          like (87+123_88-?)_(923+?_924-98)del</i></b>. Although precise one      might wonder whether such a description is attractive;      c.(87+1_88-1)_(923+1_924-1)del is as clear (<i><a href="uncertain.html">see
          Uncertainties</a></i>). <img alt="" src="new.gif">When it is      difficult to give an exact nucleotide position for a specific      probe/sequence tested, a rule of thumb is to use the central nucleotide. <br>      <br>      <em><strong>NOTE:</strong></em>&nbsp; for simplicity there are more      descriptions that are not fully correct. For example, stop codons are      reported as p.Cys123* while one could argue that p.Cys123_Met2376del is      more precise (Met2376 being the last amino acid of the protein). </p>    <p>&nbsp;</p>    <p><em><a name="exdel"></a><strong>Question</strong><br>        Is a description like c.EX17del, indicating a deletion of exon 17, still        valid? </em></p>    <p><b>Answer<br>      </b>A description like c.EX17del has never been accepted. Descriptions      should indicate the nucleotides affected by the change. Note also that for      many genes exon numbering is often not clearly defined and/or not      described accurately.</p>    <hr>    <p><a name="sub2"></a><em><strong>Question</strong></em><br>      <i>How should I describe the change TGT <u><b>GC</b></u> CA to TGT <u><b>TG</b></u>        CA. Can I call it a <b>dinucleotide mutation or is it a deletion /          insertion mutation</b> ?.</i> </p>    <p><b>Answer</b><br>      Simply describe it as c.4_5delinsTG (alternatively it can be described      c.[4G&gt;T; 5C&gt;G]). Although c.4_5GC&gt;TG is clear and unequivocal,      the description as a deletion/insertion follows the general      recommendations more precisely (<a href="recs-DNA.html#indel"><i>see          Recommendations</i></a>). </p>    <p><a name="sub2"></a><img src="new.gif" border="0" height="12" width="31"><em><strong>Question</strong></em><br>      <em>At position c.2077_2078 in the BRCA1 gene I have a TA insertion. The        published sequence for c.2076_2077 is TG however the individual has a        common variant at c.2077 (G&gt;A) and the TA insertion is on that        allele. Should I call it c.2076_2077dupTA since I know that is the        description of the change on that specific allele or should I call it        c.2077_2078insTA which would be the correct description based on the        more common sequence at that position.</em></p>    <div align="center">      <center>        <table border="1" width="59%">          <tbody>            <tr>              <td align="center" width="100%"><u><em>Summary;</em></u> the BRCA1                coding DNA reference sequence from position 2074_2080 is                ..CATGACA.. A frequent variant in the population is ..CAT<u><strong>A</strong></u>ACA.
                and the sequence found in the individual is ..CAT<u><strong>A</strong></u>                <u><strong>TA</strong></u> ACA.</td>            </tr>          </tbody>        </table>      </center>    </div>    <p><strong>Answer</strong><br>      The basic rule is to describe variants <em><strong>in relation to a          reference sequence</strong></em>. In this respect, the description      c.2076_2077dup (<em>c.2076_2077dupTA</em>) is not correct because the      reference sequence does not contain a TA dinucleotide at position      c.2076_2077 (it has TG). The description c.2077_2078insTA is also not      correct because the change c.2077G&gt;A is neglected and <em><strong>all          changes should be described</strong></em>. So the correct description      is c.2077delGinsATA (or c.2077delGinsATA).</p>    <p><em><strong>NOTE:</strong></em> in cases like the above, where frequent      variants are present at the site changed it is allowed to describe these      individually. c.[4G&gt;T; 5C&gt;G] in the first case, assuming either      c.[4G&gt;T] or c.[5C&gt;G] is a known frequent variant. c.[2077G&gt;A;      2077_2078insTA] in the second case with c.2077G&gt;A known as the frequent      variant. Of course it is essential in such cases that the variants reside      on one allele.</p>    <hr>    <p><a name="dupins"></a><em><strong>Question&nbsp;&nbsp; </strong>(Ron        Agatep, Toronto, Canada)<br>        Several groups have identified a duplication in the CDKN2A locus that        has been labeled in various ways. The mutation is a duplication of the        first 24 bp </em> </p>    <ul>      <li><font face="Courier">normal =&nbsp; ggcggcggggagcagc <u>atg</u> gag          ccG GCG GCG GGG AGC AGC<br>          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
          Met Glu Pro Ala Ala Gly Ser Ser&nbsp;<br>          &nbsp; ATG GAG CCt tcg gct<br>          &nbsp; Met Glu Pro Ser Ala<br>        </font></li>      <li><font face="Courier">variant = ggcggcggggagcagc <u>atg</u> gag ccG          GCG GCG GGG AGC AGC&nbsp;<br>          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
          Met Glu Pro Ala Ala Gly Ser Ser&nbsp;&nbsp;<br>          <br>          &nbsp; ATG GAG CC<b>G GCG GCG GGG AGC AGC ATG GAG CC</b>t tcg gct<br>          &nbsp; Met Glu Pro <b>Ala Ala Gly Ser Ser Met Glu Pro </b>Ser Ala</font></li>    </ul>    <p><i>The ATG translation initiation codon is underlined (translational        start). One group has described the mutation as 23ins24 is this correct?        My interpretation of your recent paper suggests I should name it        1_24dup. Could you provide me with the correct nomenclature ?.</i></p>    <p><b>Answer</b><br>      Correct is <b>c.9_32dup (p.Ala4_Pro11dup)</b> - the description c.1_24dup      (p.Met1_Ser8dup) seems correct but please note that for all descriptions      the <b>most 3' position</b> possible should be arbitrarily assigned to      have been changed (<i><a href="recs.html#repeat">see Recommendations</a></i>).
      c.23ins24 is not correct, first because the position of the insertion is      not clear (<a href="disc.html#ins"><i>see Discussion</i></a>), second      'ins24' does not indicate which sequence was inserted.&nbsp; </p>    <hr>    <p><a name="pdup"></a><em><strong>Question</strong><br>        How should I describe a change where ATCG-ATCGATCGATCG-A-GGGTCCC becomes        ATCG-ATCGATCGATCG-A-<u><b>ATCGATCGATCG-</b></u>GGGTCCC ?. The fact that        the inserted sequence (ATCGATCGATCG) is present in the original sequence        suggests it derives from a duplicative event. </em></p>    <p><b>Answer</b><br>      A correct description of the insertion is c.17_18ins5_16 (<i><a href="recs-DNA.html#ins">see
          Recommendations</a></i>). A description using 'dup' is not correct      since by definition a duplication is a sequence change where a copy of one      or more nucleotides are <em><strong>inserted directly 3'-flanking of the          original copy</strong> </em>(<a href="standards.html#define"><i>see          Standards</i></a>)<em>.</em> Still, the description given makes it      clear that the sequence inserted between nucleotides c.17 and c.18 is      probably derived from nearby, i.e. position c.5_16, and thus likely      derived from a duplicative event. </p>    <hr>    <p><a name="most3"></a><em><strong>Question</strong><br>        The 3' end of intron 8 of the CFTR gene contains a variable sequence;        IVS8(TG)mTn. The CFTR genomic reference sequence of the end of intron 8        is ...TGTGTGTGTGTTTTTTTAACAG[..exon9..], with a tract of (TG)11 and T7.        When we describe this sequence variation as c.1210-14(TG)9-13(T)5-9 and        that of the IVS8Tn as c.1210-6(T)5-9, are we right? Is the description        of a T5 tract variant as c.1210-14(TG)12T5 correct ?. </em></p>    <p><b>Answer</b>&nbsp;&nbsp; (<a href="recs-DNA.html#var"><i>see Repeated          sequences</i></a>)<br>      A difficult case; please note that following current recommendations it is      not a TG11 but a GT11 variant (<a href="recs.html#repeat"><i>see          Recommendations</i></a>), <em><b>overlapping one T-nucleotide</b></em>      with the T7 stretch. However, to prevent confusion it is probably best to      use in this exceptional case TG11.<br>      The correct description depends on the <em><b>reference sequence used</b></em>.      Assuming this reference sequence is as described, i.e. TG11 followed by      T7, the TG11 stretch is located at c.1210-34_1210-13 and T7 stretch at      c.1210-12_1210-6. A correct description of the variants is then      c.1210-34TG(9_13)T(4_8) (or c.1210-34_1201-33(9_13)T(4_8)). c.1210-34      because the <b>variable tract starts at that position</b>.<br>      When only the T stretch is described the correct description is      c.1210-12T(5_9). A correct description of the T5 variant is c.1210-12T[5].<br>      <em><b>NOTE:</b> to indicate the <b>range, "_"</b> must be used and not        "-". </em></p>    <hr>    <p><a name="insname"></a><em><strong>Question</strong><br>        Is the description <b>NM_012345.3:c.123+45_123+51TSDinsL1.603bp</b>        acceptable (TSD = target site duplication, L1 indicates the nature of        the insert (L1, Alu or SVA) after "ins"; 603bp = the number of inserted        base pairs) ?. </em></p>    <p><b>Answer</b><br>      Following the current recommendations the description should be <b>NM_012345.3:c.123+45_123+51dupinsAB012345.3:g.393_1295</b>      (alternatively NM_012345.3:c.123+45_123+51dupins603). So use "<b>dup</b>"      (not "TSD") and leave out "bp" (not necessary). The insertion itself is      described as AB012345.3:g.393_1295, indicating that the inserted sequences      are nucleotides 393 to 1295 from GenBank file AB012345.3. Adding "(L1)" in      the description to indicate the nature of the inserted sequence is not      recommended, it might cause confusion. The "Remarks" column of the summary      sequence variant Table can be used for this annotation. </p>    <hr>    <p><a name="2alleles"></a><em><strong>Question</strong><br>        How should we, using the most current recommendations, <b>indicate a          change in one allele</b>. The notation we envisage should indicate        that the other allele has no change compared to the reference        sequence.&nbsp;For the unchanged allele "[?]" would not be appropriate        since it is not the case that allele 2 has an unknown variant; it simply        has change. The notation "c.[76A&gt;C]" without describing the second        allele would be misleading; not enough researchers would be familiar        enough with the nomenclature to know that this refers to only one of the        two alleles present. Would the description "c.[76A&gt;C];[]" be OK ?.</em><i>&nbsp;</i>    </p>    <p><b>Answer</b><br>      The character used to indicate 'no change' is the '=' (<a href="recs-DNA.html#2in"><i>see
          Recommendations</i></a>). The recommended description is thus <i><b>"c.[76A&gt;C];[=]"</b></i>.    </p>    <hr>    <p><em><a name="2unknown"></a><strong>Question&nbsp;&nbsp; </strong>(Andrew        Grimm, Coordinator RettBASE)<br>        When I come across cases where a person has two variants and it isn't        known whether or not they are on the same chromosome how should I        describe this ?.</em></p>    <p><b>Answer</b><br>      Although we do not recommend to <i><b>describe uncertainties</b></i>, in      this case it is clear that to prevent mistakes a recommendation is      required. Two changes in one allele should be described as c.[76A&gt;C;      91C&gt;G] and two changes on different alleles as c.[76A&gt;C];[91C&gt;G].      When it is <b><i>not clear</i></b> whether the changes are on the same or      on different alleles the recommendation is to describe this using the      format <b><i>c.[76A&gt;C(;)91C&gt;G]</i></b> (<em><a href="recs-DNA.html#2in">see
          Recommendations</a></em>). </p>    <hr>    <p><em><a name="2in2"></a><strong>Question&nbsp;&nbsp; </strong>(Nancy        Carson, Ottawa, Canada)<br>        The recommendations for mutation nomenclature give guidelines on the        proper nomenclature for recessive diseases where there are two mutations        identified in one gene. I have a <b>patient</b> with hearing loss who        has a <b>mutation in GJB2 </b>(c.35delG)<b> and a mutation in GJB6</b>        (c.689_690insT). Any suggestions on how I should write this?</em></p>    <p><b>Answer</b><br>      The recommendation is to use the format <b>GJB2:c.[35delG]        GJB6:c.[689_690insT]</b> (<a href="disc.html#2in2genes">see Discussion</a>).
      This format prevents confusion regarding the reference sequence used (i.e.      <i>"GJB2:"</i>) and combines this with the normal format to describe      variants in different alleles. Using the format given it is of course      still essential to describe the reference sequence used (GenBank file with      version number). Another format, coping with this directly, is to describe      the variants as <b>NM_004004.2:c.[35delG] NM_006783.1:c.[689_690insT]</b>,      i.e. using the Genbank reference sequences in stead of the <a href="http://www.genenames.org/guidelines.html">HGNC
        Gene Symbol</a>. </p>    <hr>    <p><em><a name="fe-male"></a><strong>Question</strong><br>        I study a gene located on the X-chromosome. How should I describe the        variants detected in males and females?</em></p>    <p><b>Answer</b><br>      In <b><i>females</i></b> the description is straightforward, like <i><b>"c.[76A&gt;C];[=]"</b></i>.      In <b><i>males</i></b> there is no second allele (X-chromosome) which can      be described as <i><b>c.[76A&gt;C];[0]"</b></i> (<i><a href="recs-DNA.html#2in">see
          Recommendations</a></i>). </p>    <hr>    <p><em><a name="mosaic"></a><strong>Question</strong><br>        Detailed analysis of a DMD patient showed that it was a <b>mosaic case</b>;        consequently two different nucleotides were found at one position, a G        and a C (a G is the normal sequence). How should I describe this?</em></p>    <p><b>Answer</b><br>      <i><img src="new.gif" border="0" height="12" width="31"><b>Mosaic cases</b></i>,      i.e. two different nucleotides found at one position on one allele      (chromosome) should be <i><b>described as c.[83G=/&gt;C]</b></i> (<i><a href="recs.html#2in1">see
          Recommendations</a></i>).<br>    </p>    <p><i><b>NOTE:</b></i> this recommendation was changed (<i>Aug.2010</i> and      Nov.2015).      Initially the suggestion was to describe mosaic cases using c.[=,      83G&gt;C]. This recommendation was changed to follow standards from the      ISCN (<i>International System for Human Cytogenetic Nomenclature</i>, <a        href="changes.html"><i>see
          Recent changes</i></a>) and from c.=/83G&gt;C after acceptance of <a        href="file:///Users/Boomheide/Files/WWW/HGVS/accepted001.html">proposal
        SVD-WG001.</a> </p>    <hr>    <p><em><a name="promSNP"></a><strong>Question&nbsp;&nbsp; </strong>(Harriet        Meyer, JAMA-archives.org)<br>        The subject of promoter polymorphisms has come up, and I would be        grateful for your recommendation of how these should be described.</em></p>    <p><b>Answer</b><br>      For <i><b>variants in the promoter region</b></i> it is recommended to      describe these in relation to a <b><i>genomic reference sequence</i></b>      (like L01538.1:g.1407C&gt;T). Describing a promoter variant in relation to      a coding DNA reference sequence is possible and should be in relation to      the A of the ATG initiation codon, counting backwards to the variant      nucleotide; in the example given c.-401C&gt;T indicating a change of the C      401 nucleotides upstream of the ATG (in the promoter), to an T. To be      unequivocal, next to the coding DNA reference sequence (to identify the A      of the ATG) one should also mention the genomic reference sequence used      (to identify the C at -401) or include upstream sequences in the coding      DNA reference sequence (<a href="refseq.html#updown">see <i>Discussion</i></a>).
This
      would make it rather complex - one has to retrieve <i><b>two sequence          files</b></i>. Consequently, it would be much easier to describe the      variant directly in relation to the genomic reference sequence. A format      which one could use is "<b><i>L01538.1:g.1407C&gt;T (at -401 of the ATG)</i></b>".<br>      Please note that it is not correct to provide descriptions in relation to      the start site of the mRNA. There is often a debate as to where the RNA      exactly starts and one should not describe DNA variants in relation to      such a 'variable' site (<em><a href="disc.html#untrans">see Discussion</a></em>).
      Of course it is acceptable that the authors mention, between brackets, the      approximate position of the change in relation to the promoter.</p>    <hr>    <h3><a name="RNA"></a>RNA Level Changes</h3>    <hr>    <p>&nbsp; </p>    <hr>    <h3><a name="protein"></a>Protein Level Changes</h3>    <hr>    <p><em><strong>Question</strong><br>        How should a mutation in the 5'UTR be described that gives rise to a new        translation initiation site ?</em></p>    <p><strong>Answer</strong><br>      Description at the DNA-level should be e.g. c.-23A&gt;T&nbsp; <em>(changing
        -25 c<u>a<strong>G</strong>g</u>gt -19 to c<u>a<strong>T</strong>g</u>gt,
        creating a new ATG-triplet). </em>Description at the RNA-level should      be like r.-23a&gt;u and description at the protein level could be like      p.Met1extMet-8 (or p.M1extM-8, <i><a href="recs-prot.html#sub">see          Recommendation protein level</a></i>). This indicates that due to a      variant the protein sequence becomes <i><b>ext</b></i>ended N-terminally      by the addition of 8 new amino acids. Note that descriptions on RNA and      protein level should only be given when this was experimentally verified;      if not, changes should be placed between brackets to indicate that it is a      prediction only. </p>    <hr>    <p><em><a name="protref"></a><strong>Question&nbsp; </strong>(Dean J.        Danner, Atlanta, USA)<br>        We are characterizing mutations in nuclear encoded proteins that        function in mitochondria. The problem is in proteins that have amino        terminal mitochondrial signal peptides. The current rules for proteins        say to start numbering with the initiating methionine. However, the        functional protein has this target peptide removed and therefore many        investigators begin numbering at the amino acid residue of the <b>mature
          protein</b>. Mutations that result in changes in the targeting peptide        suggest that numbering should begin with the Met-1. An alternative would        be to give the targeting peptide negative numbers as in the nucleic        acids upstream of the transcriptional start site. It would be helpful to        have some rules for consistency in the field.</em></p>    <p><strong>Answer</strong><br>      As already suggested in your question, protein reference sequences should      always represent the <i><b>complete primary translation product</b></i>,      not a processed mature or functional protein (<a href="recs.html"><i>see          Recommendations</i></a>).&nbsp; </p>    <hr>    <p><em><a name="protDNA"></a><strong>Question&nbsp; </strong>(Sven Arnold,        Austria)<br>        There are several examples you give where changes affecting a series of        amino acids are described using the most 3' amino acid. Does this also        apply when it is known exactly which amino acid is affected? Example;        the sequence ATGTCAAGCTCT codes for MetSerSerSer. An insertion of AGT        (c.9_10insAGT) gives ATGTCAAGCAGTTCT, coding for MetSerSerSerSer.        Looking at the protein sequence you would describe the change as        p.Ser3dup. Knowing the nucleotide changes, it would be accurately        described as p.Ser2_Ser3insSer. My question is, do we describe the        protein change as it appears, or do we try and describe it according to        the (known) underlying DNA change?</em></p>    <p><strong>Answer</strong><br>      Descriptions at protein level should describe the changes observed on      protein level - one should <i><b>not try to incorporate knowledge          regarding the change at DNA-level&nbsp;</b></i>(<a href="recs-prot.html#protonly"><i>see
          Recommendations</i></a>).&nbsp;As a consequence, the amino acid change      described may be caused by a change which at DNA level lies several      nucleotides upstream, like in the example you give. Another example is      that where a frame shift deletion at DNA level does not immediately affect      the protein sequence. </p>    <hr>    <p><em><a name="frameshift"></a><strong>Question&nbsp;</strong><br>        When a protein description does not contain "<strong>fs</strong>" (frame        shift) does this mean there is no frame shift?</em></p>    <p><strong>Answer</strong><br>      By definition <em><strong>frame shifts </strong></em>are a <em><strong>special
          type of amino acid deletion/insertion</strong></em> replacing the      normal C-terminal sequence with one encoded by another reading frame      (specified <em><small><font color="#FF0000"><strong>2013-03-16</strong></font></small></em>,      see <a href="recs-prot.html#indelp"><em>Describing protein variants</em></a>).
Descriptions
      at protein level describe the consequences of a change on the protein      irrespective of the changes at DNA or protein level. Translating back from      protein to DNA (or RNA) is therefore difficult and usually only works for      simple cases like substitutions. Examples of what one might call frame      shifts that can not ne seen from the protein description include; </p>    <ul>      <li>deletions at DNA level that lead to an <em><strong>immediate stop            codon</strong></em>, e.g. c.4delC in 5'-AAT <u>C</u>TG AGC-3' gives        a predicted protein change p.Cys2Ter (C4*)</li>      <li>variants at DNA level that introduce a frame shift which is followed        by <em><strong>another variant on the same allele shifting the reading            frame back</strong></em> to normal (before the shifted frame        encountered a translation termination codon) are described as a        deletion/insertion (<em><strong>delins</strong></em>)</li>      <li><em><strong>no-stop changes</strong></em> (see <a href="recs-prot.html#nonX"><em>Describing
            protein variants</em></a>), i.e.variants affecting the translation        termination codon (Ter/*1) introducing a new downstream termination        codon extending the C-terminus of the encoded protein, are described as        extension.<br>        <em><strong>NOTE:</strong></em>&nbsp; since technically there is no        reading frame after the translation termination codon there is also no        shifted frame.</li>    </ul>    <hr>    <p><em><a name="nostop"></a><strong>Question&nbsp; </strong>(Giampaolo        Trivellin, London)<br>        How should we describe the consequences of a duplication of a G at DNA        level that causes a frameshift at the protein level where the shifted        frame does not encounter a new stop codon. I was thinking to describe it        using the short description (e.g. p.Ile327fs), but we expect that the        protein is not formed since the aberrant RNA will be degraded lacking a        stop codon.</em></p>    <p><strong>Answer</strong><br>      The description p.(Ile327fs) can indeed be used (please note the brackects      to indicate RNA was not yet analysed). It circumvents however the problem      that the current recommendations did not yet indicate how to describe a      frame shift that does not encounter a stop codon. The recommendation is <strong><em>to
          describe this using "fs*?"</em></strong>, so p.(Ile327Argfs*?) (<a href="recs-prot.html#nonX"><i>see
          Recommendations</i></a>). The "?" indicates uncertainty, in this case      that the position of the stop codon is not known. When you have analysed      RNA and it is indeed undetectable (degraded) the RNA description would be      r.0 (no RNA) and the protein description p.0 (no protein).</p>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">Homepage</a>      | <a href="checklist.html">Check-list</a> | <a href="standards.html">Symbols,
        codons, etc.</a> |<br>      | <b>Recommendations:</b>&nbsp; <a href="recs.html">general</a>,&nbsp;<a        href="recs-DNA.html">DNA</a>,&nbsp;
      <a href="recs-RNA.html">RNA</a>,&nbsp; <a href="recs-prot.html">protein</a>,      <a href="uncertain.html">uncertain</a> |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="standards.html">Symbols,
        codons, etc.</a>| <a href="history.html">History</a> |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef</a>,&nbsp;
      <a href="examplesDNA.html">DNA</a>,&nbsp; <a href="examplesRNA.html">RNA</a>,&nbsp;
      <a href="examplesAA.html">protein</a> | </p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright
                  ©</a> </font><font size="1">HGVS 2007 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>