<html>
<!--
	Author: 	Dr. Johan T. den Dunnen
	Date: 	17-11-2002	
	URL: 	recs.html
-->

<head>
<title>HGVS recommendations: nomenclature for the description of sequence variants</title>
<style type="text/css">

<!--  

h2 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #ff9900;

}

h3 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h4 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h5 {
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #ffffff;

}

.norm {

 font-weight : normal;

 }
 -->
</style>
</head>

<body text="#000000" bgcolor="#FFFFFF" link="#0000FF" vlink="#9999ff">

<table width="100%" border="0" cellspacing="0" cellpadding="10" bgcolor="#CCFF99">
  <tr>
    <td width="25%" valign="top" bgcolor="#FFFFFF"><a href="http://www.HGVS.org"><img
    src="small.gif" border="0" width="155" height="94"></a></td>
    <td width="75%" valign="CENTER" bgcolor="#FFFFFF" bordercolor="#FFFFFF"><h3 align="center">Recommendations
    for the description of sequence variants </h3>
    </td>
  </tr>
</table>

<table width="100%" border="0" cellspacing="0" cellpadding="10">
  <tr>
    <td width="100%" valign="BOTTOM" bgcolor="#0099cc"><h4 align="center"><font size="3"
    color="#FFFFFF"><i><br>
    Last modified July 8, 2013</i></font></h4>
    </td>
  </tr>
</table>

<p align="center"><i><font face="Times New Roman">Since references to WWW-sites are not
yet acknowledged as citations, please mention <a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den Dunnen JT
and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when referring to these pages.</font></i></p>

<hr>

<h2>Contents</h2>

<ul>
  <li><a HREF="#intro"><b>Introduction</b></a></li>
  <li><b>Recommendations</b> <ul>
      <li><a href="#general">general</a></li>
      <li><a href="recs-DNA.html">DNA level</a></li>
      <li><a href="recs-RNA.html">RNA level</a></li>
      <li><a href="recs-prot.html">protein level</a></li>
    </ul>
  </li>
  <li><b>Explanations / examples</b> <ul>
      <li><a href="quickref.html">Quick Reference</a></li>
      <li><a HREF="examplesDNA.html">changes at DNA-level</a></li>
      <li><a HREF="examplesRNA.html">changes at RNA-level</a></li>
      <li><a HREF="examplesAA.html">changes at protein-level</a></li>
    </ul>
  </li>
</ul>

<hr>

<h2><a NAME="intro"></a>Introduction</h2>

<hr>

<p>Discussions regarding the uniform and unequivocal description of sequence variants in
DNA and protein sequences (mutations, polymorphisms) were initiated by two papers
published in 1993; <a
href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8401532&amp;dopt=Abstract">Beaudet
AL &amp; Tsui LC</a> and <a
href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=8352285&amp;dopt=Abstract">Beutler
E</a>. The original suggestions presented were widely discussed, modified, extended and
ultimately resulted in nomenclature recommendations that have been largely accepted and
are applied world-wide (<a href="history.html"><i>see History</i></a>).</p>

<p>Current rules (<a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den Dunnen,
JT and Antonarakis, SE [2000]</a>) however do not extensively cover all types of variants
and the more complex changes. These pages will list, based on the last publication, the
existing nomenclature recommendations as well as the most recent suggestions (<i>in
italics and marked <a href="#del"><img border="0" src="new.gif" width="31" height="12"></a></i>).
More details regarding the&nbsp; latest additions can be found at the <a HREF="disc.html"><i>Discussion
page</i></a>.&nbsp;These pages can be used as a guide to describe any sequence variant
identified and should help to get a uniformly accepted standard.</p>

<p>Discussions regarding the advantages and disadvantages of the recommendations made are
necessary in order to continuously improve the system. What is listed on these pages
represents the current consensus of the discussions. We invite investigators to
communicate to us regarding the recommendations as well as to send us complicated cases
not yet covered, with a suggestion of how to describe these (E-mail to: <a
href="mailto:J.T.den_Dunnen%20@%20LUMC.nl,%20Stylianos.Antonarakis%20@%20medecine.unige.ch">J.T.den_Dunnen
@ LUMC.nl and Stylianos.Antonarakis @ medecine.unige.ch</a>).&nbsp;<br>
</p>

<h3>Mutation and polymorphism</h3>

<p>In some disciplines the term<b><i> </i>&quot;mutation&quot;</b> is used to indicate
&quot;<em>a change</em>&quot; while in other disciplines it is used to indicate &quot;<em>a
disease-causing change</em>&quot;. Similarly, the term <b>&quot;polymorphism&quot; </b>is
used both to indicate &quot;<em>a non disease-causing change</em>&quot; or &quot;<em>a
change found at a frequency of 1% or higher in the population</em>&quot;. To prevent this
confusion we do not use the terms mutation and polymorphism (including SNP or Single
Nucleotide Polymorphism) but use neutral terms like <strong>&quot;sequence variant&quot;</strong>,
<strong>&quot;alteration&quot; </strong>and <strong>&quot;allelic variant&quot;</strong>. <a
HREF="http://www3.interscience.wiley.com/cgi-bin/issuetoc?ID=88512035">Human Mutation</a>
(Vol. 19 ( 1) of 2002) contains several contributions discussing these issues as well as
the fact that the term <b>&quot;mutation&quot;</b> has developed a negative connotation
(see <a
href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=11754097&amp;query_hl=5&amp;itool=pubmed_DocSum">Cotton
RGH - p.2</a>, <a
href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=11754105&amp;query_hl=10&amp;itool=pubmed_docsum">Condit
CM et al. - p.69</a> and <a
href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=11754106&amp;query_hl=8&amp;itool=pubmed_docsum">Marshall
JH - p.76</a>).<br>
</p>

<h3><img border="0" src="new.gif" width="31" height="12">Pathogenic</h3>

<p>Another confusing term used frequently is &quot;a <strong>pathogenic</strong>
variant&quot;. While a non-expert concludes the variant described &quot;<em>causes disease</em>&quot;,
the expert probably means &quot;<em>causes disease when in a specific context</em>&quot; 

<ul>
  <li>causes disease when found in a male (X-linked recessive disorder)</li>
  <li>causes disease when combined with a similar chance in the other allele (autosomal
    recessive)</li>
  <li>causes disease when inherited from the father (imprinted)</li>
</ul>

<p>To prevent confusion it therefore seems best not to use the term &quot;<em>pathogenic</em>&quot;.
A good alternative seems a neutral term like &quot;<strong>affects function</strong>&quot;.
In fact this properly describes what one actually means, the variant affects the normal
function of the gene/protein (in whatever way). This also solves the issue of what term to
use for non-disease phenotypes like skin/hair/eye colour or blood group. In such cases it
is problematic to choose the phenotype to call &quot;<em>norma</em>l&quot; or &quot;<em>pathogenic</em>&quot;.
Using &quot;<strong>affects function</strong>&quot; is clear and effective. To classify
variants people use most frequently 5 categories. Based on affects function these could
be; <em>affects funtcion,&nbsp; probably affects function,&nbsp; unknown,&nbsp; probably
does not affect function (or probably no functional effect),&nbsp; does not affect
function (no functional effect)</em>. Variants for which a functional effect is unknown
can together be called &quot;<strong>variants of unknown significance</strong>&quot; (<strong>VUS</strong>).</p>

<hr>

<h2><a NAME="general"></a>General recommendations</h2>

<p><em>(suggestions extending the <a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">published</a>
recommendations in italics)</em></p>

<hr>

<p><a NAME="version"></a>The most important rule is that all variants should be described
at the most basic level, i.e. the DNA level. Descriptions should always be in relation to
a <b>reference sequence</b>, either a <b>genomic</b> or a <b>coding DNA</b> reference
sequence. Discussions on which type of reference sequence to prefer, genomic or coding
DNA, have been lively. Although theoretically a genomic reference sequence seems best, in
practice a coding DNA reference sequence is preferred (see <a href="refseq.html"><i>Reference
Sequence discussions</i></a>). 

<ul>
  <li><i><b>describing genes / proteins, only official <a
    href="http://www.genenames.org/guidelines.html">HGNC gene symbols</a> should be used<br>
    </b></i>all changes reported must be <b>described at DNA-level</b>. When descriptions at
    RNA or protein level are given in the text (including TITLE and ABSTRACT), on first
    mention, a format like &quot;c.78G&gt;C (p.Trp26Cys)&quot; should be used. </li>
  <li>when several changes are described in one manuscript, a tabular listings should be
    provided summarizing the findings, using separate columns for DNA, RNA and protein and
    clearly indicating whether the changes were <strong>experimentally determined</strong> or
    only <b>theoretically deduced</b>. When changes in patients with a recessive disease are
    described, the listing should make clear in which combination the changes were found. An
    additional column can be used to mention additional findings and to make remarks.</li>
  <li><a NAME="prefix"></a>to avoid confusion in the description of a variant it should be
    preceded by a letter indicating the type of reference sequence used. Several different
    reference sequences can be used (<i><a href="figure_refseq.html">see Figure</a></i>);<ul>
      <li>&quot;c.&quot; for a <strong>coding DNA</strong> sequence (like&nbsp; c.76A&gt;T)</li>
      <li>&quot;g.&quot; for a <strong>genomic</strong> sequence (like g.476A&gt;T)</li>
      <li>&quot;m.&quot; for a <strong>mitochondrial</strong> sequence (like m.8993T&gt;C, <i><a
        href="refseq.html#mtDNA">see Reference Sequence</a></i>) </li>
      <li><i><img border="0" src="new.gif" width="31" height="12"></i><em>&quot;n&quot; for a <strong>non-coding
        RNA</strong> reference sequence (gene producing an RNA transcript but not a protein)</em></li>
      <li>&quot;r.&quot; for an <strong>RNA</strong> sequence (like r.76a&gt;u)</li>
      <li>&quot;p.&quot; for a <strong>protein</strong> sequence (like&nbsp; p.Lys76Asn)</li>
    </ul>
  </li>
  <li><i><img border="0" src="new.gif" width="31" height="12">the DNA reference sequence used
    should preferably be</i> a LRG (Locus Reference Genomic sequence, <a
    href="refseq.html#general"><i>see Reference Sequence</i></a>).<i> from the <a
    href="http://www.ncbi.nlm.nih.gov/RefSeq/">RefSeq database</a>, listing both database
    accession and version number (like NM_004006.2) </i><ul>
      <li><a NAME="indicator"></a>within one document only one DNA reference sequence should be
        used. When variants in more than one sequence (gene) are described, any confusion should
        be prevented by including a unique indicator&nbsp; in the description. Indicator and
        sequence description should be separated by a colon (&quot;:&quot;) like in
        NM_004006.1:c.3G&gt;T or GJB2:c.76A&gt;C.<br>
        <i><b>NOTE:</b></i> this format is especially important for unequivocal descriptions of
        SNP's (<a HREF="disc.html#SNP"><i>see Discussion</i></a>). <a href="#del"><i><img
        border="0" src="new.gif" width="31" height="12">When both HGNC-approved gene symbol and
        database accession.version number are indicated this should be done using the format
        NM_004006.1(DMD):c.3G&gt;T&nbsp;</i> (</a><a HREF="disc.html#indicator"><i>see Discussion</i></a>)<i>.</i></li>
      <li><i><img border="0" src="new.gif" width="31" height="12">the coding DNA reference
        sequence used should represent the major and largest transcript of the gene. Alternatively
        spliced exons (5'-first, internal or 3'-terminal) derived from <b>within the gene</b> can
        be numbered as for intronic sequences. Variants in transcripts initiating or terminating <b>outside
        this region</b> can be described as upstream / downstream sequences (<a href="refseq.html">see
        Reference Sequence discussions</a>).&nbsp;</i></li>
      <li>protein reference sequences should represent the primary translation product, not a
        processed mature protein (<a href="FAQ.html#protref"><i>see FAQ</i></a>).</li>
      <li><i><img border="0" src="new.gif" width="31" height="12">when changes start or end in
        another sequence (gene), e.g. for large deletions, the nucleotide numbering for that end
        is based on the nucleotide numbering of that sequence (like c.827_NM_004004.3:c.235del).
        When the endpoint occurs on the opposite, non-transcribed strand (anti-sense strand),
        an&nbsp; &quot;o&quot; precedes the reference identifier (like
        c.827_oNM_004004.3:c.233del, <a HREF="disc.html#fusion">see Discussion</a>).</i></li>
      <li><a NAME="twogenes"></a><i><img border="0" src="new.gif" width="31" height="12">when a
        variant affectsmore then one gene, to prevent confusion, the variant should be described
        in relation to all genes affected.&nbsp;</i></li>
    </ul>
  </li>
  <li>for a clear distinction, descriptions at DNA, RNA and protein level are unique;<ul>
      <li><u>DNA-level</u><br>
        in capitals, starting with a number referring to the first nucleotide affected (like
        c.76A&gt;T or g.476A&gt;T)</li>
      <li><u>RNA-level</u><br>
        in lower-case, starting with a number referring to the first nucleotide affected (like
        r.76a&gt;u)</li>
      <li><u>protein level</u><br>
        in capitals, starting with a letter referring to first the amino acid affected (like
        p.Lys76Asn)</li>
    </ul>
  </li>
  <li><a NAME="ntinumber"></a><b>nucleotide numbering</b>&nbsp; <i>(for details and examples <a
    href="refseq.html">see Reference Sequence discussions</a>)</i> <ul>
      <li><u>coding DNA Reference Sequence</u>&nbsp; (<i><a href="figure_refseq.html">see Figure</a>
        and <a href="standards.html#number">Numbering</a></i>) <ul>
          <li>there is no nucleotide 0</li>
          <li>nucleotide 1 is the A of the ATG-translation initiation codon</li>
          <li>the nucleotide <b>5' of the ATG-translation initiation</b> codon is -1, the previous -2,
            etc.<br>
            <i><img border="0" src="new.gif" width="31" height="12"></i><em><strong>NOTE:</strong> den
            Dunnen&amp;Antonarakis (<a
            href="http://dx.doi.org/10.1002/(SICI)1098-1004(200001)15:1%3c7::AID-HUMU4%3e3.0.CO;2-N">Hum.Mut.
            15: 7-12</a>) write &quot;For <strong>genomic DNA</strong> and cDNA sequences, the A of
            the ATG of the initiator Methionine codon is denoted nucleotide +1&quot;. This is an
            error, correct is; &quot;In coding DNA reference sequences, the A of the ATG of the
            initiator Methionine codon is denoted nucleotide +1&quot;.</em></li>
          <li><i><img border="0" src="new.gif" width="31" height="12">the nucleotide <b>3' of the
            translation stop codon</b> is *1, the next *2, etc.</i></li>
          <li>intronic nucleotides<ul>
              <li><u>beginning of the intron;</u> the number of the last nucleotide of the preceding exon,
                a plus sign and the position in the intron, like c.77+1G, c.77+2T, etc.</li>
              <li><u>end of the intron;</u> the number of the first nucleotide of the following exon, a
                minus sign and the position upstream in the intron, like c.78-1G.</li>
              <li>in the <u>middle of the intron,</u> numbering changes from &quot;c.77+..&quot; to
                &quot;c.78-..&quot;; for introns with an uneven number of nucleotides the central
                nucleotide is the last described with a &quot;+&quot; <i>(<a href="refseq.html#IVS">see
                Reference Sequence discussions</a>)</i></li>
            </ul>
          </li>
        </ul>
      </li>
      <li><u>genomic Reference Sequence</u>&nbsp; (<i><a href="figure_refseq.html">see Figure</a></i>)
        <ul>
          <li>nucleotide numbering is purely arbitrary and starts with 1 at the first nucleotide of
            the database reference file<br>
            <em><img border="0" src="new.gif" width="29" height="12"><strong>NOTE:</strong> in den
            Dunnen&amp;Antonarakis (<a
            href="http://dx.doi.org/10.1002/(SICI)1098-1004(200001)15:1%3c7::AID-HUMU4%3e3.0.CO;2-N">Hum.Mut.
            15: 7-12</a>) write &quot;For <strong>genomic DNA</strong> and cDNA sequences, the A of
            the ATG of the initiator Methionine codon is denoted nucleotide +1&quot;. This is an
            error, correct is; &quot;In genomic reference sequences, the first nucleotide is
            nucleotide 1&quot;.</em></li>
          <li>no +, - or other signs are used</li>
          <li>the sequence should include all nucleotides covering the sequence (gene) of interest and
            should start well 5' of the promoter of a gene</li>
          <li>when the complete genomic sequence is not known, a coding DNA reference sequence should
            be used</li>
        </ul>
      </li>
    </ul>
  </li>
  <li><b>specific changes</b> <ul>
      <li>&quot;&gt;&quot; indicates a <b><i>substitution</i></b> at DNA level (like&nbsp;
        c.76A&gt;T)</li>
      <li>&quot;_&quot; (underscore) indicates a <i><b>range</b></i> of affected residues,
        separating the first and last residue affected (like c.76_78delACT, <em><a
        HREF="disc.html#range">see Discussion</a></em>)</li>
      <li>&quot;del&quot; indicates a <i><b>deletion</b></i> (like&nbsp; c.76delA)</li>
      <li>&quot;dup&quot; indicates a <i><b>duplication</b></i> (like&nbsp; c.76dupA); <i><img
        border="0" src="new.gif" width="31" height="12">duplicating insertions are described as <b>duplications</b>,
        not as insertions; ACTTTGTGCC to ACTTTGTG<u><b>G</b></u>CC is described as c.8dupG (not as
        c.8_9insG, <a HREF="disc.html#dupins">see Discussion</a>)</i></li>
      <li>&quot;ins&quot; indicates a <i><b>insertion</b></i> (like&nbsp; c.76_77insG)</li>
      <li><i><img border="0" src="new.gif" width="31" height="12"></i>&quot;inv&quot; indicates an
        <i><b>inversion</b></i> (like&nbsp; c.76_83inv)</li>
      <li><i><img border="0" src="new.gif" width="31" height="12">&quot;con&quot; indicates a <b>conversion</b>
        (like&nbsp;c.123_678conNM_004006.1:c.123_678, <a href="recs-DNA.html#con">see
        Recommendations</a>)</i></li>
      <li>&quot;[]&quot; indicates an allele (like&nbsp;c.[76A&gt;T], <a href="#2in1"><i>see
        Recommendations</i></a>)</li>
      <li>&nbsp;<a NAME="uncertain"></a><i><a href="#del"><img border="0" src="new.gif" width="31"
        height="12"></a>&quot;()&quot; is used when the exact position of a change is not known,
        the range of the uncertainty is described as precisely as possible and listed between
        brackets (like c.(67_70)insG, <a href="uncertain.html">see Uncertainties</a>)</i></li>
    </ul>
  </li>
  <li><b>miscellaneous&nbsp;</b> <ul>
      <li><a NAME="repeat"></a>for all descriptions the <b>most 3' position</b> possible is
        arbitrarily assigned to have been changed, this is important especially in single residue
        (nucleotide or amino acid) stretches or tandem repeats (<a href="recs-DNA.html#del"><i>see
        Recommendations</i></a>, <i><a href="disc.html#loss">see Discussion</a></i>)</li>
      <li><i>variability in the number of repeated sequences (e.g. ATGCGA<b>TGTGTG</b>CC) are
        described as c.123+74TG(3_6) (<a href="recs-DNA.html#var">see Recommendations</a>)</i></li>
      <li><i><img border="0" src="new.gif" width="31" height="12">triplications, quadruplications,
        etc. are described as alleles of variable short sequence repeats; c.87_93[3] describes a
        triplication of the 7 nucleotides from coding DNA position 87 to 93 (not as c.87_93tri, <a
        HREF="recs-DNA.html#tri">see Discussion</a>)</i></li>
      <li><a NAME="2in1"></a><i><b>two sequence variants in one individual</b></i> <ul>
          <li><u>two sequence changes in different alleles</u> (e.g. for recessive diseases) are
            listed between square brackets, separated by a &quot;;&quot;-character;
            c.[76A&gt;C];[87delG] (<a HREF="disc.html#2in2"><i>see Discussion</i></a>)</li>
          <li><u>two sequence variants in one allele</u> are listed between square brackets, separated
            by a &quot;;&quot;-character; c.[76A&gt;C; 83G&gt;C] (<em><a HREF="disc.html#2in1">see
            Discussion</a></em>)</li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><u>two sequence changes with
            alleles unknown</u> are listed between square brackets, separated by &quot;(;)&quot;;
            c.[76A&gt;C(;)83G&gt;C] (<a href="FAQ.html#2unknown">see FAQ</a>)</i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12">descriptions of sequence changes
            in different genes (e.g. for recessive diseases) are listed between square brackets,
            separated by a &quot;;&quot;-character and include a reference to the sequence (gene)
            changed; DMD:c.[76A&gt;C];GJB:c.[87delG] (<a HREF="disc.html#2in2genes">see Discussion</a>)</i></li>
        </ul>
      </li>
      <li><i><img border="0" src="new.gif" width="31" height="12"><u><b>mosaic cases:</b> two
        different nucleotides at one position in one allele</u> are listed between square
        brackets, separated by a &quot;/&quot;-character; c.[=/83G&gt;C] (<a href="changes.html">see
        Recent changes</a>)</i></li>
      <li><i><img border="0" src="new.gif" width="31" height="12"><u><b>chimeric cases:</b> two
        different nucleotides at one position in one allele</u> are listed between square
        brackets, separated by a &quot;//&quot;-character; c.[=//83G&gt;C] (<a href="changes.html">see
        Recent changes</a>)</i></li>
    </ul>
  </li>
  <li>a unique identifier should be assigned to each variant; when available, the
    OMIM-identifier can be used, otherwise database curators should assign a unique
    identifier.</li>
</ul>

<h3>Detailed recommendations</h3>

<ul>
  <li><a href="recs-DNA.html">DNA level</a> </li>
  <li><a href="recs-RNA.html">RNA level</a> </li>
  <li><a href="recs-prot.html">protein level</a> </li>
</ul>

<hr>

<p align="center">| <a HREF="#top">Top of page</a> | <a HREF="index.html">Homepage</a> | <a
href="checklist.html">Check-list</a> | <a href="standards.html">Symbols, codons, etc.</a>
|<br>
| <b>Recommendations:</b>&nbsp; <a href="recs.html">general</a>,&nbsp;<a
href="recs-DNA.html">DNA</a>,&nbsp; <a href="recs-RNA.html">RNA</a>,&nbsp; <a
href="recs-prot.html">protein</a>, <a href="uncertain.html">uncertain</a> |<br>
| <a HREF="disc.html">Discussions</a> | <a HREF="FAQ.html">FAQ's</a> | <a
href="history.html">History</a> |<br>
| <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef</a>,&nbsp; <a
HREF="examplesDNA.html">DNA</a>,&nbsp; <a HREF="examplesRNA.html">RNA</a>,&nbsp; <a
HREF="examplesAA.html">protein</a> | </p>

<table width="100%" border="0" cellspacing="0" cellpadding="15" bgcolor="#ff9900">
  <tr>
    <td><p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"
    size="-5"><a href="http://www.HGVS.org/copyright.html">Copyright ©</a> </font><font
    size="1">HGVS 2010 All Rights Reserved<br>
    Website Created by Rania Horaitis, Nomenclature by J.T. Den Dunnen - <a
    href="http://www.HGVS.org/disclaimer.html">Disclaimer</a> </font></td>
  </tr>
</table>
</body>
</html>
