<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"><html>  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>HGVS recommendations: general, DNA level</title>    <style type="text/css"><!--  h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body style="             color: black; background-color: white;" link="#0000FF"    vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Recommendations for the description of DNA              sequence variants - v2.0 </h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified May 20, 2015</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i>Since references to WWW-sites are not yet acknowledged        as citations, please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den
          Dunnen JT and Antonarakis SE (2000). Hum.Mutat. 15: 7-12</a> when        referring to t</i>separated by at least one nucleotide<i>hese pages.</i></p>    <hr>    <h2>Contents</h2>    <ul>      <li><b>Recommendations</b>        <ul>          <li><a href="recs.html">general recommendations</a>            <ul>              <li><a href="#number">nucleotide numbering</a></li>            </ul>          </li>          <li><b>changes at DNA level</b>            <ul>              <li><a href="#DNA">general</a></li>              <li><a href="#sub">substitutions</a></li>              <li><a href="#del">deletions</a></li>              <li><a href="#dup">duplications</a></li>              <li><a href="#ins">insertions</a></li>              <li><a href="#indel">insertion/deletions&nbsp; (indels)</a></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="#inv">inversions</a></i></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="#con">conversions</a></i></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="#tra">translocations</a></i></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="#2in">two
                    or more changes in one chromosome</a>&nbsp; (incl.                  mosaicism, chimerism)</i></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="#var">repeated
                    sequences (incl. short sequence repeats)</a></i></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="uncertain.html">uncertainties</a></i></li>              <li><i><img src="new.gif" border="0" height="12" width="31"><a href="#com">complex</a></i></li>            </ul>          </li>          <li><a href="recs-RNA.html">changes at RNA level</a></li>          <li><a href="recs-prot.html">changes at protein level</a></li>        </ul>      </li>      <li><b>Explanations / examples</b>        <ul>          <li><a href="quickref.html">Quick Reference</a></li>          <li><a href="examplesDNA.html">changes at DNA level</a></li>          <li><a href="examplesRNA.html">changes at RNA level</a></li>          <li><a href="examplesAA.html">changes at protein level</a></li>        </ul>      </li>    </ul>    <hr>    <h2><a name="DNA"></a>DNA level</h2>    <p><em>(suggestions extending the <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">published</a>        recommendations are </em><i><img src="new.gif" border="0" height="12" width="31"></i><em>        in italics)</em></p>    <hr>    <ul>      <li><b>nucleotides<br>        </b>description of nucleotides at DNA level follows the recommendations        of the <a href="http://www.chem.qmul.ac.uk/iubmb/misc/naseq.html#500">IUPAC-IUBMB</a>.Nucleotides
        are designated by the bases, in upper case, A (adenine), C (cytosine), G        (guanine), T (thymidine), including those for uncertain nucleotides like        Y (pYrimidine) and R (puRine), (<i><a href="standards.html#nucleotide">see
            Standards</a></i>). </li>      <li><a name="number"></a><b>nucleotide numbering</b><i>&nbsp; (for details          and examples <a href="refseq.html">see Reference Sequence discussions</a>)</i>        <ul>          <li><u>coding DNA reference sequence</u>&nbsp; (<i><a href="refseq_figure.html">see
                Examples and Figure</a></i>)            <ul>              <li>there is no nucleotide 0 </li>              <li>nucleotide 1 is the A of the ATG-translation initiation codon              </li>              <li>the nucleotide <b>5' of the ATG-translation initiation</b>                codon is -1, the previous -2, etc.</li>              <li><i><img src="new.gif" border="0" height="12" width="31">the                  nucleotide <b>3' of the translation stop codon</b> is *1, the                  next *2, etc.</i></li>              <li><b>intronic nucleotides</b> (coding DNA reference sequence                only)                <ul>                  <li><u>beginning of the intron;</u> the number of the last                    nucleotide of the preceding exon, a plus sign and the                    position in the intron, like c.77+1G, c.77+2T, ....</li>                  <li><u>end of the intron;</u> the number of the first                    nucleotide of the following exon, a minus sign and the                    position upstream in the intron, like ..., c.78-2A, c.78-1G.</li>                  <li>in the <u>middle of the intron,</u> numbering changes                    from "c.77+.." to "c.78-.."; for introns with an uneven                    number of nucleotides the central nucleotide is the last                    described with a "+" <i>(<a href="refseq.html#IVS">see                        Discussion</a>)</i></li>                  <li><i><b>NOTE:</b></i> the format c.IVS1+1G and c.IVS1-2G                    should not be used <i>(<a href="refseq.html#IVS">see                        Discussion</a>)</i></li>                </ul>              </li>            </ul>          </li>          <li><u>genomic reference sequence</u>&nbsp; (<i><a href="refseq_figure.html">see
                Examples and Figure</a></i>)            <ul>              <li>nucleotide numbering starts with 1 at the first nucleotide of                the sequence<br>                <b><i>NOTE:</i></b> the sequence should include all nucleotides                covering the sequence (gene) of interest and should start well                5' of the promoter of a gene</li>              <li>no +, - or other signs are used</li>              <li>when the complete genomic sequence is not known, a coding DNA                reference sequence should be used</li>            </ul>          </li>          <li>for all descriptions the <b>most 3' position</b> possible is            arbitrarily assigned to have been changed (<i><a href="#except">see                Exception</a></i>) </li>        </ul>      </li>    </ul>    <hr>    <h3><a name="sub"></a><b>Substitutions</b></h3>    <p>A nucleotide <b>substitution</b> is a sequence change where one      nucleotide is replaced by one other nucleotide (<a href="standards.html#define"><i>see
          Standards - Definition</i></a>). Nucleotide substituions are described      using a <i><b>"&gt;"</b></i>-character (indicating "changes to").&nbsp;<br>      <b><i>NOTE:</i></b> changes involving two or more consecutive nucleotides      are described as deletion/insertions (indels, <i><a href="#indel">see          Deletion/insetions</a></i>). </p>    <ul>      <li>c.76A&gt;C denotes that at nucleotide 76 an A is changed to a C</li>      <li>c.-14G&gt;C denotes a G to C substitution 14 nucleotides 5' of the ATG        translation initiation codon</li>      <li>c.88+1G&gt;T denotes the G to T substitution at nucleotide +1 of an        intron (in the coding DNA positioned between nucleotides 88 and 89)</li>      <li>c.89-2A&gt;C denotes the A to C substitution at nucleotide -2 of an        intron (in the coding DNA positioned between nucleotides 88 and 89)</li>      <li>c.*46T&gt;A denotes a T to A substitution 46 nucleotides 3' of the        translation termination codon</li>      <li>the description c.76_77delinsTT is preferred over c.[76A&gt;T;        77G&gt;T]<br>        <i><img src="new.gif" border="0" height="12" width="31"><b>NOTE:</b></i>&nbsp;
        based on the definition of a substitution (<i><a href="standards.html#define">see
            Standards - Definition</a>; one nucleotide replaced by one other          nucleotide</i>) this change can not be described as a substitution        (like c.76_77AG&gt;TT or c.76AG&gt;TT)</li>    </ul>    <p><em><b>NOTE:</b></em> it is not correct to describe "<i>polymorphisms</i>"      as c.76A/G<em> (</em><i><a href="disc.html#polymorphism">see Discussion</a>)</i>.&nbsp;</p>    <hr>    <h3><a name="del"></a>Deletions</h3>    <p>A nucleotide deletion is a sequence change where one or more nucleotides      are removed (<i>see <a href="standards.html#define">see Standards -          Definition</a></i>). Deletions are described using <i><b>"del"</b></i>      after an indication of the first and last nucleotide(s) deleted, separated      by a "_" (<i>underscore</i>). For all descriptions the <em><b>most 3'          position</b></em> possible is arbitrarily assigned to have been      changed.<br>      <i><b>NOTE: </b>to discriminate known variable sequences from other        changes it is recommended to describe individual alleles differing from        the reference sequence like g.210T[5] (preferred over g.210_211delTT) or        g.121T[9] (preferred over g.210_211dupTT) </i>(<i><a href="#var">see          Repeated sequences</a></i>).&nbsp; </p>    <ul>      <li>c.76_78del (alternatively c.76_78delACT) denotes a ACT deletion from        nucleotides 76 to 78</li>      <li>deletions with <em><strong>uncharacterised breakpoints</strong></em>        (<i><a href="uncertain.html">see Uncertainties</a></i>)        <ul>          <li>c.(87+1_88-1)_(923+1_924-1)del denotes a deletion of exons 3 to7            starting at an unknown position in intron 2 (between coding DNA            nucleotides 87+1 and 88-1) and ending at an unknown position in            intron 7 (between coding DNA nucleotides 923+1 and 924-1). The            description indicates that exons 2 and 8 have been tested and shown            not to be deleted<br>            <span style="font-weight: bold; font-style: italic;">NOTE:</span>            the description c.88-?_923+?del does not specify start/end of the            deletion and is not correct when flanking sequences have been tested            <em><strong></strong></em> (<i><a href="http://www.hgvs.org/mutnomen/uncertain.html#exondel">see
                Uncertainties</a></i>) </li>          <li>c.(?_-30)_(*220_?)del denotes the deletion of the entire gene            (coding DNA reference sequence running from -30 (cap site) to *220            (polyA-addition site)</li>          <li><img src="new.gif" border="0" height="12" width="31">c.88+101_oGJB2:c.355-1045del
            denotes a deletion which ends in the flanking GJB2 gene at position            355-1045 (in the intron between nucleotides 354 and 355) on the            reverse strand (the genes are thus located and fused in opposite            transcriptional directions,<em> </em><i><a href="disc.html#fusion">see
                Discussion</a></i>)</li>        </ul>      </li>      <li><a name="except"></a>for all descriptions the <b>most 3' position</b>        possible is arbitrarily assigned to have been changed (<a href="FAQ.html#most3"><i>see
            FAQ</i></a>);        <ul>          <li>ACTT<b>TGT</b>GCC to&nbsp;ACTTGCC is described as c.5_7del            (c.5_7delTGT, not as c.4_6delTTG)</li>          <li>ctt<b>t</b>agGCATG to cttagGCATG in an intron is described as            c.301-3delT (not as c.301-5delT)</li>          <li>TCACTG<b>TCTG</b>CGGTAATC to TCACTG CGGTAATC is described as            c.7_10del (c.7_10delTCTG) and not as c.4_7del (c.4_7delCTGT).</li>          <li>AAAG<b>AAGAG</b>GAG to AAAG GAG is described as c.5_9del            (c.5_9delAAGAG) and not as c.3_7del (c.3_7delAGAAG)</li>          <li><b><i>Exceptions</i></b>            <ul>              <li>using a <i>coding DNA reference sequence</i> there is <b><i>an
                    exception</i></b> to the rule around exon/intron and                exon/exon borders when identical nucleotides flank the                exon/intron or exon/exon border;                <ul>                  <li>when the exon 3/intron 3 border is ..CAGgtg.. and RNA                    analysis shows no effect on splicing but a deletion of a G                    the change ..CAGgtg.. to ..CAgtg.. is described as c.3delG <i><b>and
                        not</b></i> c.3+1delG.</li>                  <li>when exon 3 ends with ..CAA.. and exon 4 starts with                    ..ACG.. and the sequence of genomic DNA shows that the last                    A-nucleotide of exon 3 is deleted (and not the first                    A-nucleotide in exon 4), the deletion changing ..CAAACG.. to                    ..CAACG.. is described as c.3delA <b><i>and not</i></b>                    c.4delA</li>                </ul>              </li>              <li>c.1210-12T(5_9) (<i>not c.1210-6T(5_9)</i>) describes the                variable stretch of 5 to 9 T-residues in intron 9 of the CFTR                gene. The most commonly used CFTR coding DNA reference sequence                contains a stretch of 7 T's (<i><a href="#var">see Repeated                    sequences</a></i>).&nbsp;</li>            </ul>          </li>        </ul>      </li>    </ul>    <hr>    <h3><a name="dup"></a>Duplications</h3>    <p>Duplications are designated by <i><b>"dup"</b></i> after an indication      of the first and last nucleotide(s) duplicated. It should be noted that      the description<strong> <em>"dup"</em></strong> (<em><a href="standards.html#define">see
          Standards</a></em>) may <strong><em>by definition</em></strong> only      be used when the sequence copy is directly 3'-flanking the original      copy.&nbsp; For all descriptions the <em><b>most 3' position</b></em>      possible is arbitrarily assigned to have been changed. For the addition of      more then 1 copy (3, 4, 5, etc.) <i><a href="#var">see Repeated sequences</a></i>      and<em> <a href="disc.html#tri">see Discussion</a>.</em><br>      <i><b>NOTE: </b>to discriminate known variable sequences from other        changes it is recommended to describe individual alleles differing from        the reference sequence like g.210T[5] (preferred over g.215_216del) or        g.210T[9] (preferred over g.215_216dup) </i>(<i><a href="#var">see          Repeated sequences</a></i>).&nbsp; </p>    <ul>      <li>duplicating insertions should be described as duplications (<a href="disc.html#dupins"><i>see
            Discussion</i></a>)        <ul>          <li>g.5dupT (or g.5dup, <em>not g.5_6insT</em>) denotes a duplication            (<i>"insertion"</i>) of the T nucleotide at position 5 in the            genomic reference sequence changing ACTCTGTGCC to ACTCT<u><strong>T</strong></u>GTGCC</li>          <li>g.7dupT (or g.7dup, <em>not g.5dupT, not g.7_8insT</em>) denotes            a duplication (<i>"insertion"</i>) of the T nucleotide at position 7            in the genomic reference sequence changing AGACTTTGTGCC to AGACTTT<u><strong>T</strong></u>GTGCC</li>          <li>g.7_8dup (or g.7_8dupTG, <em>not g.5_6dup, not g.8_9insTG</em>)            denotes a TG duplication in the TG-tandem repeat sequence changing            ACTTTGTGCC to ACTTTGTG<strong><u>TG</u></strong>CC</li>          <li>g.7_8[4] (or g.5_6[4], or g.5TG[4], <em>not g.7_10dup</em>) is            the preferred description of&nbsp; the addition of two extra TG's to            the variable TG repeated sequence changing ACTTTGTGCC to ACTTTGTG<strong><u>TGTG</u></strong>CC
            (<i><a href="#var">see Repeated sequences</a></i>)</li>        </ul>      </li>      <li>c.77_79dup (or c.77_79dupCTG) denotes that the three nucleotides 77 to        79 are duplicated (present twice)</li>      <li>duplications with <em><strong>uncharacterised breakpoints</strong></em>        (<i><a href="file:///Users/Boomheide/Files/WWW/HGVS/uncertain.html">see            Uncertainties</a></i>) </li>      <ul>        <li>c.(87+1_88-1)_(301+1_302-1)dup denotes a duplication of exons 3 to 4          starting at an unknown position in intron 2 (between coding DNA          nucleotides 87+1 and 88-1) and ending at an unknown position in intron          5 (between coding DNA nucleotides 301+1 and 302-1). The description          indicates that exons 2 and 5 have been tested and shown not to be          duplicated<br>          <span style="font-weight: bold; font-style: italic;">NOTE:</span> the          description c.88-?_301+?dup does not specify start/end of the          duplication and is not correct when flanking sequences have been          tested <em><strong></strong></em> (<i><a href="http://www.hgvs.org/mutnomen/uncertain.html#exondel">see
              Uncertainties</a></i>)<br>          <em><strong>NOTE:</strong> the description<strong> "dup"</strong> (<a              href="standards.html#define">see
              Standards</a>) may <strong>by definition</strong> only be used            when the additional copy is directly 3'-flanking of the original            copy (tandem duplication). In many cases there will be <strong>no              experimental proof</strong>, the additional copy may be anywhere            in the genome (i.e. inserted / transposed). It has been suggested            that, unless there is experimental evidence, exonic duplications            should be described using the format, i.e. c.88-?_301+?<b>(2)</b> (<a              href="recs-DNA.html#var">see
              Recommendations</a>).</em></li>        <li><a name="tri"></a><i><img src="new.gif" border="0" height="12" width="31"></i>c.(1031+1_1032-1)_(1357+1_1358+1)(3)&nbsp;
          denotes the presence of two additional copies of an exon, starting at          an unknown position in the flanking upstream intron (upstream of          coding DNA nucleotide 1032) and ending at an unknown position in the          flanking downstram intron (downstream of coding DNA nucleotide 1357).          The description, using "(3)" indicates there is no proof that the          three copies are in tandem<i> (<a href="#var">see Repeated sequences</a>,          </i><em><a href="uncertain.html">see Uncertainties</a></em>)</li>        <li><i><img src="new.gif" border="0" height="12" width="31"></i>c.(1031+1_1032-1)_(1357+1_1358+1)[3]&nbsp;
          denotes a <b>direct triplication</b> of an exon, starting at an          unknown position in the flanking upstream intron (upstream of coding          DNA nucleotide 1032) and ending at an unknown position in the flanking          downstram intron (downstream of coding DNA nucleotide 1357)<i> (<a href="#var">see
              Repeated sequences</a>)</i></li>      </ul>    </ul>    <hr>    <h3><a name="ins"></a>Insertions</h3>    <p>Insertions are designated by <i><b>"ins"</b></i> after an indication of      the nucleotides flanking the insertion site, followed by a description of      the nucleotides inserted. Duplicating insertions should be described as      duplications (<i><a href="disc.html#dupins">see Discussion</a></i>), not      as insertion. For large insertions the number of inserted nucleotides      should be mentioned, together with an accession.version number referring      to a sequence database file containing the complete inserted      sequence.&nbsp; </p>    <ul>      <li>c.76_77insT denotes that a T is inserted between nucleotides 76 and 77        of the coding DNA reference sequence </li>      <li>c.123+54_123+55insAB012345.2:g.76_420 denotes an intronic insertion (        between nucleotides c.123+54 and 123+55) of 345 nucleotides (nucleotides        76 to 420 like in GenBank file AB012345 version 2)<br>        <em><strong>NOTE</strong>: descriptions like c.123+54_123+55ins345 and          c.123+54_123+55insAlu are not allowed: "ins345" and "insAlu" are not          specified and the description can not be used to reconstruct the exact          change described.</em></li>    </ul>    <hr>    <h3><a name="indel"></a>Deletion / insertions (indels)</h3>    <p>Deletion/insertions of two or more consecutive nucleotides (indels) are      described as a <strong><em>deletion followed by an insertion</em></strong>      (<i><a href="disc.html#indel">see Discussion</a></i>). </p>    <ul>      <li>c.112_117delinsTG (alternatively c.112_117delAGGTCAinsTG) denotes the        replacement of nucleotides 112 to 117 (AGGTCA) by TG</li>      <li>c.113delinsTACTAGC (alternatively c.113delGinsTACTAGC) denotes the        replacement of nucleotide 113 by 7 new nucleotides (TACTACG)</li>      <li>c.114_115delinsA (alternative c.[114G&gt;A; 115delT])</li>    </ul>    <hr>    <h3><a name="inv"></a><i><img src="new.gif" border="0" height="12" width="31"></i>Inversions</h3>    <p>Inversions are designated by <b>"inv"</b> after an indication of the      first and last nucleotides affected by the inversion. </p>    <ul>      <li><i>c.203_506inv (or 203_506inv304) denotes that the 304 nucleotides          from position 203 to 506 have been inverted</i></li>    </ul>    <hr>    <h3><a name="con"></a><i><img src="new.gif" border="0" height="12" width="31"></i>Conversions</h3>    <p><u>Conversions </u>are designated by <b>"con"</b> after an indication      of the first and last nucleotides affected by the conversion, followed by      a description of the origin of the new nucleotides (<a href="disc.html#con">see
        Discussion</a>). </p>    <ul>      <li><em>g.123_678conNG_012232.1:g.9456_10011 describes a gene conversion          replacing nucleotides 123 to 678 of the reference genomic sequence          with nucleotides 9456 to 10011 from the sequence as present in GenBank          file NG_012232.1</em></li>    </ul>    <hr>    <h3><a name="tra"></a><i><img src="new.gif" border="0" height="12" width="31"></i>Translocations</h3>    <p>Translocations are described at the molecular level using the format      "t(X;4)(p21.2;q34)", followed by the usual numbering, indicating the      position translocation breakpoint. The sequences of the translocation      breakpoints need to be submitted to a sequence database (Genbank, EMBL,      DDJB) and the accession.version numbers should be given (<a href="disc.html#translocation">see
        Discussion</a>). </p>    <ul>      <li><i>t(X;4)(p21.2;q35)(c.857+101_857+102) denotes a translocation          breakpoint in the intron between coding DNA nucleotides 857+101 and          857+102, joining chromosome bands Xp21.2 and 4q34 </i></li>    </ul>    <hr>    <h3><a name="2in"></a>More changes in one individual</h3>    Two or more changes in a gene are described by combining the changes, per    chromosome (maternal and paternal), between square brackets (<i><b>"[;];[;]"</b></i>)    and using a semicolon <span style="font-style: italic;">(";"</span>) as    separator:&nbsp; <span style="font-style: italic;">[first change maternal;      second change maternal]; [first change paternal; second change paternal]</span>"    (<i><a href="file:///Users/Boomheide/Files/WWW/HGVS/disc.html#haplo">see        Discussion</a></i>). When changes are in different genes on different    chromosomes a space (<span style="font-style: italic;">" "</span>) is used    to separate the different chromosomes (<i><b>"[;] [;]"</b></i>).<br>    <span style="font-weight: bold; font-style: italic;">NOTE:</span> <span style="font-style: italic;">mixed
      descriptions like <b>c.</b>[76A&gt;C];<b>g.</b>[91C&gt;G] should not be      used. </span>    <ul>      <li><span style="font-weight: bold;">two changes in one gene on one          chromosome</span><br>        c.[76A&gt;C; 83G&gt;C] describes two changes found in a gene on one        chromosome; A to C change at nucleotide 76 and a G to C change at        nucleotide 83<br>        <br>      </li>      <li><span style="font-weight: bold;">two changes in one gene on both          chromosomes</span> (e.g. in <i><b>recessive diseases</b></i>)<br>        c.[76A&gt;C];[83G&gt;C] describes two changes found in a gene on each        chromosome (one paternal, one maternal); A to C change at nucleotide 76        on one chromosome and a G to C change at nucleotide 83 on the other        chromosome <br>        <h4>Examples</h4>      </li>      <ul>        <li>c.[76A&gt;C];[76A&gt;C] denotes a homozygous A to C change at          nucleotide 76</li>        <li>c.[76A&gt;C];[(76A&gt;C)] denotes a homozygous A to C change at          nucleotide 76, not confirmed by analysis of both parents, leaving the          possibility of non-amplification of the sequences analysed on the          other chromosome (e.g. due to a primer mismatch or a deletion) </li>        <li>c.[76A&gt;C];[?] denotes a A to C change at nucleotide 76 in a gene          on one chromosome and an expected not yet detected change on the other          chromosome</li>        <li>c.[76A&gt;C];[=] denotes a A to C change at nucleotide 76 in a gene          on one chromosome and a normal coding DNA Reference Sequence of the          other chromosome (<a href="FAQ.html#2alleles"><i>see FAQ</i></a>) </li>        <li><i><img src="new.gif" border="0" height="12" width="31">c.[76A&gt;C];[0]
            denotes a A to C change at nucleotide 76 in a gene on one chromosome            and the absence of the entire coding DNA Reference Sequence on the            other chromosome<br>            <span style="font-weight: bold;">NOTE:</span> the description c.0            should <span style="font-weight: bold;">preferably not be used</span>,            it does not specify the extent (begin / end) of the deletion.<br>            <br>          </i> </li>      </ul>      <li> <img src="new.gif" &gt;<i="" border="0" height="12" width="31"><b>two
          changes in one gene with chromosomes unknown</b> are described as        "[change1 (;) change2]" (<a style="font-style: italic;" href="file:///Users/Boomheide/Files/WWW/HGVS/FAQ.html#2unknown">see
          FAQ</a>)<br>        <i>c.[76A&gt;C(;)283G&gt;C] denotes that two changes were identified in          one individual (an A to C change at nucleotide 76 and a G to C change          at nucleotide 283), but it is not known whether these changes are on          the same chromosome (in cis) or on different chromosomes (in trans)<br>          <br>        </i></li>      <li><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif" &gt;<i="" border="0"          height="12"          width="31"><b>changes
          in different genes on one chromosome</b> are described as <span style="font-style: italic;">          "[change1;change2]"</span><br>        <span style="font-style: italic;">When a coding DNA reference sequence          is used the description should clearly indicate based on which          reference sequence each variant is described. This can be done using          either the <span style="font-weight: bold;">accession.version number</span>          or the <span style="font-weight: bold;">Gene Symbol</span> (i.e. when          elsewhere the reference sequence connected to each gene is specified).        </span></li>      <h4>Examples</h4>      <ul>        <li><span style="font-style: italic;">hg19            chrX:g.[30683643A&gt;G;33038273T&gt;G]<br>            describes a A to G change at nucleotide g.30683643 (GK gene) and a T            to G change at nucleotide g.33038273 (DMD gene) on one X-chromosome            based on the genomic reference sequence of genome build hg19.</span></li>        <span style="font-style: italic;"> </span>        <li><span style="font-style: italic;">c.[NM_000167.5:94A&gt;G;NM_004006.2:76A&gt;C]<br>            describes a A to G change at nucleotide c.94 (GK gene, based on            coding DNA reference sequence NM_000167.5) and a A to C change at            nucleotide c.76 (DMD gene, based on coding DNA reference sequence            NM_004006.2) on one X-chromosome.</span></li>        <span style="font-style: italic;"> </span>        <li><i>c.[GK:94A&gt;G;DMD:76A&gt;C]<br>            describes a A to G change at nucleotide c.94 (GK gene) and a A to C            change at nucleotide c.76 (DMD gene) on one X-chromosome. Elsewhere            the coding DNA reference sequences are specified as NM_000167.5 for            GK and NM_004006.2 for DMD.<br>          </i> </li>      </ul>    </ul>    <ul>      <li><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif" &gt;<i="" border="0"          height="12"          width="31"><b>changes
          in different genes on different chromosome</b> are described as <span          style="font-style: italic;">"[change1]
          [change2]"</span><br>        <span style="font-style: italic;">When a coding DNA reference sequence          is used the description should clearly indicate based on which          reference sequence each variant is described. This can be done using          either the <span style="font-weight: bold;">accession.version number</span>          or the <span style="font-weight: bold;">Gene Symbol</span> (i.e. when          elsewhere the reference sequence connected to each gene is specified).        </span><span style="font-style: italic;"></span>        <h4>Examples</h4>      </li>      <ul>        <li><span style="font-style: italic;">hg19 chr1:g.[35227587C&gt;G]&nbsp;            chr13:g.[20763083A&gt;T]<br>            describes a C to G change at nucleotide g.</span><span style="font-style: italic;"><span              style="font-style: italic;">35227587</span>            (GJB4 gene) on chromosome-1 and </span><span style="font-style: italic;">an
            A to T change at nucleotide</span><span style="font-style: italic;"><span              style="font-style: italic;"></span>            g.</span><span style="font-style: italic;"><span style="font-style: italic;">35250398</span>            (GJB2 gene) on chromosome-13 based on the genomic reference sequence            of genome build hg19.</span></li>        <li><span style="font-style: italic;"></span><span style="font-style: italic;">NM_153212.2:c.[732C&gt;G]&nbsp;
            </span><span style="font-style: italic;">NM_004004.5:c.[638T&gt;A]<br>            describes a C to G change at nucleotide c.732 (GJB4 gene, chromosome            1, based on coding DNA reference sequence </span><span style="font-style: italic;"><span              style="font-style: italic;">NM_153212.2</span>)            and </span><span style="font-style: italic;"><span style="font-style: italic;"></span><span              style="font-style: italic;">an
              A to T change at nucleotide</span><span style="font-style: italic;"><span                style="font-style: italic;"></span></span>            c.638 (GJB2 gene, chromosome 13, based on coding DNA reference            sequence </span><span style="font-style: italic;"><span style="font-style: italic;">NM_004004.5</span>).<br>          </span></li>        <li><i>GJB4:c.[732C&gt;G]&nbsp; GJB2:c.[</i><i><span style="font-style: italic;">638T&gt;A</span>]<br>            describes a C to G change at nucleotide c.732 (GJB4 gene, chromosome            1) and </i><i><span style="font-style: italic;">deletion of a G at              nucleotide c.35</span> (GJB3 gene, chromosome 13). Elsewhere the            coding DNA reference sequences are specified as </i><i><span style="font-style: italic;">NM_153212.2</span>            for GJB4 and </i><i><span style="font-style: italic;">NM_004004.5</span>            for GJB2.</i></li>      </ul>    </ul>    <h4>Mosaicism</h4>    <p> </p>    <p><img src="new.gif" border="0" height="12" width="31"><em><strong>Mosaicism
          - two different nucleotides in one position caused by somatic          mosaicims</strong></em><i> are described as "[=/nucleotide 2]" (<a href="FAQ.html#mosaic">see
          FAQ</a>)</i>. </p>    <ul>      <li><i>c.[83G=/83G&gt;C] describes a mosaic case where at position 83          besides the normal sequence (a G, described as '=') also chromosomes          are found containing a C (c.83G&gt;C)</i></li>    </ul>    <h4>Chimerism</h4>    <p><img style="width: 31px; height: 12px;" src="new.gif" border="0"><em><strong>Chimerism
          - two different nucleotides in one position caused by chimerism</strong></em><i>        are described as "[=//nucleotide 2]"</i> </p>    <ul>      <li><i>c.[=//83G&gt;C] describes a chimeric case where at position 83          besides the normal sequence (a G, described as '=') also cells are          found containing another chromosome containing a C at this position          (c.83G&gt;C)</i></li>    </ul>    <hr>    <h3><a name="var"></a><a href="#del"><i><img src="new.gif" border="0" height="12"            width="31"></i></a>Repeated
      sequences</h3>    <p>A frequently occuring sequence change is the variability of repeated      sequences. Within this category we discriminate both small sequences      (mono-, di-, tri-, etc nucleotide repeats) as well as the much larger      ones. Such changes are described using the format <em><strong>"position-first-repeat-unit_[number]</strong></em>"      (e.g. g.123_124[4]) where <em><strong>position-first-repeat-unit</strong>        gives the location of the first unit</em> of the variable sequence      repeat and <em><strong>[number]</strong> the number of units present</em>      in the allele described. </p>    <ul>      <li>the first unit of the repeat is preferably described based on        position, like. g.123_124. <em>For short/simple repeats it is          acceptable to include the content of the repeated unit, using the          format "position-first-nucleotide-repeat_content" like g.123TG[4].</em>        <strong>Do not use</strong> a mix of these descriptions like        g.123_124TG[4]. This contains redundant information (123_124 and TG)        with the danger of being in conflict. <br>        <em><span style="font-weight: bold;">NOTE:</span> including the content          of the sequence involved quickly gives descriptions which become too          lengthy.</em></li>      <li>uncertainties regarding the number of repeated copies are given        between brackets, like c.-128GGC[(600_800)]. </li>      <li><span style="font-weight: bold; font-style: italic;"><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif"              border="0"              height="12"              width="31"></i></span>the        repeated sequences may have complex structures, consisting of a mix of        several repeated sequences directly following each other. When        sequenced, one should describe it including the individual repeated        elements, like g.456TG[4]TA[9]TG[3] (or        g.456_465[4]466_489[9]490_499[3]). When not sequenced, but <span style="font-weight: bold; font-style: italic;">based          on fragment size</span>, it should be described like g.456_465[16].</li>      <li>the same format can be used to describe the presence of multiple        copies (triplication, quadruplication, etc.) of larger sequences, e.g.        exons. Square brackets (<em><strong>"[ ]"</strong></em>) should only be        used when their is experimental evidence that the additional copies are        in tandem on the same chromosome.<br>        <em><strong>NOTE:</strong> the description<strong> "dup"</strong> (<a href="standards.html#define">see
            Standards</a>) may <strong>by definition</strong> only be used when          the additional copy is directly 3'-flanking of the original copy          (tandem duplication). In many cases there will be <strong>no            experimental proof</strong>, the additional copy may be anywhere in          the genome (i.e. inserted / transposed). It has been suggested that,          unless there is experimental evidence, exonic duplications should be          described using the format, i.e. c.88-?_301+?<b>(2)</b>.</em></li>    </ul>    <h4>Examples</h4>    <ul>      <li>g.123_124[4] (<em>alternatively g.123TG[4]</em>) describes a sequence        variable in the number of TG repeats where the first unit is present at        position g.123_124 in the genomic reference sequence and the allele        described contains 4 units.</li>      <li>c.1210-12T[5_9] describes a famous variable stretch of 5 to 9        T-residues in intron 9 of the CFTR gene. The most commonly used CFTR        coding DNA reference sequence contains a stretch of 7 T's. It is        recommended to describe individual alleles differing from this reference        sequence as c.1210-12T[5] (<i>not c.1210-7_1210-6delTT</i>) or        c.1210-12T[9] (<i>not c.1210-7_1210-6dupTT</i>).<br>        <i><b>NOTE: </b>the repeat should not be described as c.1210-6T[5_9]</i></li>      <li>c.123+74TG[3_6] (<i>alternatively c.123+74_123+75[3_6]</i>) indicates        that a TG di-nucleotide repeat is present, starting at nucleotide 74 in        the intron following cDNA nucelotide c.123, which is found varying from        3 to 6 copies in the population        <ul>          <li><i>c.123+74TG[4];[5] denotes that a person carries a TG              di-nucleotide repeat of length 4 on one chromosome and of length 5              on the other chromosome</i></li>        </ul>      </li>      <li>in literature the Fragile-X tri-nucleotide repeat is known as the        CGG-repeat, but based on the coding DNA reference sequence (GenBank        NM_002024.4) and the rule that for all descriptions the <span style="font-weight: bold; font-style: italic;">most
          3' position possible</span> should be arbitrarily assigned the repeat        has to be described as a GGC-repeat. In addition the repeat is        interrupted by GGA triplets (see e.g. <a style="font-style: italic;" href="http://www.ncbi.nlm.nih.gov/pubmed/8634688">Eichler          1995</a>) making it a complex repeat which can not be accurately        described based on sizing only. The sequence represented by the FMR1        coding DNA Reference Sequence (GenBank <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002024.5">NM_002024.5</a>)        is c.-128GGC[9]GGA[1]GGC[10].<br>        <span style="font-weight: bold; font-style: italic;">NOTE:</span> the        sequence represented by coding DNA reference sequence <a href="http://www.ncbi.nlm.nih.gov/nucleotide/NM_002024.3">NM_002024.3</a>        is c.GGC[9]GGA[1]GGC[9]GGA[1]GGC[10]. Based on this reference sequence        descriptions of variants will be significantly different. To prevent        such differences the recommendation is to use a LRG (Locus Reference        Genomic sequence, <span style="text-decoration: underline;"><span style="font-style: italic;"></span></span><a          href="http://www.genomemedicine.com/content/pdf/gm145.pdf"><i>Dalgleish
            et al. 2010</i></a>) for variant descriptions.<br>        <ul>          <li>c.-128_-126[79] describes the presence of an allele with an            extended GGC-repeat of <i><b>exactly 79 units</b></i>.<br>            <span style="font-weight: bold; font-style: italic;"><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif"                  border="0"                  height="12"                  width="31"></i>NOTE:</span>            the description c.-128GGC[79] can not be used since the repeat is            probably interrupted by one or more GGA-triplets</li>          <li>c.-128_-126[(600_800)] describes the presence of an extended            GGC-repeat with an estimated length between 600 and 800 copies<br>            <b><i><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif" border="0"                    height="12"                    width="31"></i>NOTE:</i></b>            brackets are used to indicate uncertainties (<a href="file:///Users/Boomheide/Files/WWW/HGVS/uncertain.html"><i>see
                Uncertainties</i></a>), the description c.-128GGC[(600_800)] can            not be used since the repeat is probably interrupted by one or more            GGA-triplets </li>          <li>c.-128_-126[(1000)] describes the presence of an extended            GGC-repeat of about 1000 units.<br>            <b><i><i><img src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif" border="0"                    height="12"                    width="31"></i>NOTE:</i></b>            brackets are used to indicate uncertainties (<a href="file:///Users/Boomheide/Files/WWW/HGVS/uncertain.html"><i>see
                Uncertainties</i></a>), the description c.-128GGC[(1000)] can            not be used since the repeat is probably interrupted by one or more            GGA-triplets</li>        </ul>      </li>      <li><a name="tri"></a><i><img src="new.gif" border="0" height="12" width="31"></i>c.1032-?_1357+?(3)&nbsp;
        denotes the presence of two additional copies of an exon, starting at an        unknown position in the flanking upstream intron (upstream of coding DNA        nucleotide 1032) and ending at an unknown position in the flanking        downstream intron (downstream of coding DNA nucleotide 1357). The        description, using "(3)" indicates there is no proof that the three        copies are in tandem<i> (</i><em><a href="uncertain.html">see            Uncertainties</a></em>)</li>      <li><i><img src="new.gif" border="0" height="12" width="31"></i>c.1032-?_1357+?[3]&nbsp;
        denotes a <b>direct triplication</b> of an exon, starting at an unknown        position in the flanking upstream intron (upstream of coding DNA        nucleotide 1032) and ending at an unknown position in the flanking        downstram intron (downstream of coding DNA nucleotide 1357)<br>        <i><b>NOTE:</b> the use of tri (for triplication), qua (for          quadruplication), etc. is not recommended</i> (<a href="disc.html#tri"><i>see
            Discussion</i></a>) </li>      <li>g.1209_4523[12_45] denotes that a 3.3 kb repeat sequence of which the        first copy is present in the genomic reference sequence from nucleotides        1209 to 4523 can be found repeated 12 to 45 times in the population        <ul>          <li><i>g.1209_4523[14];[23] denotes that a person carries a 3.3 kb              repeat with 14 copies on one chromosome and of 23 copies on the              other chromosome<br>            </i></li>        </ul>      </li>    </ul>    <hr>    <h3><a name="com"></a><img src="new.gif" border="0" height="12" width="31">Complex
      changes</h3>    <p>Sequence changes can be very complex, involving several changes at a      specific location. The description of such changes using the      recommendations given above can become rather complicated and at some      point, although literally correct, effectively meaningless. In such cases      the recommendation is to submit the sequence that has been determined to      GenBank and to use the accession.version number in the description. </p>    <ul>      <li><i>c.123_678conNM_004006.1:c.123_678 describes a gene conversion          replacing nucleotides c.123 to c.4567 of the coding DNA sequence of          the transcript of interest with nucleotides c.123 to c.678 from a          transcript sequence as present in GenBank file NM_004006 (version 1)</i></li>      <li><i>c.88+101_oGJB2:c.355-1045del denotes a deletion which ends in the          flanking GJB2 gene at position 355-1045 (in the intron between          nucleotides 354 and 355) on the reverse strand (the genes are thus          located and fused in opposite transcriptional directions, <a href="disc.html#fusion">see
            Discussion</a>)</i></li>      <li><i>c.123+54_123+55insAB012345.2:g.76_420 denotes an intronic insertion          (between nucleotides c.123+54 and 123+55) of 345 nucleotides          (nucleotides 76 to 420 like in GenBank file AB012345 version 2)</i></li>    </ul>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">MutNomen
        homepage</a> | <a href="checklist.html">Check-list</a> | <a href="standards.html">Symbols,
codons,
        etc.</a> |<br>      | <b>Recommendations:</b>&nbsp; <a href="recs.html">general</a>,&nbsp;<a        href="recs-DNA.html">DNA</a>,&nbsp;
      <a href="recs-RNA.html">RNA</a>,&nbsp; <a href="recs-prot.html">protein</a>,      <a href="uncertain.html">uncertain</a> |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="standards.html">Symbols,
        codons, etc.</a> | <a href="history.html">History</a> |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef</a>,&nbsp;
      <a href="examplesDNA.html">DNA</a>,&nbsp; <a href="examplesRNA.html">RNA</a>,&nbsp;
      <a href="examplesAA.html">protein</a> | </p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" size="-5"><a href="http://www.HGVS.org/copyright.html">Copyright
                  &#65533;</a> </font><font size="1">HGVS 2007 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>