<html>  <!--	Author: 	Dr. Johan T. den Dunnen	Date: 	02-01-2005	URL: 	recs-protein.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>HGVS recommendations: protein sequence variants</title>    <style type="text/css"><!--  h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body style="color: black; background-color: white;" link="#0000FF" vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                height="94"                border="0"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Recommendations for the description of protein              sequence variants (v2.0) </h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified November 20, 2015</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i>Since references to WWW-sites are not yet acknowledged        as citations, please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den
          Dunnen JT and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when        referring to these pages.</i></p>    <hr>    <h2>Contents</h2>    <ul>      <li><b>Recommendations</b>        <ul>          <li><a href="recs.html">general recommendations</a> </li>          <li><a href="recs-DNA.html">changes at DNA level</a></li>          <li><a href="recs-RNA.html">changes at RNA level</a></li>          <li><a href="#prot"><b>changes at protein level</b> </a>            <ul>              <li><a href="#silent">silent changes</a></li>              <li><a href="#sub">substitutions</a></li>              <li><a href="#del">deletions</a> &nbsp; <em>-&nbsp;                  in-frame,&nbsp; nonsense,</em><i> &nbsp; initiation codon</i></li>              <li><a href="#dup">duplications</a></li>              <li><a href="#ins">insertions</a></li>              <li><i><img src="new.gif" height="12" border="0" width="31"><a href="#var">variability
                    of short sequence repeats</a></i></li>              <li><a href="#indelp">deletion/insertion&nbsp; (indels)</a>&nbsp;                -&nbsp; <em>frame shift</em></li>              <li><i><img src="new.gif" height="12" border="0" width="31"></i><em><a                    href="#extp">extensions</a>                  &nbsp; -&nbsp; </em><i>initiation codon,&nbsp; no-stop change</i></li>              <li><i><img src="new.gif" height="12" border="0" width="31"><a href="#2in">more
                    changes in one individual</a></i>&nbsp; -&nbsp; <em>alleles,&nbsp;
                  mosaicism, &nbsp; chimerism</em></li>            </ul>          </li>        </ul>      </li>      <li><b>Explanations / examples</b>        <ul>          <li><a href="quickref.html">Quick Reference</a></li>          <li><a href="examplesDNA.html">changes at DNA level</a></li>          <li><a href="examplesRNA.html">changes at RNA level</a></li>          <li><a href="examplesAA.html">changes at protein level</a></li>        </ul>      </li>      <li><a href="standards.html"><strong>Codons and encoded amino acids </strong></a>        <ul>          <li>genetic code </li>          <li>amino acid descriptions&nbsp; <em>(1- and 3-letter code)</em></li>        </ul>      </li>    </ul>    <hr>    <h2><a name="prot"></a>Protein level </h2>    <p><em>(suggestions extending the <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">published</a>        recommendations </em><img src="new.gif" height="12" border="0" width="31"><em>in
        italics)</em></p>    <hr>    <p><em><strong>NOTE:</strong> definitions of protein changes have been        extensively reviewed (2013-Q2). This did not affect HGVS recommendations        for variant descriptions but it did change under which category specific        types are listed below. For example, where a <a href="#nonsense">nonsense
          variant</a> (p.Trp26Ter or p.W26*) was originally listed under <a href="#sub">Substitutions</a>        it is now listed under <a href="#del">Deletions</a>.</em></p>    <p>The recommendations for the description of protein variants explain how      changes in the sequence of a protein should be described. It should be      noted that these changes are a consequence of a variant at DNA level that      may or may not have influenced the processing of the RNA before it is      translated into protein. Experimental evidence of protein level variants,      e.g. from mass spectrometry amino acid sequencing, will rarely exist. In      some cases indirect evidence might come from protein sizing (Western blot      analysis) or localisation (immuno-histochemical staining). In most cases      protein descriptions will however be <em><strong>deduced only</strong></em>,      predicted from the changes detected on DNA and/or RNA level.</p>    <p>Specific terms are used to describe the consequences of a change at      protein level, like <strong><em>missense</em>, <em>nonsense</em>, <em>silent</em>      </strong>and <strong><em>frame shift</em></strong>. These terms are not      used in the descriptions given below. Missense is under substitution,      nonsense under deletion, silent under no change and frame shift under      deletion/insertion (indel).</p>    <h3>General</h3>    <p>Sequence changes at protein level are described like those at the DNA      level with the following modifications / additions; </p>    <ul>      <li>descriptions at protein level may only be given <em><strong>in            addition to</strong></em> a description at DNA (and RNA) level</li>      <li>descriptions at protein level should describe the changes observed on        protein level and <i><b>not try to incorporate any knowledge regarding            the change at DNA-level&nbsp;</b></i> (<a href="file:///Users/Boomheide/Files/WWW/HGVS/FAQ.html#protDNA"><i>see
            FAQ</i></a>)</li>      <li><img src="new.gif" height="12" border="0" width="31">to indicate that        the description at protein level is without any experimental evidence it        is recommended that, <b><i>when RNA nor protein has been analysed</i></b>,        the description is given <i><b>between brackets</b></i>, like <em>p.(Arg22Ser)&nbsp;
          (<a href="disc.html">see Discussion</a> <small><font color="#FF0000"><strong>2012-10-12</strong></font></small>)</em></li>      <li>a <b><i>"p."</i></b> preceding the change is used to indicate a        description at protein level</li>      <li>amino acids are described as "<em><strong>Trp26</strong></em>" or "<em><strong>W26</strong></em>",
        i.e. with capital first letter (not as "<i>trp26</i>" or "<i>Trp<sup>26</sup></i>")
        </li>      <li>the <a href="codon.html">3-letter amino acid code</a> is        preferred&nbsp;to describe the amino acid residues (<i><a href="disc.html#1or3">see
            Discussion</a></i>)</li>      <li><a name="protonly"></a>for all descriptions the <em><strong>most            C-terminal position possible</strong></em> is arbitrarily assigned        to have been changed</li>      <li>for <a href="#nonsense"><span style="font-weight: bold;">nonsense            variants</span></a> the description <span style="font-style: italic; font-weight: bold;">does
          not include</span> the deletion at protein level from the site of the        change to the C-terminal end of the protein (<em>so p.Trp26Ter, not          p.Trp26_Leu833del</em>)</li>      <li><strong><a href="#2in">alleles</a></strong> are described using square        brackets (<b><i>"p.[ ]"</i></b>)</li>      <li><em><strong>Miscellaneous</strong></em>        <ul>          <li><i><u>unknown effect</u></i>            <ul>              <li><i>p.?&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </i>- protein has                not been analysed, an effect is expected but difficult to                predict</li>              <li><i>p.(=)&nbsp;&nbsp;&nbsp; </i>- protein has not been                analysed, but no change is expected</li>              <li><i>p.=&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </i>- protein has not                been analysed, RNA was, but no change is expected</li>            </ul>          </li>          <li><i><u>no protein<br>              </u></i>changes which affect the promoter of a gene, the            transcription initiation site (cap site), the translation initiation            site, etc. may affect the amount of protein produced;            <ul>              <li><i>p.0&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </i>- no protein                can be detected (<em>experimental data should be available</em>)</li>              <li><i>p.0?&nbsp;&nbsp;&nbsp;</i>&nbsp; - probably no protein is                produced</li>            </ul>          </li>          <li><i><u>amount of protein<br>              </u></i>changes which do not affect the protein sequence itself            but only the <span style="font-weight: bold; font-style: italic;">amount
              of protein</span> produced (other then <strong><em>no protein</em></strong>)            are described as p.= (<em>no change</em>). Remarks on the amount of            protein should be made separately (e.g. under Remarks).</li>          <li><i><u>protein modifications<br>              </u></i>currently no recommendations exist for the description of            protein modifications. Remarks on protein modifications should be            made separately (e.g. under Remarks).</li>        </ul>      </li>    </ul>    <h4><b>Amino acid coding and numbering</b> </h4>    <ul>      <li>the Methionine encoded by the translation initiation site (<em>start          codon</em>) is numbered as residue 1 ("<em><strong>Met1</strong></em>"        or <em><strong>" M1"</strong></em> )</li>      <li><img src="new.gif" height="12" border="0" width="31">the protein        coding sequence ends at a translation termination codon (<em>stop codon</em>),
        described at protein level as "<em><strong>Ter</strong></em>" or "<span          style="font-weight: bold; font-style: italic;">*</span>"        ("<em><strong>*</strong></em>" in 1- and 3-letter amino acid code) (<a href="versioning.html#important"><i>see
            Important changes</i></a>)</li>      <li>the protein reference sequence should represent the<b><i> primary            translation product</i></b>, not a processed mature protein, and        thus include e.g. signal peptide sequences (<a href="FAQ.html#protref"><i>see
            FAQ</i></a>) </li>      <li>amino acids originating from changes introducing <strong><em>upstream            translation initiation</em></strong> are numbered like nucleotides;        <em>..., Gln-2, Thr-1</em></li>      <li>amino acids originating from changes resulting in <em><strong>translation
            of intronic sequences</strong></em> are numbered like nucleotides; <em>Val4+1,
          Ser4+2, ..., Phe5-2, Gln5-1</em></li>      <li>amino acids originating from <a href="#nonX">no-stop changes</a>        causing <em><strong>translation downstream of the translation            termination codon</strong></em> are numbered like nucleotides; <em>Gln*1,
          Ser*2, ...</em></li>    </ul>    <hr>    <h3><a name="silent"></a><b>Silent changes</b></h3>    <p>Description of so called "silent" changes can be described using <strong><em>p.(Leu54=)</em></strong>      (<a style="font-style: italic;" href="file:///Users/Boomheide/Files/WWW/HGVS/accepted001.html">see
        SVD-WG001</a>). The format p.(Leu54Leu) (or p.(L54L)) should not be      used. These descriptions can only be given in addition to a description at      <b><i>DNA level</i></b>&nbsp;(<a href="disc.html#silent"><i>see Discussion</i></a>).</p>    <hr>    <h3><a name="sub"></a><b>Substitutions</b></h3>    <p>Substitutions (missense changes) <em><strong>replace one amino acid by          one other amino acid</strong></em> and are described using the format      <b><i>p.Trp26Cys</i></b>. The description does not use the "<b><i>&gt;</i></b>"-character
      used on DNA- and RNA level (indicating "<em>changes to</em>"). </p>    <ul>      <li><strong>missense variant</strong><br>        p.Trp26Cys denotes that amino acid Tryptophan-26 (Trp, W) is changed to        a Cysteine (Cys)</li>      <li><em><strong>start codon</strong>&nbsp; (initiating methionine change -          Met1)</em>&nbsp; (<i><a href="disc.html#Met">see Discussion</a>,&nbsp;          <a href="examplesAA.html#sub">see Examples</a></i>)<br>        a change affecting the translation initiation codon (Met-1) is,        depending on its consequence, either        <ul>          <li>a change which results in no protein being produced (<em>p.0</em>)<br>            <i>Met1?</i> -&nbsp; denotes that amino acid Methionine-1            (translation initiation site) is changed and that it is unclear what            the consequences of the change are</li>          <li>an N-terminal <a href="#del">deletion</a> (<i>p.Phe2_Met46del</i>,            i.e. activating downstream translation initiation)<br>            <i><span style="font-weight: bold;">NOTE:</span> up to August 2015              the example given was p.Met1_Lys45del which is not correct, the 3'              rule should be applied<br>            </i></li>          <li>an <a href="#extp">extension</a> (<i>p.Met1ValextMet-12</i>,            activating upstream translation initiation)</li>        </ul>      </li>      <li><em><strong>nonsense variant</strong></em><br>        is a special type of amino acid deletion introducing an immediate        translation stop codon and is described like an amino acid substitution        (<span style="font-style: italic;">p.Trp26Ter or p.Trp26*</span>)<br>        <span style="font-weight: bold; font-style: italic;">NOTE:</span> the        description <span style="font-style: italic; font-weight: bold;">does          not include</span> the deletion at protein level of the entire        C-terminal amino acid sequence like <em>p.Trp26_Leu833del</em> </li>      <li><em><strong><u><a name="nonX"></a></u></strong></em><i><b>no-stop            change (Ter)</b><img src="new.gif" height="12" border="0" width="31">&nbsp;
          (change in stop codon, Ter/*)<br>        </i>a change affecting the translation termination codon (Ter, *) is        described as an <a href="#extp">extension</a> (<i>p.Ter110GlnextTer17</i>        or <i>p.*110Glnext*17</i>).</li>    </ul>    <hr>    <h3><a name="del"></a>Deletions</h3>    <p>Deletions remove one or more amino acid residues from the protein and are      described using "<em><strong>del</strong></em>" after an indication of the      first and last amino acid(s) deleted separated by a "_" (<i>underscore</i>).
      Deletions remove either a small internal segment of the protein (<em>in-frame
        deletion</em>), part of the N-terminus of the protein (<em>initiation        codon change</em>) or the entire C-terminal part of the protein (<em>nonsense
        change</em>). A <em><strong>nonsense</strong></em> change is a special      type of deletion removing the entire C-terminal part of a protein starting      at the site of the variant (specified <em><small><font color="#FF0000"><strong>2013-03-16</strong></font></small></em>).
      </p>    <ul>      <li><strong>in-frame deletions</strong>&nbsp; -&nbsp; are described using        <i><b>"del"</b></i> after an indication of the first and last amino        acid(s) deleted separated, by a "_" (<i>underscore</i>).        <ul>          <li>p.Gln8del in the sequence MKMGHQQ<u><strong>Q</strong></u>CC            denotes a Glutamine-8 (Gln, Q) deletion to MKMGHQQCC</li>          <li>p.(Cys28_Met30del) denotes RNA nor protein was analysed but the            predicted change is a deletion of three amino acids, from            Cysteine-28 to Methionine-30</li>        </ul>      </li>      <li><strong>initiating methionine change (Met1) causing a N-terminal          deletion<em>&nbsp;</em></strong> (<i><a href="disc.html#Met">see            Discussion</a>,&nbsp; <a href="examplesAA.html#sub">see Examples</a></i>)        <br>        <em><strong>NOTE:</strong></em>&nbsp; changes extending the N-terminal        protein sequence are described as an <a href="#extp">extension</a>        <ul>          <li><i>p.0</i>&nbsp; -&nbsp; no protein is produced (experimental data            should be available)<br>            <em><strong>NOTE:</strong></em> this change is not described as <em>p.Met1_Leu833del</em>,            i.e. as a deletion removing the entire protein coding sequence</li>          <li><i>p.Met1?</i> -&nbsp; denotes that amino acid Methionine-1            (translation initiation site) is changed and that it is unclear what            the consequence of this change is</li>          <li><i>p.Met1_Lys45del&nbsp;</i> -&nbsp; a new translation initiation            site is activated (at <em>Met46</em>)</li>        </ul>      </li>      <li><strong><a name="nonsense"></a>nonsense variant</strong>&nbsp; -&nbsp;        is a <em><strong>special type of amino acid deletion</strong></em>        removing the entire C-terminal part of a protein starting at the site of        the variant. A nonsense change is described as a <a href="#sub">substitution</a>,        using the format <i>p.Trp26Ter</i> (alternatively <i>p.Trp26*</i>).        The description does not include the deletion at protein level from the        site of the change to the C-terminal end of the protein (stop codon)        like <em>p.Trp26_Leu833del</em> (the deletion of amino acid residue        Trp26 to the last amino acid of the protein Leu833).        <ul>          <li>p.(Trp26Ter) indicates RNA nor protein was analysed but amino acid            Tryptophan26 (Trp, W) is predicted to change to a stop codon (Ter) (<i>alternatively
              p.(W26*) or p.(Trp26*)</i>)</li>        </ul>      </li>    </ul>    <p><i><b>NOTE:</b></i> for all descriptions the <b>most C-terminal position</b>      possible is arbitrarily assigned to have been changed</p>    <hr>    <h3><a name="dup"></a>Duplications</h3>    <p>Duplications are described using "<b><i>dup</i></b>" after an indication      of the first and last amino acid(s) duplicated separated by a "_" (<i>underscore</i>).      In-frame duplications containing a translation stop codon in the      duplicated sequence are described as an insertion of a <a style="font-weight: bold;"        href="file:///Users/Boomheide/Files/WWW/HGVS/recs-prot.html#nonsense">nonsense        variant</a>, not as a deletion-insertion removing the entire C-terminal      amino acid sequence. </p>    <ul>      <li>p.Gly4_Gln6dup in the sequence MKMGHQQQCC denotes a duplication of        amino acids Glycine-4 (Gly, G) to Glutamine-6 (Gln, Q) (i.e. MKMGHQ<strong><u>GHQ</u></strong>QQCC)</li>      <li>duplicating insertions in single amino acid stretches (or short tandem        repeats) are described as a duplication, e.g. a duplicating HQ insertion        in the HQ-tandem repeat sequence of MKMGHQHQCC to MKMGHQHQ<u><strong>HQ</strong></u>CC
        is described as p.His7_Gln8dup (not p.Gln8_Cys9insHisGln)</li>    </ul>    <p><i><b>NOTE:</b> </i>for all descriptions the <b>most C-terminal        position</b> possible is arbitrarily assigned to have been changed</p>    <hr>    <h3><a name="ins"></a>Insertions</h3>    <p>Insertions add one or more amino acid residues between two existing amino      acids and this insertion is not a copy of a sequence immediately      5'-flanking (<a href="#dup"><i>see Duplication</i></a>). Insertions are      described using <i><b>"ins"</b></i> after an indication of the amino      acids flanking the insertion site, separated by a "_" (<i>underscore</i>)      and followed by a description of the amino acid(s) inserted. In-frame      insertions containing a translation stop codon in the inserted sequence      are described as an insertion of a <a style="font-weight: bold;" href="#nonsense">nonsense        variant</a>, not as a deletion-insertion removing the entire C-terminal      amino acid sequence. Since for large insertions the amino acids can be      derived from the DNA and/or RNA descriptions they need not to be described      exactly but the total number may be given (<i>like "ins17"</i>). </p>    <ul>      <li><span style="font-weight: bold;">in frame</span></li>      <ul>        <li>p.Lys2_Met3insGlnSerLys denotes that the sequence GlnSerLys (QSK)          was inserted between amino acids Lysine-2 (Lys, K) and Methionine-3          (Met, M), changing MKMGHQQQCC to MK<u><strong>QSK</strong></u>MGHQQQCC</li>        <li>p.(Pro2_Ile3insGlyTer) is the predicted consequence of the insertion          c.6_7insGGGTAG (coding reference sequence NM_000059.3)<br>          <span style="font-weight: bold; font-style: italic;">NOTE:</span> this          is not described as p.(Ile3_Ile3418delinsGly), a deletion-insertion          removing the entire protein coding sequence</li>      </ul>      <ul>        <li>p.Trp182_Gln183ins17 describes a variant that inserts 17 amino acids          between amino acids Trp182 and Gln183<br>          <b><i>NOTE:</i></b> it must be possible to deduce the 17 inserted          amino acids from the description given at DNA or RNA level<em><strong><u><a                  name="nonX"></a></u></strong></em></li>      </ul>    </ul>    <p><em><strong>NOTE:</strong></em>&nbsp; duplicating insertions should be      described as duplications (<i><a href="disc.html#dupins">see Discussion</a></i>),
      not as insertion. </p>    <hr>    <h3><a name="var"></a>Variability of short sequence repeats<img src="new.gif"        height="12"        border="0"        width="31"></h3>    <p>Variability of short sequence repeats are described as p.Gln6(3_6); the      description indicates that a stretch of Glutamines (Gln, Q) is present,      starting at amino acid position 6 (e.g. in MKMGHQQQCC), which is found      with a variable length from 3 to 6 in the population</p>    <p><i><b>NOTE:</b></i> the underscore is used to indicate the range (3 to 6      times).</p>    <hr>    <h3><a name="indelp"></a>Deletion/insertions (indels)</h3>    <p>Deletion/insertions (indels) replace one or more amino acid residues with      one or more other amino acid residues. Deletion/insertions are described      using "<b><i>delins</i></b>" as a deletion followed by an insertion after      an indication of the amino acid(s)deleted separated by a <i><b>"_"</b></i>      (<i>underscore</i>, <i><a href="disc.html#indel">see Discussion</a></i>)<em>.        <strong><a name="fs"></a>Frame shifts </strong></em>are a <em><strong>special
          type of amino acid deletion/insertion</strong></em> affecting an amino      acid <em><strong>between</strong></em> the first (<em>initiation, ATG</em>)      and last codon (<em>termination, stop</em>), replacing the normal      C-terminal sequence with one encoded by <strong><em>another reading frame</em></strong>      (specified <em><small><font color="#FF0000"><strong>2013-10-11</strong></font></small></em>).
      A frame shift is described using "<em><strong>fs</strong></em>" after the      first amino acid affected by the change. Descriptions either use a <i><b>short</b>        ("fs") or <b>long</b> ("fsTer#") <b>description</b></i>. The      description of frame shifts does not include the deletion at protein level      from the site of the frame shift to the natural end of the protein (stop      codon). The inserted amino acid residues are not described, only the total      length of the new shifted frame is given (i.e. including the first amino      acid changed).<br>      <i><b>NOTE:</b> <a href="disc.html#frameshift">typing error in den          "Dunnen &amp; Antonarakis (2000)"</a></i>. The suggestion to use <i><b>"&gt;"</b></i>      to indicate <b><i>"delins"</i></b> in frame shift descriptions has been      retracted. <em><br>      </em><i><strong>NOTE:</strong>&nbsp; </i>when <em><strong>one</strong></em>      nucleotide is replaced by <em><strong>one</strong></em> other nucleotide      the change is called a <a href="#sub"><strong><em>substitution</em> </strong></a>    </p>    <ul>      <li><strong>in-frame</strong>        <ul>          <li>p.(Cys28_Lys29delinsTrp) indicates RNA nor protein was analysed            but the predicted change is a 3 bp deletion that affects the codons            for Cysteine-28 and Lysine-29, substituting them for a codon for            Tryptophan</li>          <li>p.Cys28delinsTrpVal denotes a 3 bp insertion in the codon for            Cysteine-28, generating codons for Tryptophan (Trp, W) and Valine            (Val, V)</li>          <li>p.(Pro578_Lys579delinsLeuTer) is a deletion-insertion variant            resulting from the change c.1733_1735delinsTTT. The predicted            consequence of variant c.1732_1794del is p.(Pro578_Gln598del). <span              style="font-style: italic;">Note
              that although the proteins resulting from these changes are              identical, their HGVS description is different.</span><br>            <span style="font-weight: bold; font-style: italic;">NOTE:</span>            these example derive from the SLC34A3 gene (NM_080877.2)</li>        </ul>      </li>      <li><strong><a name="fs"></a>frame shifts<br>        </strong>are described using the format <i>p.Arg97Glyfs*26</i>        (alternatively <i>p.Arg97GlyfsTer26</i>, or short <em>p.Arg97fs</em>)        where <i>Arg97Gly</i> describes the change of the first amino acid        affected (Arg97 replaced by a Pro residue), <b><i>"fs"</i></b>        indicating the <em><strong>frame shift</strong></em> and <i>*16</i>        giving the position of the translation termination codon (stop codon) in        the new reading frame.<br>        <em><strong>NOTE:</strong></em>&nbsp; the description does not include a        description of the deletion from the site of the change to the        C-terminal end of the protein (stop codon) like <i>p.Arg97_Leu833delinsGlyfsTer26)</i>        nor a specific description of the inserted amino acid residues.<br>        <i><b>NOTE:</b></i> the shifted reading frame includes the first new        amino acid (<i>Gly</i>) and encounters a translation termination codon        at position 26 (<em>Ter26</em> or <em>*26</em>)<i>. </i>The shifted        reading frame is thus open for 'Ter26-1' amino acids<i>.</i>        <ul>          <li><u>short description</u>&nbsp; -&nbsp; uses "fs" only, e.g. <i>p.Arg97fs</i></li>          <li><u>long description</u>&nbsp; -&nbsp; uses "fsTer#" (alternatively            "fs*#") (<a href="disc.html#frameshift"><i>see Discussion</i></a>)            <ul>              <li>includes the change occurring at the site of the frame shift,                e.g. <i>p.Arg97Gly</i></li>              <li>"fsTer#" (or "fs*#") indicates at which position the new                reading frame encounters a translation termination (stop) codon                stop (Ter# / *#). The position of the stop in the new reading                frame is calculated starting from the first amino acid changed                by the frame shift, and ending at the first stop codon (fsTer#                or fs*#)</li>            </ul>          </li>          <li><u>Examples</u>            <ul>              <li><i>p.Arg97ProfsTer23 (alternatively p.Arg97Profs*23; <b>short</b>                  p.Arg97fs)</i> denotes a frame shifting change with                Arginine-97 as the first affected amino acid, replacing it for a                Proline and creating a new reading frame ending at a stop at                position 23 (counting starts with the Proline as amino acid 1)</li>              <li><i>p.Glu5Valfs*5</i> describes a frame shifting insertion (do                not use <i>p.Glu5Valins2fs*3</i>)</li>              <li><i>p.(Tyr4*)</i> indicates RNA nor protein was analysed but                the predicted consequence of the change c.12delC in the sequence                ATG-GAT-GCA-TA<i><b>C</b></i>-GTG-ACG to ATG-GAT-GCA-<u>TA.-G</u>                <u>TG-A</u> CG is a Tyr to translation termination codon.</li>              <li><i>p.Asp2Metfs*4 (alternatively p.Asp2fs) </i>describes the                consequence of the change c.4delG in the sequence ATG-<b><i>G</i></b>AT-GCA-TAC-GTG-ACG
                to ATG- <u>.AT-G</u> <u>CA-T</u> <u>AC-G</u> <u>TG-A</u> CG.</li>              <li><i>p.Glu5Valfs*5 (alternatively p.Glu5fs) </i>describes the                consequence of the change c.6_13dup in the sequence ATG-GA<b><i>T-GCA-TAC-G</i></b>AG-ATG-AGG&nbsp;
                to ATG-GAT-GCA-TAC-<u>G<i><b>T-G</b></i></u><i><b> <u>CA-T</u>                    <u>AC-G</u> </b></i><u>AG-A</u> <u>TG-A</u> GG.</li>              <li><em><strong><font color="#FF0000"><small>date 2012-11-01</small></font>                  </strong>p.Ile327Argfs*? (a</em><i>lternatively p.Ile327fs) </i>describes
                the consequences of a frame shifting change (e.g. a 1-nucleotide                insertion) with Isoleucine-327 as the first affected amino acid,                replacing it for an Arginine and creating a new reading frame                which does not encounter a new stop codon (<a href="FAQ.html#nostop"><i>see
                    FAQ</i></a>).</li>            </ul>          </li>        </ul>      </li>    </ul>    <p><i><b>NOTE:</b> </i>the changes observed should be described on protein      level and not try to incorporate any knowledge regarding the change at      DNA-level&nbsp;(<i><a href="#protonly">see Recommendation</a></i>). Thus,      <i>p.His150Hisfs*10</i> is not correct, but <i>p.Gln151Thrfs*9</i> is.</p>    <hr>    <h3><a name="extp"></a>Extensions<img src="new.gif" height="12" border="0" width="31"></h3>    <p>Extensions affect either the first (<em>start, translation initiation,        N-terminus. ATG</em>) or last codon (<em>translation termination, stop</em>)      and as a consequence extend the protein sequence N- or C-terminally with      one or more amino acids. Extensions are described using "<em><strong>ext</strong></em>"      after a description of the change at the first amino acid affected and      followed by a description of the position of the new translation      initiation or termination codon. </p>    <ul>      <li><strong>new translation initiation site</strong><img src="new.gif" height="12"          border="0"          width="31">&nbsp;
        (<a href="disc.html#p-initiation"><i>see Discussion</i></a>)<em><strong><font              color="#FF0000"><small>&nbsp;
                date 2012-08-31</small></font> </strong></em><br>        a change affecting the translation initiation codon (Met-1) introducing        a <em><strong>new upstream initiation codon</strong></em> extending the        N-terminus of the encoded protein described using "<em><strong>ext-#</strong></em>"        where "<em>-#</em>" is the position of the new initiation codon (<em>Met-#</em>)        <ul>          <li><i>p.Met1ext-5</i>&nbsp; -&nbsp; a variant in the 5' UTR activates            a new upstream translation initiation site starting with amino acid            <em>Met-5</em> (Methionine -5)</li>          <li><i>p.Met1Valext-12</i>&nbsp; -&nbsp; amino acid <em>Met1</em> is            changed to <em>Val</em> activating an upstream translation            initiation site at position -12 (Methionine -12)<br>            <em><strong>NOTE:</strong></em> recently modified from <i>p.Met1ValextMet-12</i>&nbsp;
            (<a href="disc.html#p-initiation"><i>see Discussion</i></a>)</li>        </ul>      </li>      <li><b>no-stop change</b><i><img src="new.gif" height="12" border="0" width="31">&nbsp;
          (substitution in stop codon)<br>        </i>a change affecting the translation termination codon (Ter/*)        introducing a <em><strong>new downstream termination codon</strong></em>        extending the C-terminus of the encoded protein described using "<i><b>extTer#</b></i>"        (alternatively "<em>ext*#</em>") where "<em>#</em>" is the position of        the new stop codon (<em>Ter# / *#</em>)        <ul>          <li><i>p.*110Glnext*17 </i>(alternatively <i>p.Ter110GlnextTer17 </i>or<i>              p.*110Qext*17</i>)<i> </i>describes a variant in the stop codon            (Ter/*) at position 110, changing it to a codon for Glutamine (Gln,            Q) and adding a tail of new amino acids to the protein's C-terminus            ending at a new stop codon (Ter17/*17)</li>          <li><strong><em><font color="#FF0000"><small>date 2012-11-01</small></font>              </em></strong><i>p.*327Argext*? </i>(alternatively<i>              p.Ter327ArgextTer?</i> or<i> p.*327Rext*?</i>)<i> </i>describes a            variant in the stop codon (Ter/*) at position 327, changing it to a            codon for Arginine (Arg, R) and adding a tail of new amino acids of            unknown length since the shifted frame does not contain a new stop            codon (<a href="FAQ.html#nostop"><i>see FAQ</i></a>).</li>        </ul>      </li>    </ul>    <hr>    <h3><a name="2in"></a>More changes in one individual</h3>    <p>Two or more changes in one individual are described by combining the      changes, per chromosome (maternal and paternal), between square brackets (<i><b>"[;];[;]"</b></i>)      and using a semicolon <span style="font-style: italic;">(";"</span>) as      separator:&nbsp; <span style="font-style: italic;">[first change        maternal; second change maternal] ; [first change paternal; second        change paternal]</span>". When changes are in different genes on      different chromosomes a space (<span style="font-style: italic;">" "</span>)      is used to separate the different chromosomes (<i><b>"[;] [;]"</b></i>).<span        style="font-style: italic;"></span></p>    <ul>      <li><b>Two changes in one gene on one chromosome</b>        <ul>          <li>deriving from <i><u>two independent changes at DNA level</u></i>            are described as "[first change;second change]" (<a href="file:///Users/Boomheide/Files/WWW/HGVS/disc.html#2in1"><i>see
                Discussion</i></a>).&nbsp;            <ul>              <li>p.[(Ala25Thr; Gly28Val)] indicates two predicted changes                derived from one chromosome (RNA or protein not analysed); amino                acid Alanine25 to Threonine and Glycine-28 to Valine</li>            </ul>          </li>          <li>deriving from <u><i>one change at DNA level</i></u> that has more            than one effect on RNA/protein level are described as "[first            change, second change]" (<a href="file:///Users/Boomheide/Files/WWW/HGVS/disc.html#moreRNAs"><i>see
                Discussion</i></a>).&nbsp;            <ul>              <li>p.[Asn26His, Ala25_Gly29del] describes two protein changes                deriving from one change on a chromosome (c.76A&gt;C at DNA                level) resulting in two transcripts (RNA level r.[76a&gt;c,                73_88del]) yielding two predicted proteins, one where amino acid                Asparagine25 changes to Histidine and one with a deletion of                amino acids Asparagine25 to Glycine29</li>            </ul>          </li>        </ul>      </li>      <li><b>Two changes in one gene on different chromosomes</b> (e.g. in <i>recessive
          diseases</i>)<br>        p.[Ala25Thr];[Gly28Val] describes two changes derived from a gene on        each chromosome (one paternal, one maternal); predicted change amino        acid Alanine25 to Threonine on one chromosome and Glycine28 to Valine on        the other chromosome (RNA or protein analysed)<br>        <h4>Examples</h4>      </li>      <ul>        <li>p.[(Ala25Thr)];[(Gly28Val)] describes two changes derived from one          gene on each chromosome (one paternal, one maternal); predicted change          amino acid Alanine25 to Threonine on one chromosome and Glycine28 to          Valine on the other chromosome (RNA or protein not analysed)<br>          <em><strong>NOTE:</strong></em> the description          p.([Ala25Thr];[Gly28Val]) should not be used</li>        <li>p.[Ala25Thr];[(Pro323Leu)] described a predicted change of amino          acid Alanine25 to Threonine derived from one chromosome (RNA or          protein analysed) and Proline323 to Leucine derived from the other          chromosome (RNA or protein not analysed)</li>        <li>p.[Ala25Thr];[?] describes a change of amino acid Alanine-25 to          Threonine derived from one chromosome and an unknown change iderived          from the other (RNA or protein not analysed)<br>          <b><i>NOTE:</i></b> "<i>unknown change in the other allele</i>" does          not only mean that no DNA-change was detected in the other chromosome          but includes cases where the consequence of a detected change is          unclear or can not be predicted (e.g. the consequence of a change at          the splice site)</li>        <li>p.[Ala25Thr];[=] denotes a change of amino acid Alanine25 to          Threonine derived from one chromosome and a normal sequence (indicated          by "=") of the other chromosome (<a href="FAQ.html#2alleles"><i>see              FAQ</i></a>)</li>      </ul>      <li><br>      </li>      <li><i><img src="new.gif" height="12" border="0" width="31"><b>Two            sequence changes in one gene with chromosomes unknown</b> are          described as "[change 1(;)change 2]" (<a href="disc.html#2unknown">see            Disucssion</a>).</i>        <ul>          <li><span style="font-style: italic;">p.[Ala25Thr(;)Pro323Leu]              describes that two changes were identified in one individual              (amino acid Alanine25 to Threonine and Proline323 to Leucine, RNA              or protein analysed), but it is not known whether these changes              are on the same chromosome (in cis) or on different chromosomes              (in trans)</span></li>          <li><span style="font-style: italic;">p.[(Ala25Thr(;)Pro323Leu)]              describes that two changes were identified in one individual              (amino acid Alanine25 to Threonine and Proline323 to Leucine, RNA              nor protein analysed), but it is not known whether these changes              are on the same chromosome (in cis) or on different chromosomes              (in trans). Alternatively p.[(Ala25Thr)(;)(Pro323Leu)] can be              used.</span></li>        </ul>      </li>      <li><img src="new.gif" height="12" border="0" width="31"><b>Mosaicism</b>        is described using "/"        <ul>          <li><i>p.[Arg83=/Arg83Ser] describes a somatic case where a chromosome              in some cells contains a normal sequence (p.Arg83=), while other              cells contain a Ser at this position (p.Arg83Ser)<br>            </i></li>        </ul>      </li>      <li><img src="new.gif" height="12" border="0" width="31"><b>Chimerism</b>        is described using "//"        <ul>          <li><i>p.[Arg83=//Arg83Ser] describes a chimeric organism where a              chromosome in some cells contain a normal sequence (Arg83=), while              other cells contain anotehr chromosome with Ser at this position </i><i>(p.Arg83Ser)</i></li>        </ul>      </li>    </ul>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">Homepage</a>      | <a href="checklist.html">Check-list</a> | <a href="standards.html">Symbols,
        codons, etc.</a> |<br>      | <b>Recommendations:</b>&nbsp; <a href="recs-DNA.html">DNA</a>,&nbsp; <a        href="recs-RNA.html">RNA</a>,&nbsp;
      <a href="recs-prot.html">protein</a>, <a href="uncertain.html">uncertain</a>      |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="history.html">History</a>      |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef</a>,&nbsp;
      <a href="examplesDNA.html">DNA</a>,&nbsp; <a href="examplesRNA.html">RNA</a>,&nbsp;
      <a href="examplesAA.html">protein</a> | </p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                size="-5"><a                  href="http://www.HGVS.org/copyright.html">Copyright
                  ©</a> </font><font size="1">HGVS 2010 All Rights Reserved<br>                Website Created by Rania Horaitis, Nomenclature by J.T. Den                Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>              </font></p>          </td>        </tr>      </tbody>    </table>  </body></html>