<html>
<!--
	Author: 	Dr. Johan T. den Dunnen
		Date: 	02-01-2005	
		URL: 	recs-DNA.html
-->

<head>
<title>HGVS recommendations: general, DNA level</title>
<style type="text/css">

<!--  

h2 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #ff9900;

}

h3 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h4 {
 font-family : Verdana,Arial,Helvetica,Geneva,sans-serif;
 color : #0099cc;

}

h5 {
font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;
color : #ffffff;

}

.norm {

 font-weight : normal;

 }
 -->
</style>
</head>

<body text="#000000" bgcolor="#FFFFFF" link="#0000FF" vlink="#9999ff">

<table width="100%" border="0" cellspacing="0" cellpadding="10" bgcolor="#CCFF99">
  <tr>
    <td width="25%" valign="top" bgcolor="#FFFFFF"><a href="http://www.HGVS.org"><img
    src="small.gif" border="0" width="155" height="94"></a></td>
    <td width="75%" valign="CENTER" bgcolor="#FFFFFF" bordercolor="#FFFFFF"><h3 align="center">Recommendations
    for the description of DNA sequence variants - v2.0 </h3>
    </td>
  </tr>
</table>

<table width="100%" border="0" cellspacing="0" cellpadding="10">
  <tr>
    <td width="100%" valign="BOTTOM" bgcolor="#0099cc"><h4 align="center"><font size="3"
    color="#FFFFFF"><i><br>
    Last modified November 11, 2013</i></font></h4>
    </td>
  </tr>
</table>

<p align="center"><i>Since references to WWW-sites are not yet acknowledged as citations,
please mention <a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den Dunnen JT
and Antonarakis SE (2000). Hum.Mutat. 15: 7-12</a> when referring to t</i>separated by at
least one nucleotide<i>hese pages.</i></p>

<hr>

<h2>Contents</h2>

<ul>
  <li><b>Recommendations</b> <ul>
      <li><a href="recs.html">general recommendations</a> <ul>
          <li><a href="#number">nucleotide numbering</a></li>
        </ul>
      </li>
      <li><b>changes at DNA level</b> <ul>
          <li><a href="#DNA">general</a></li>
          <li><a href="#sub">substitutions</a></li>
          <li><a href="#del">deletions</a></li>
          <li><a href="#dup">duplications</a></li>
          <li><a href="#ins">insertions</a></li>
          <li><a href="#indel">insertion/deletions&nbsp; (indels)</a></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="#inv">inversions</a></i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="#con">conversions</a></i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="#tra">translocations</a></i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="#2in">two or more
            changes in one allele</a>&nbsp; (incl. mosaicism, chimerism)</i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="#var">repeated
            sequences (incl. short sequence repeats)</a></i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="uncertain.html">uncertainties</a></i></li>
          <li><i><img border="0" src="new.gif" width="31" height="12"><a href="#com">complex</a></i></li>
        </ul>
      </li>
      <li><a HREF="recs-RNA.html">changes at RNA level</a></li>
      <li><a href="recs-prot.html">changes at protein level</a></li>
    </ul>
  </li>
  <li><b>Explanations / examples</b> <ul>
      <li><a href="quickref.html">Quick Reference</a></li>
      <li><a HREF="examplesDNA.html">changes at DNA level</a></li>
      <li><a HREF="examplesRNA.html">changes at RNA level</a></li>
      <li><a HREF="examplesAA.html">changes at protein level</a></li>
    </ul>
  </li>
</ul>

<hr>

<h2><a NAME="DNA"></a>DNA level</h2>

<p><em>(suggestions extending the <a
href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">published</a>
recommendations are </em><i><img border="0" src="new.gif" width="31" height="12"></i><em>
in italics)</em></p>

<hr>

<ul>
  <li><b>nucleotides<br>
    </b>description of nucleotides at DNA level follows the recommendations of the <a
    href="http://www.chem.qmul.ac.uk/iubmb/misc/naseq.html#500">IUPAC-IUBMB</a>.Nucleotides
    are designated by the bases, in upper case, A (adenine), C (cytosine), G (guanine), T
    (thymidine), including those for uncertain nucleotides like Y (pYrimidine) and R (puRine),
    (<i><a href="standards.html#nucleotide">see Standards</a></i>). </li>
  <li><a NAME="number"></a><b>nucleotide numbering</b><i>&nbsp; (for details and examples <a
    href="refseq.html">see Reference Sequence discussions</a>)</i> <ul>
      <li><u>coding DNA reference sequence</u>&nbsp; (<i><a href="refseq_figure.html">see Examples
        and Figure</a></i>) <ul>
          <li>there is no nucleotide 0 </li>
          <li>nucleotide 1 is the A of the ATG-translation initiation codon </li>
          <li>the nucleotide <b>5' of the ATG-translation initiation</b> codon is -1, the previous -2,
            etc.</li>
          <li><i><img border="0" src="new.gif" width="31" height="12">the nucleotide <b>3' of the
            translation stop codon</b> is *1, the next *2, etc.</i></li>
          <li><b>intronic nucleotides</b> (coding DNA reference sequence only)<ul>
              <li><u>beginning of the intron;</u> the number of the last nucleotide of the preceding exon,
                a plus sign and the position in the intron, like c.77+1G, c.77+2T, ....</li>
              <li><u>end of the intron;</u> the number of the first nucleotide of the following exon, a
                minus sign and the position upstream in the intron, like ..., c.78-2A, c.78-1G.</li>
              <li>in the <u>middle of the intron,</u> numbering changes from &quot;c.77+..&quot; to
                &quot;c.78-..&quot;; for introns with an uneven number of nucleotides the central
                nucleotide is the last described with a &quot;+&quot; <i>(<a href="refseq.html#IVS">see
                Discussion</a>)</i></li>
              <li><i><b>NOTE:</b></i> the format c.IVS1+1G and c.IVS1-2G should not be used <i>(<a
                href="refseq.html#IVS">see Discussion</a>)</i> </li>
            </ul>
          </li>
        </ul>
      </li>
      <li><u>genomic reference sequence</u>&nbsp; (<i><a href="refseq_figure.html">see Examples
        and Figure</a></i>) <ul>
          <li>nucleotide numbering starts with 1 at the first nucleotide of the sequence<br>
            <b><i>NOTE:</i></b> the sequence should include all nucleotides covering the sequence
            (gene) of interest and should start well 5' of the promoter of a gene</li>
          <li>no +, - or other signs are used</li>
          <li>when the complete genomic sequence is not known, a coding DNA reference sequence should
            be used</li>
        </ul>
      </li>
      <li>for all descriptions the <b>most 3' position</b> possible is arbitrarily assigned to
        have been changed (<i><a href="#except">see Exception</a></i>) </li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="sub"></a><b>Substitutions</b></h3>

<p>A nucleotide <b>substitution</b> is a sequence change where one nucleotide is replaced
by one other nucleotide (<a href="standards.html#define"><i>see Standards - Definition</i></a>).
Nucleotide substituions are described using a <i><b>&quot;&gt;&quot;</b></i>-character
(indicating &quot;changes to&quot;).&nbsp;<br>
<b><i>NOTE:</i></b> changes involving two or more consecutive nucleotides are described as
deletion/insertions (indels, <i><a href="#indel">see Deletion/insetions</a></i>). 

<ul>
  <li>c.76A&gt;C denotes that at nucleotide 76 an A is changed to a C</li>
  <li>c.-14G&gt;C denotes a G to C substitution 14 nucleotides 5' of the ATG translation
    initiation codon</li>
  <li>c.88+1G&gt;T denotes the G to T substitution at nucleotide +1 of an intron (in the
    coding DNA positioned between nucleotides 88 and 89)</li>
  <li>c.89-2A&gt;C denotes the A to C substitution at nucleotide -2 of an intron (in the
    coding DNA positioned between nucleotides 88 and 89)</li>
  <li>c.*46T&gt;A denotes a T to A substitution 46 nucleotides 3' of the translation
    termination codon</li>
  <li>the description c.76_77delinsTT is preferred over c.[76A&gt;T; 77G&gt;T]<br>
    <i><img border="0" src="new.gif" width="31" height="12"><b>NOTE:</b></i>&nbsp; based on
    the definition of a substitution (<i><a href="standards.html#define">see Standards -
    Definition</a>; one nucleotide replaced by one other nucleotide</i>) this change can not
    be described as a substitution (like c.76_77AG&gt;TT or c.76AG&gt;TT)</li>
</ul>

<p><em><b>NOTE:</b></em> it is not correct to describe &quot;<i>polymorphisms</i>&quot; as
c.76A/G<em> (</em><i><a HREF="disc.html#polymorphism">see Discussion</a>)</i>.&nbsp;</p>

<hr>

<h3><a NAME="del"></a>Deletions</h3>

<p>A nucleotide deletion is a sequence change where one or more nucleotides are removed (<i>see
<a href="standards.html#define">see Standards - Definition</a></i>). Deletions are
described using <i><b>&quot;del&quot;</b></i> after an indication of the first and last
nucleotide(s) deleted, separated by a &quot;_&quot; (<i>underscore</i>). For all
descriptions the <em><b>most 3' position</b></em> possible is arbitrarily assigned to have
been changed.<br>
<i><b>NOTE: </b>to discriminate known variable sequences from other changes it is
recommended to describe individual alleles differing from the reference sequence like
g.210T[5] (preferred over g.210_211delTT) or g.121T[9] (preferred over g.210_211dupTT) </i>(<i><a
href="#var">see Repeated sequences</a></i>).&nbsp; 

<ul>
  <li>c.76_78del (alternatively c.76_78delACT) denotes a ACT deletion from nucleotides 76 to
    78</li>
  <li>deletions with <em><strong>uncharacterised breakpoints</strong></em> (<i><a
    HREF="uncertain.html">see Uncertainties</a></i>) <ul>
      <li>c.88-?_923+?del denotes an exonic deletion starting at an unknown position in the intron
        5' of coding DNA nucleotide 88 and ending at an unknown position in the intron 3' of
        coding DNA nucleotide 923</li>
      <li>c.(?_-30)_(*220_?)del denotes the deletion of the entire gene (coding DNA reference
        sequence running from -30 (cap site) to *220 (polyA-addition site)</li>
      <li><img border="0" src="new.gif" width="31" height="12">c.88+101_oGJB2:c.355-1045del
        denotes a deletion which ends in the flanking GJB2 gene at position 355-1045 (in the
        intron between nucleotides 354 and 355) on the reverse strand (the genes are thus located
        and fused in opposite transcriptional directions,<em> </em><i><a HREF="disc.html#fusion">see
        Discussion</a></i>)</li>
    </ul>
  </li>
  <li><a NAME="except"></a>for all descriptions the <b>most 3' position</b> possible is
    arbitrarily assigned to have been changed (<a href="FAQ.html#most3"><i>see FAQ</i></a>); <ul>
      <li>ACTT<b>TGT</b>GCC to&nbsp;ACTTGCC is described as c.5_7del (c.5_7delTGT, not as
        c.4_6delTTG)</li>
      <li>ctt<b>t</b>agGCATG to cttagGCATG in an intron is described as c.301-3delT (not as
        c.301-5delT)</li>
      <li>TCACTG<b>TCTG</b>CGGTAATC to TCACTG CGGTAATC is described as c.7_10del (c.7_10delTCTG)
        and not as c.4_7del (c.4_7delCTGT).</li>
      <li>AAAG<b>AAGAG</b>GAG to AAAG GAG is described as c.5_9del (c.5_9delAAGAG) and not as
        c.3_7del (c.3_7delAGAAG)</li>
      <li><b><i>Exceptions</i></b> <ul>
          <li>using a <i>coding DNA reference sequence</i> there is <b><i>an exception</i></b> to the
            rule around exon/intron and exon/exon borders when identical nucleotides flank the
            exon/intron or exon/exon border; <ul>
              <li>when the exon 3/intron 3 border is ..CAGgtg.. and RNA analysis shows no effect on
                splicing but a deletion of a G the change ..CAGgtg.. to ..CAgtg.. is described as c.3delG <i><b>and
                not</b></i> c.3+1delG.</li>
              <li>when exon 3 ends with ..CAA.. and exon 4 starts with ..ACG.. and the sequence of genomic
                DNA shows that the last A-nucleotide of exon 3 is deleted (and not the first A-nucleotide
                in exon 4), the deletion changing ..CAAACG.. to ..CAACG.. is described as c.3delA <b><i>and
                not</i></b> c.4delA</li>
            </ul>
          </li>
          <li>c.1210-12T(5_9) (<i>not c.1210-6T(5_9)</i>) describes the variable stretch of 5 to 9
            T-residues in intron 9 of the CFTR gene. The most commonly used CFTR coding DNA reference
            sequence contains a stretch of 7 T's (<i><a href="#var">see Repeated sequences</a></i>).&nbsp;</li>
        </ul>
      </li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="dup"></a>Duplications</h3>

<p>Duplications are designated by <i><b>&quot;dup&quot;</b></i> after an indication of the
first and last nucleotide(s) duplicated. It should be noted that the description<strong> <em>&quot;dup&quot;</em></strong>
(<em><a href="standards.html#define">see Standards</a></em>) may <strong><em>by definition</em></strong>
only be used when the sequence copy is directly 3'-flanking the original copy.&nbsp; For
all descriptions the <em><b>most 3' position</b></em> possible is arbitrarily assigned to
have been changed. For the addition of more then 1 copy (3, 4, 5, etc.) <i><a href="#var">see
Repeated sequences</a></i> and<em> <a HREF="disc.html#tri">see Discussion</a>.</em><br>
<i><b>NOTE: </b>to discriminate known variable sequences from other changes it is
recommended to describe individual alleles differing from the reference sequence like
g.210T[5] (preferred over g.215_216del) or g.210T[9] (preferred over g.215_216dup) </i>(<i><a
href="#var">see Repeated sequences</a></i>).&nbsp; 

<ul>
  <li>duplicating insertions should be described as duplications (<a HREF="disc.html#dupins"><i>see
    Discussion</i></a>) <ul>
      <li>g.5dupT (or g.5dup, <em>not g.5_6insT</em>) denotes a duplication (<i>&quot;insertion&quot;</i>)
        of the T nucleotide at position 5 in the genomic reference sequence changing ACTCTGTGCC to
        ACTCT<u><strong>T</strong></u>GTGCC</li>
      <li>g.7dupT (or g.7dup, <em>not g.5dupT, not g.7_8insT</em>) denotes a duplication (<i>&quot;insertion&quot;</i>)
        of the T nucleotide at position 7 in the genomic reference sequence changing AGACTTTGTGCC
        to AGACTTT<u><strong>T</strong></u>GTGCC</li>
      <li>g.7_8dup (or g.7_8dupTG, <em>not g.5_6dup, not g.8_9insTG</em>) denotes a TG duplication
        in the TG-tandem repeat sequence changing ACTTTGTGCC to ACTTTGTG<strong><u>TG</u></strong>CC</li>
      <li>g.7_8[4] (or g.5_6[4], or g.5TG[4], <em>not g.7_10dup</em>) is the preferred description
        of&nbsp; the addition of two extra TG's to the variable TG repeated sequence changing
        ACTTTGTGCC to ACTTTGTG<strong><u>TGTG</u></strong>CC (<i><a href="#var">see Repeated
        sequences</a></i>)</li>
    </ul>
  </li>
  <li>c.77_79dup (or c.77_79dupCTG) denotes that the three nucleotides 77 to 79 are duplicated
    (present twice)</li>
  <li><i>c.88-?_301+?dup </i>denotes the duplication of exons 3 to 4, starting at an unknown
    position in intron 2 (upstream of coding DNA nucleotide 88) and ending at an unknown
    position in intron 4 (downstream of coding DNA nucleotide 301). Using this description
    exons 2 and 5 have been shown not to be duplicated (<em><a HREF="uncertain.html">see
    Uncertainties</a></em>)<br>
    <em><strong>NOTE:</strong> the description<strong> &quot;dup&quot;</strong> (<a
    href="standards.html#define">see Standards</a>) may <strong>by definition</strong> only be
    used when the additional copy is directly 3'-flanking of the original copy (tandem
    duplication). In many cases there will be <strong>no experimental proof</strong>, the
    additional copy may be anywhere in the genome (i.e. inserted / transposed). It has been
    suggested that, unless there is experimental evidence, exonic duplications should be
    described using the format, i.e. c.88-?_301+?<b>(2)</b> (<a href="recs-DNA.html#var">see
    Recommendations</a>).</em></li>
  <li><a NAME="tri"></a><i><img border="0" src="new.gif" width="31" height="12"></i>c.1032-?_1357+?(3)&nbsp;
    denotes the presence of two additional copies of an exon, starting at an unknown position
    in the flanking upstream intron (upstream of coding DNA nucleotide 1032) and ending at an
    unknown position in the flanking downstram intron (downstream of coding DNA nucleotide
    1357). The description, using &quot;(3)&quot; indicates there is no proof that the three
    copies are in tandem<i> (<a href="#var">see Repeated sequences</a>, </i><em><a
    HREF="uncertain.html">see Uncertainties</a></em>)</li>
  <li><i><img border="0" src="new.gif" width="31" height="12"></i>c.1032-?_1357+?[3]&nbsp;
    denotes a <b>direct triplication</b> of an exon, starting at an unknown position in the
    flanking upstream intron (upstream of coding DNA nucleotide 1032) and ending at an unknown
    position in the flanking downstram intron (downstream of coding DNA nucleotide 1357)<i> (<a
    href="#var">see Repeated sequences</a>)</i></li>
</ul>

<hr>

<h3><a NAME="ins"></a>Insertions</h3>

<p>Insertions are designated by <i><b>&quot;ins&quot;</b></i> after an indication of the
nucleotides flanking the insertion site, followed by a description of the nucleotides
inserted. Duplicating insertions should be described as duplications (<i><a
HREF="disc.html#dupins">see Discussion</a></i>), not as insertion. For large insertions
the number of inserted nucleotides should be mentioned, together with an accession.version
number referring to a sequence database file containing the complete inserted
sequence.&nbsp; 

<ul>
  <li>c.76_77insT denotes that a T is inserted between nucleotides 76 and 77 of the coding DNA
    reference sequence </li>
  <li>c.123+54_123+55insAB012345.2:g.76_420 denotes an intronic insertion ( between
    nucleotides c.123+54 and 123+55) of 345 nucleotides (nucleotides 76 to 420 like in GenBank
    file AB012345 version 2)<br>
    <em><strong>NOTE</strong>: descriptions like c.123+54_123+55ins345 and
    c.123+54_123+55insAlu are not allowed: &quot;ins345&quot; and &quot;insAlu&quot; are not
    specified and the description can not be used to reconstruct the exact change described.</em></li>
</ul>

<hr>

<h3><a NAME="indel"></a>Deletion / insertions (indels)</h3>

<p>Deletion/insertions of two or more consecutive nucleotides (indels) are described as a <strong><em>deletion
followed by an insertion</em></strong> (<i><a HREF="disc.html#indel">see Discussion</a></i>).

<ul>
  <li>c.112_117delinsTG (alternatively c.112_117delAGGTCAinsTG) denotes the replacement of
    nucleotides 112 to 117 (AGGTCA) by TG</li>
  <li>c.113delinsTACTAGC (alternatively c.113delGinsTACTAGC) denotes the replacement of
    nucleotide 113 by 7 new nucleotides (TACTACG)</li>
  <li>c.114_115delinsA (alternative c.[114G&gt;A; 115delT])</li>
</ul>

<hr>

<h3><a NAME="inv"></a><i><img border="0" src="new.gif" width="31" height="12"></i>Inversions</h3>

<p>Inversions are designated by <b>&quot;inv&quot;</b> after an indication of the first
and last nucleotides affected by the inversion. 

<ul>
  <li><i>c.203_506inv (or 203_506inv304) denotes that the 304 nucleotides from position 203 to
    506 have been inverted</i></li>
</ul>

<hr>

<h3><a NAME="con"></a><i><img border="0" src="new.gif" width="31" height="12"></i>Conversions</h3>

<p><u>Conversions </u>are designated by <b>&quot;con&quot;</b> after an indication of the
first and last nucleotides affected by the conversion, followed by a description of the
origin of the new nucleotides (<a href="disc.html#con">see Discussion</a>). 

<ul>
  <li><em>g.123_678conNG_012232.1:g.9456_10011 describes a gene conversion replacing
    nucleotides 123 to 678 of the reference genomic sequence with nucleotides 9456 to 10011
    from the sequence as present in GenBank file NG_012232.1</em></li>
</ul>

<hr>

<h3><a NAME="tra"></a><i><img border="0" src="new.gif" width="31" height="12"></i>Translocations</h3>

<p>Translocations are described at the molecular level using the format
&quot;t(X;4)(p21.2;q34)&quot;, followed by the usual numbering, indicating the position
translocation breakpoint. The sequences of the translocation breakpoints need to be
submitted to a sequence database (Genbank, EMBL, DDJB) and the accession.version numbers
should be given (<a HREF="disc.html#translocation">see Discussion</a>). 

<ul>
  <li><i>t(X;4)(p21.2;q35)(c.857+101_857+102) denotes a translocation breakpoint in the intron
    between coding DNA nucleotides 857+101 and 857+102, joining chromosome bands Xp21.2 and
    4q34 </i></li>
</ul>

<hr>

<h3><a NAME="2in"></a>More changes in one individual</h3>

<p>Two or more changes in one individual are described by combining the changes, per
allele (chromosome) between square brackets (<i><b>&quot;[]&quot;</b></i>).</p>
<b>

<p>Changes in different alleles</b> (e.g. in <i><b>recessive diseases</b></i>) are
described as &quot;[change allele 1];[change allele 2]&quot;. Mixed descriptions like <b>c.</b>[76A&gt;C];<b>g.</b>[91C&gt;G]
should not be used. 

<ul>
  <li>c.[76A&gt;C];[76A&gt;C] denotes a homozygous A to C change at nucleotide 76</li>
  <li>c.[76A&gt;C];[(76A&gt;C)] denotes a homozygous A to C change at nucleotide 76, not
    confirmed by analysis of both parents, leaving the possibility of non-amplification of the
    second allele due to a primer mismatch or a deletion </li>
  <li>c.[76A&gt;C];[?] denotes a A to C change at nucleotide 76 in one allele and an unknown
    change in the other allele </li>
  <li>c.[76A&gt;C];[=] denotes a A to C change at nucleotide 76 in one allele and a normal
    sequence in the other allele (<a href="FAQ.html#2alleles"><i>see FAQ</i></a>) </li>
  <li><i><img border="0" src="new.gif" width="31" height="12">c.[76A&gt;C];[0] denotes a A to
    C change at nucleotide 76 in one allele and the absence of a sequence from the other
    allele (e.g. for a variant in a gene on the X-chromosome in a male where only one allele
    is present)</i> </li>
  <li><i><img border="0" src="new.gif" width="31" height="12">descri</i><em>ptions of sequence
    changes in different genes (e.g. for recessive diseases) are listed between square
    brackets, separated by a &quot;;&quot;-character and include a reference to the sequence
    (gene) changed; [DMD:c.76A&gt;C];[GJB:c.87delG] (<a HREF="disc.html#2in2genes">see
    Discussion</a>)</em> </li>
</ul>
<b>

<p>Two variations in one allele</b>, separated by at least one nucleotide, are described
as &quot;[first change ; second change]&quot;.&nbsp;For the description of haplotypes <i><a
href="disc.html#haplo">see Discussion</a></i>.<br>
<em><strong>NOTE: </strong>&quot;separated by at least one nucleotide&quot;</em> means the
description c.76_77delinsTT is preferred over c.[76A&gt;T; 77G&gt;T]. 

<ul>
  <li>c.[76A&gt;C; 83G&gt;C] denotes two changes in one allele; A to C change at nucleotide 76
    and a G to C change at nucleotide 83</li>
</ul>

<p><img border="0" src="new.gif" width="31" height="12"><em><strong>Mosaicism - two
different nucleotides in one position caused by somatic mosaicims</strong></em><i> are
described as &quot;[=/nucleotide 2]&quot; (<a HREF="FAQ.html#mosaic">see FAQ</a>)</i>. 

<ul>
  <li><i>c.[=/83G&gt;C] describes a mosaic case where at position 83 besides the normal
    sequence (a G, described as '=') also chromosomes are found containing a C (c.83G&gt;C)</i></li>
</ul>

<p><img border="0" src="new.gif" width="31" height="12"><em><strong>Chimerism - two
different nucleotides in one position caused by chimerism</strong></em><i> are described
as &quot;[=//nucleotide 2]&quot;</i> 

<ul>
  <li><i>c.[=//83G&gt;C] describes a chimeric case where at position 83 besides the normal
    sequence (a G, described as '=') also cells are found containing a C (c.83G&gt;C)</i></li>
</ul>

<p><img border="0" src="new.gif" width="31" height="12"><i><b>Two sequence changes with
alleles unknown</b> are described as &quot;[change allele 1 (;) change allele 2]&quot; (<a
HREF="FAQ.html#2unknown">see FAQ</a>)</i>. 

<ul>
  <li><i>c.[76A&gt;C(;)283G&gt;C] denotes that two changes were identified in one individual
    (an A to C change at nucleotide 76 and a G to C change at nucleotide 283), but it is not
    known whether these changes are in the same allele or in different alleles</i> </li>
</ul>

<hr>

<h3><a NAME="var"></a><a href="#del"><i><img border="0" src="new.gif" width="31"
height="12"></i></a>Repeated sequences</h3>

<p>A frequently occuring sequence change is the variability of repeated sequences. Within
this category we discriminate both small sequences (mono-, di-, tri-, etc nucleotide
repeats) as well as the much larger ones. Such changes are described using the format <em><strong>&quot;position-first-repeat-unit_[number]</strong></em>&quot;
(e.g. g.123_124[4]) where <em><strong>position-first-repeat-unit</strong> gives the
location of the first unit</em> of the variable sequence repeat and <em><strong>[number]</strong>
the number of units present</em> in the allele described. 

<ul>
  <li>the first unit of the repeat is preferably described based on position, like. g.123_124.
    <em>For short/simple repeats it is acceptable to include the content of the repeated unit,
    using the format &quot;position-first-nucleotide-repeat_content&quot; like g.123TG[4].</em>
    <strong>Do not use</strong> a mix of these descriptions like g.123_124TG[4]. This contains
    redundant information (123_124 and TG) with the danger of being in conflict. <em>Note that
    including the content of the sequence involved quikly gives descriptions which become too
    lengthy.</em></li>
  <li>the repeated sequences may have complex structures, consisting of a mix of several
    repeated sequences directly following each other. To describe these one should describe
    these using the individual repeated elements, like g.456TG[4]TA[9]TG[3] (or
    g.456_465[4]466_489[9]490_499[3].</li>
  <li>the same format can be used to describe the presence of multiple copies (triplication,
    quadruplication, etc.) of larger sequences, e.g. exons. Square brackets (<em><strong>&quot;[
    ]&quot;</strong></em>) should only be used when their is experimental evidence that the
    additional copies are in tandem on the same allele. When there is no such proof the
    presence of additional copies should be described using normal brackets (<em><strong>&quot;(
    )&quot;</strong></em>).<br>
    <em><strong>NOTE:</strong> the description<strong> &quot;dup&quot;</strong> (<a
    href="standards.html#define">see Standards</a>) may <strong>by definition</strong> only be
    used when the additional copy is directly 3'-flanking of the original copy (tandem
    duplication). In many cases there will be <strong>no experimental proof</strong>, the
    additional copy may be anywhere in the genome (i.e. inserted / transposed). It has been
    suggested that, unless there is experimental evidence, exonic duplications should be
    described using the format, i.e. c.88-?_301+?<b>(2)</b>.</em></li>
</ul>

<h4>Examples</h4>

<ul>
  <li>g.123_124[4] (<em>alternatively g.123TG[4]</em>) describes a sequence variable in the
    number of TG repeats where the first unit is present at position g.123_124 in the genomic
    reference sequence and the allele described contains 4 units.</li>
  <li>c.1210-12T(5_9) describes a famous variable stretch of 5 to 9 T-residues in intron 9 of
    the CFTR gene. The most commonly used CFTR coding DNA reference sequence contains a
    stretch of 7 T's. It is recommended to describe individual alleles differing from this
    reference sequence as c.1210-12T[5] (<i>not c.1210-7_1210-6delTT</i>) or c.1210-12T[9] (<i>not
    c.1210-7_1210-6dupTT</i>).<br>
    <i><b>NOTE: </b>the repeat should not be described as c.1210-6T(5_9)</i></li>
  <li>c.123+74TG(3_6) (<i>alternatively c.123+74_123+75(3_6)</i>) indicates that a TG
    di-nucleotide repeat is present, starting at nucleotide 74 in the intron following cDNA
    nucelotide c.123, which is found varying from 3 to 6 copies in the population <ul>
      <li><i>c.123+74TG[4];[5] denotes that a person carries a TG di-nucleotide repeat of length 4
        on one allele and of length 5 on the other allele</i></li>
    </ul>
  </li>
  <li>based on the FMR1 coding DNA Reference Sequence (GenBank <a
    href="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=NM_002024.3&amp;dopt=gb">NM_002024.3</a>),
    c.-158GGC(1000) describes the presence of an extended GGC-repeat of <i><b>about 1000 units</b></i><br>
    <b><i>NOTE:</i></b> &quot;(1000)&quot; is used to indicate uncertainties (<a
    href="uncertain.html"><i>see Uncertainties</i></a>) <ul>
      <li>c.-158GGC[79]&nbsp; describes the presence of an allele with an extended GGC-repeat of <i><b>exactly
        79 units</b></i></li>
    </ul>
  </li>
  <li><a NAME="tri"></a><i><img border="0" src="new.gif" width="31" height="12"></i>c.1032-?_1357+?(3)&nbsp;
    denotes the presence of two additional copies of an exon, starting at an unknown position
    in the flanking upstream intron (upstream of coding DNA nucleotide 1032) and ending at an
    unknown position in the flanking downstram intron (downstream of coding DNA nucleotide
    1357). The description, using &quot;(3)&quot; indicates there is no proof that the three
    copies are in tandem<i> (</i><em><a HREF="uncertain.html">see Uncertainties</a></em>)</li>
  <li><i><img border="0" src="new.gif" width="31" height="12"></i>c.1032-?_1357+?[3]&nbsp;
    denotes a <b>direct triplication</b> of an exon, starting at an unknown position in the
    flanking upstream intron (upstream of coding DNA nucleotide 1032) and ending at an unknown
    position in the flanking downstram intron (downstream of coding DNA nucleotide 1357)<br>
    <i><b>NOTE:</b> the use of tri (for triplication), qua (for quadruplication), etc. is not
    recommended</i> (<a HREF="disc.html#tri"><i>see Discussion</i></a>) </li>
  <li>g.1209_4523(12_45) denotes that a 3.3 kb repeat sequence of which the first copy is
    present in the genomic reference sequence from nucleotides 1209 to 4523 can be found
    repeated 12 to 45 times in the population <ul>
      <li><i>g.1209_4523[14];[23] denotes that a person carries a 3.3 kb repeat with 14 copies on
        one allele and of 23 copies on the other allele</i></li>
    </ul>
  </li>
</ul>

<hr>

<h3><a NAME="com"></a><img border="0" src="new.gif" width="31" height="12">Complex changes</h3>

<p>Sequence changes can be very complex, involving several changes at a specific location.
The description of such changes using the recommendations given above can become rather
complicated and at some point, although literally correct, effectively meaningless. In
such cases the recommendation is to submit the sequence that has been determined to
GenBank and to use the accession.version number in the description. 

<ul>
  <li><i>c.123_678conNM_004006.1:c.123_678 describes a gene conversion replacing nucleotides
    c.123 to c.4567 of the coding DNA sequence of the transcript of interest with nucleotides
    c.123 to c.678 from a transcript sequence as present in GenBank file NM_004006 (version 1)</i></li>
  <li><i>c.88+101_oGJB2:c.355-1045del denotes a deletion which ends in the flanking GJB2 gene
    at position 355-1045 (in the intron between nucleotides 354 and 355) on the reverse strand
    (the genes are thus located and fused in opposite transcriptional directions, <a
    HREF="disc.html#fusion">see Discussion</a>)</i></li>
  <li><i>c.123+54_123+55insAB012345.2:g.76_420 denotes an intronic insertion (between
    nucleotides c.123+54 and 123+55) of 345 nucleotides (nucleotides 76 to 420 like in GenBank
    file AB012345 version 2)</i></li>
</ul>

<hr>

<p align="center">| <a HREF="#top">Top of page</a> | <a HREF="index.html">MutNomen
homepage</a> | <a href="checklist.html">Check-list</a> | <a href="standards.html">Symbols,
codons, etc.</a> |<br>
| <b>Recommendations:</b>&nbsp; <a href="recs.html">general</a>,&nbsp;<a
href="recs-DNA.html">DNA</a>,&nbsp; <a href="recs-RNA.html">RNA</a>,&nbsp; <a
href="recs-prot.html">protein</a>, <a href="uncertain.html">uncertain</a> |<br>
| <a HREF="disc.html">Discussions</a> | <a HREF="FAQ.html">FAQ's</a> | <a
href="standards.html">Symbols, codons, etc.</a> | <a href="history.html">History</a> |<br>
| <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef</a>,&nbsp; <a
HREF="examplesDNA.html">DNA</a>,&nbsp; <a HREF="examplesRNA.html">RNA</a>,&nbsp; <a
HREF="examplesAA.html">protein</a> | </p>

<table width="100%" border="0" cellspacing="0" cellpadding="15" bgcolor="#ff9900">
  <tr>
    <td><p align="center"><font color="#FFFFFF" size="-5"><a
    href="http://www.HGVS.org/copyright.html">Copyright ©</a> </font><font size="1">HGVS 2007
    All Rights Reserved<br>
    Website Created by Rania Horaitis, Nomenclature by J.T. Den Dunnen - <a
    href="http://www.HGVS.org/disclaimer.html">Disclaimer</a> </font></td>
  </tr>
</table>
</body>
</html>
