<html>  <!--	Author: 	Dr. Johan T. den Dunnen		Date: 	02-01-2005			URL: 	recs-DNA.html-->  <head>    <meta content="text/html; charset=windows-1252" http-equiv="content-type">    <title>Uncertainties in the description of sequence variants</title>    <style type="text/css"><!--  h2 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #ff9900;}h3 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h4 { font-family : Verdana,Arial,Helvetica,Geneva,sans-serif; color : #0099cc;}h5 {font-family : Verdana, Arial, Helvetica, Geneva, sans-serif;color : #ffffff;}.norm { font-weight : normal; } --></style> </head>  <body style="        color: black; background-color: white;" link="#0000FF" vlink="#9999ff">    <table bgcolor="#CCFF99" border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#FFFFFF" valign="top" width="25%"><a href="http://www.HGVS.org"><img                src="small.gif"                border="0"                height="94"                width="155"></a></td>          <td bordercolor="#FFFFFF" bgcolor="#FFFFFF" valign="CENTER" width="75%">            <h3 align="center">Coping with uncertainties in the description of              sequence variants</h3>          </td>        </tr>      </tbody>    </table>    <table border="0" cellpadding="10" cellspacing="0" width="100%">      <tbody>        <tr>          <td bgcolor="#0099cc" valign="BOTTOM" width="100%">            <h4 align="center"><font color="#FFFFFF" size="3"><i><br>                  Last modified April 14, 2015</i></font></h4>          </td>        </tr>      </tbody>    </table>    <p align="center"><i>Since references to WWW-sites are not yet acknowledged        as citations, please mention <a href="http://www3.interscience.wiley.com/cgi-bin/fulltext/68503056/PDFSTART">den
          Dunnen JT and Antonarakis SE (2000). Hum.Mutat. 15:7-12</a> when        referring to these pages.</i></p>    <hr>    <h2>Contents</h2>    <ul>      <li><a href="#sum"><b>Summary</b></a></li>      <li><a href="#intro"><b>Introduction</b></a>        <ul>          <li><span style="color: red;"><span style="font-style: italic;"><span                  style="font-weight: bold;"><a                    href="#note-dup"><span                      style="color: red;">NOTE</span></a></span><a                  href="#note-dup"><span                    style="color: red;"></span>                  - duplications</a></span></span></li>        </ul>      </li>      <li><b>Subjects</b>        <ul>          <li><a href="#incomplete">incomplete descriptions&nbsp; - examples</a>          </li>          <li><a href="recs.html">exonic d</a><a href="#exondel">eletions /              duplications</a> </li>          <li><a href="#FISH">FISH-detected rearrangements</a> </li>          <li><a href="#array">Array-detected rearrangements</a> </li>          <li><a href="#cyto">Cytogenetic rearrangements</a> </li>          <li><a href="#2in1">two changes in one individual</a> - <i>alleles              unknown</i> </li>        </ul>      </li>    </ul>    <hr>    <h2><a name="sum"></a>Summary</h2>    <p>This page discusses how to cope with <em><strong>uncertainties</strong></em>      in the description of sequence variants and where available we link to      existing recommendations. Do you agree / disagree?, did we miss cases?, do      you want to make suggestions?; please contact us <em>by E-mail</em> (<strong>to:</strong>      HGVSmn @ JohanDenDunnen.nl) or use the <a href="https://www.facebook.com/HGVSmutnomen">HGVS-mutnomen
        Facebook</a><em></em>. </p>    <hr>    <h2><a name="intro"></a>Introduction</h2>    <p>Often clear changes can be detected in the genome, but a precise      description in relation to the genomic sequence is not possible. Examples      include cytogenetically detectable changes, changes detected using      Fluorescence In Situ Hybridisation (FISH) and changes detected using array      technology.&nbsp;Other examples are cases were genomic deletions /      duplications are detected indirectly using RNA analysis; although the      change can be described exactly at RNA-level, the genomic sequence      spanning the break point of the rearrangement is required to describe the      change at DNA-level. Finally there are examples, mostly from older      publications, where the changes detected are not described precisely (<i>"an
        11 nucleotide deletion in exon 3"</i>) or uniquely (<i>"changing amino        acid Gly-17 to Arg"</i>).</p>    <p>In many diseases changes are found which delete or duplicate (sets of)      whole exons and thereby seriously affect the function of the gene. These      changes are detected with technologies like PCR, Southern blotting, MAPH      and MLPA. The analysis however does not reveal the breakpoints at a      molecular level. Incorporation of these variants in sequence variation      databases is very important, most importantly to be able to determine      their frequency. In addition, e.g. in Duchenne/Becker Muscular Dystrophy      (DMD/BMD), the breakpoints can be predictors for the severity of the      disease: a truncated reading frame causes DMD, an open reading frame BMD.      There is a clear demand to be able to include such changes in catalogs of      all DNA sequence variants found. &nbsp; </p>    <p>It is clear that the goal should be to describe these variants <em><b>as          precise as possible</b></em>. However, in real life this is often not      possible and one should wonder whether a lot of detail adds useful      information. The deletion of genomic sequences can be detected using      several different methods, incl. FISH, Southern blot, quantitative PCR,      MAPH and MLPA. One could argue that - to be as precise as possible - the      description of the deletion should include information of the probe      sequences used. Using some methods, the presence of a probe signal only      indicates that a (large) part of the probe is present (e.g. BAC-derived      FISH signals). In PCR-based methods the absence of a signal does not mean      that the entire target sequence is missing; PCR results are negative when      only one of the two primer annealing sites is deleted or mutated (<a href="file:///Users/Boomheide/Files/WWW/HGVS/FAQ.html#exdel"><i>see
          FAQ</i></a>).</p>    <p>In MLPA were a pair of short oligonucleotides is used to detect the copy      number of a speficic sequence (mostly an exon). Effectively, when the      signal for a probe is decreased, it only indicates that the pair of oligo      nucleotide sequences (20-30 nucleotides) do not both hybridize to the      target sequence and can therefore not be ligated, extended and detected.      It is common practice (after excluding other variants when only 1 exon      probe is affected) to describe the result as a change affecting the entire      set sequences (exons), like "<em>a deletion of exons 23-27"</em> or "<em>c.3163-?_3786+?del</em>".
      When indeed the location of the probe would need to be included the      description becomes too complex to be useful. Furthermore, assuming the      probe hybridizes from position c.3211 to c.3236, which location should be      taken?, only part of the target sequence might be deleted.</p>    <p><a name="unrec"></a><b><u>Main recommendation</u></b><i><br>      </i>To indicate uncertainties in the description of sequence changes the      question mark (<em><strong>"?"</strong></em>) and brackets (<em><strong>"()"</strong></em>)      are used. When the exact position of a change is not known, <em><strong>the
          range of the uncertainty is listed between brackets</strong></em>      (like <strong><i>(5' border_3' border)</i></strong>) and one should <i><b>describe
          the change on DNA-level as precise as possible</b></i>. <img alt="" src="new.gif">When
      it is difficult to give an exact nucleotide position for a specific      probe/sequence tested, a rule of thumb is to use the central nucleotide. </p>    <p>Based on these considerations the description of a deletion has the      format: <em><strong>(last-positive_first-negative)_          (last-negative_first-positive)</strong></em>. Details of the      description are based on the technology used to detect the change:</p>    <ul>      <li> <span style="font-style: italic;">microscopy</span>&nbsp; -&nbsp;        chromosome banding</li>      <li><span style="font-style: italic;">FISH</span>&nbsp; -&nbsp; genomic        coordinates (probe names)</li>      <li><span style="font-style: italic;">arrays</span>&nbsp; -&nbsp; genomic        coordinates (probes, SNP IDs)</li>      <li><span style="font-style: italic;">MLPA</span>&nbsp; -&nbsp; coding        DNA/genomic coordinates (exons)</li>      <li><span style="font-style: italic;">PCR</span>&nbsp; -&nbsp; coding        DNA/genomic coordinates</li>    </ul>    <p>For further details see below. </p>    <h4><strong><a name="note-dup"></a>NOTE</strong></h4>    <p>For duplications the same recommendations hold, except that duplications      are designated by <b>"dup"</b> in stead of "del": <i>(last-positive_first-negative)_
        (last-negative_first-positive)dup</i> (<i><a href="recs-DNA.html#dup">see
          Recommendations</a></i>). It should be noted however that the      description <b>"dup"</b> may <b>by definition</b> (<i><a href="standards.html#define">see
          Standards</a>) </i> only be used when the additional copy is located      directly 3'-flanking the original copy (a tandem duplication). In most      cases there will be no experimental proof, one simply detects the presence      of an additional copy that can be anywhere in the genome (inserted /      transposed). Discussions are ongoing how to include this uncertainty best      in the description<i> (<i><a href="recs-DNA.html#var">see Recommendations</a></i>).</i>      It should be clear though that describing a duplication like<i> <span style="font-weight: bold;">c.3163-?_3786+?dup
          in general is not correct</span></i>.<br>    </p>    <hr>    <h2>Subjects</h2>    <hr>    <h3>Incomplete descriptions</h3>    <p>When the exact position of a change is not known, the range of the      uncertainty is listed between brackets (<b>"()"</b>, <em><a href="#unrec">see
          Recommendation</a></em>). Similarly, when insertions have not been      specified (e.g. <span style="font-style: italic;">"ins5"</span>) or when      an insertion was not sequenced but its length estimated (e.g. from gel      electrophoresis), brackets are used to indicate the uncertainty. </p>    <h4><u>Examples</u></h4>    <ul>      <li><i>c.(67_70)insG (p.Gly23fs) </i>indicates the insertion of a G at an        unknown position in sequence of amino acid codon 23 </li>      <li><span style="font-style: italic;">g.11_12ins(100)</span>        (alternatively <span style="font-style: italic;">c.11_12insN[100]</span>)        indicates the insertion of about 100 nucleotides between position g.11        and g.12</li>      <ul>        <li><span style="font-style: italic;">g.11_12ins(1) </span>(alternatively
          <span style="font-style: italic;">g.11_12insN</span>) indicates the          insertion of one not specified nucleotide (N) between position g.11          and g.12</li>        <li><span style="font-style: italic;">g.11_12ins(5) </span>(alternatively
          <span style="font-style: italic;">g.11_12insNNNNN</span>) indicates          the insertion of 5 not specified nucleotides (NNNNN) between position          g.11 and g.12</li>      </ul>      <li><span style="font-style: italic;">p.(fs*)</span> is the best        description to describe at protein level the consequence of        c.577_578ins(4) (with the 4 nucleotides inserted not specified)<br>        <br>      </li>      <li><i>c.(165_253)del11</i> indicates an 11 nucleotide deletion in exon 3<i>          (NOTE: in relation to the coding DNA reference sequence, exon 3 is          located from position 165 to 253)</i></li>      <ul>        <li> <span style="font-style: italic;">c.(1521_1524)del3</span> is the          description that should be connected to the "<em><strong>deltaF508</strong></em>"          (p.(Phe508del)) variant often found in patients with Cystic Fibrosis <span            style="font-style: italic;">when
            no DNA data are given</span>. The sequence surrounding codon Phe-508          in the CFTR gene is ..-ATC-TTT-GGT-... (<span style="font-style: italic;">c.1519
            to c.1527</span>) and three different deletions (TC-T, C-TT and          -TTT-) would give the reported change at protein level. Adhering to          the 3' rule for variant descriptions (<a href="http://www.hgvs.org/mutnomen/recs-DNA.html#del"><span              style="font-style: italic;">see
              Recommendations</span></a>) this goes with 2 different changes at          DNA level: c.1521_1523del and c.1522_1524del. When one assumes the          change at DNA level is c.1522_1524delTTT, deletion of exactly the          Phe-508 encoding triplet, one is wrong. The reported change is mostly          c.1521_1523delCTT. So, without specification in the manuscript one can          not be certain.</li>      </ul>    </ul>    <em>      <hr>      <h3><a name="exondel"></a>Exonic deletions / duplications</h3>    </em>    <p>The description should use the basic format <em><strong>(last-positive_first-negative)_
          (last-negative_first-positive)</strong></em>, be based on the      reference sequence used and include the position of the most extreme      region(s) tested (e.g. segment PCR-ed, probes used for hybridisation,      etc.).</p>    <p>For clarity and to make descriptions form specific tests, e.g. MLPA, not      too complicated, it is allowed to describe changes assuming that when a      probe for a specific exon scores deleted (duplicated) that the entire exon      is deleted (duplicated), i.e. detailed knowledge regarding the exact      location of the probe sequence used is not used in the description.</p>    <p> <em><strong>NOTE:</strong></em> it should be clearly indicated which      technology was used (MLPA, PCR, etc.) and where primer/probe sequences can      be found. For rearrangements affecting 1-exon only it should be indicated      whether DNA sequencing was performed to exclude variants affecting the      primer/probe target sequences. When different probes for one exon score      different, this information must be used in the description of the      sequence change (<a href="FAQ.html#exdel"><i>see FAQ</i></a>).</p>    <p>Deletions are designated by <b>"del"</b> after an indication of the      first and last nucleotide(s) deleted (<a href="recs-DNA.html#del">see        Recommendations</a>).</p>    <h4><u>Examples</u></h4>    <ul>      <li><i><b><strong><font color="#FF0000"><small>date 2012-11-01</small></font></strong></b></i><i><b><strong><font                color="#FF0000"><small><i><img                      src="file:///Users/Boomheide/Files/WWW/HGVS/new.gif"                      border="0"                      height="12"                      width="31"></i></small></font></strong>c.(87+1_88-1)_(300+1_301-1)del</b>        </i>(alternatively <span style="font-style: italic;">c.88-?_301+?del</span>)        denotes the deletion of exons 3 to 4, starting at an unknown position in        intron 2 (c.87+1_88-1, downstream of c.87 (the last nucleotide of exon        2) but upstream of c.88 (the first nucleotide of exon 3)), and ending at        an unknown position in intron 4 (c.300+1_301-1).<br>        <span style="font-weight: bold; font-style: italic;">NOTE:</span> the        original suggestion was to describe this change as <span style="font-style: italic;">c.88-?_301+?del</span>.        However, this description is confusing because it can not be        discriminated from&nbsp;<span style="font-style: italic;">c.88-?_301+?del</span>        where <span style="font-weight: bold; font-style: italic;">probes were          not tested</span> upstream of <span style="font-style: italic;">c. 88</span>        or downstream of <span style="font-style: italic;">c. 301</span>. Only        when one knows the details, e.g. that an MLPA test was used that did        include probes up- and downstream, one would understand the description        which is not acceptable.</li>      <li><i><b>c.(?_-30)_(12+1_13-1)del</b> </i>describes a deletion starting        somewhere upstream from the 5' end of a gene (located at coding DNA        nucleotide -30) and ending in the intron between coding DNA nucleotides        12+1 and 13-1 (intron 1).</li>      <li><i><b>c.(?_-1)_(*1_?)del</b>&nbsp; </i>is a standard way to describe        the deletion of the entire protein coding region of a gene (coding DNA        reference sequence).<br>        <i><span style="font-weight: bold;">NOTE: </span>when more details are          available regarding the deletion, based on the probes tested to          determine its location, the description can be specified like          c.(?_-189)_(*884_?)del, meaning the deletion starts 5' of c.-189 and          extends to beyond c.*884 in the 3'UTR.<br>        </i></li>    </ul>    <i> <b>NOTE:</b> when regions outside the gene have been tested and found      to be present / absent this can be indicated in a "Remarks" field.      Alternatively, one can determine what the position of the genomic      sequences tested is and use these positions in the description (e.g.      position of oligonucleotide used in PCR, MLPA, SNParray, and/or of the      probe used for hybridisation).</i> <em>      <hr>      <h3><a name="FISH"></a>FISH-detected rearrangements</h3>    </em>    <p>Many chromosomal rearrangements, especially in the past, have been      detected using techniques like Southern blotting and Fluorescence In Situ      Hybridisation (FISH). The description of these changes was often in      tabular or graphical format based on either position-ordered probe names      or relative chromosomal positions. Especially when FISH was used,      relatively little or often even no actual DNA sequence of the probe(s)      used was known. In those days, even when a probe sequence was known, this      information was of little help to determine the probe location more      precisely.</p>    <p>With the availability of the reference human genome sequence, this      situation has dramatically changed. Now, any piece of DNA probe sequence      can be used to position that probe with great precision on the human      genome map. In addition, the clones used to generate the human genome      sequence are freely available and have become the preferred probes for new      FISH experiments. The latter is especially true for genome-wide array-CGH      experiments using ordered PAC/BAC-clones. As a consequence of these      developments it is now possible to describe these changes based on DNA      sequences. <i><b>NOTE: </b><a href="disc.html#indicator">see Discussion</a></i>.      <em></em></p>    <em> </em>    <p><em></em>Following the recommendation to describe rearrangements using      the format <strong><i>(5' border_3' border)</i></strong> for FISH probes      this becomes&nbsp; <em><strong>(last-positive-clone_first-negative-clone)_
          (last-negative-clone_first-positive-clone)</strong></em>.<em></em></p>    <em> </em>    <h4><em></em><u>Examples</u><em></em></h4>    <em> </em>    <ul>      <em> </em>      <li><em><b>chrX:g.(AC096506.5_AL109609.5)_(AL451144.5_AL050305.9)del</b> </em>describes
        a genomic deletion on the X-chromosome detected using FISH. The deletion        spans from PAC probes RP4-556A22 (GenBank AL109609.5) to RP11-151J4        (GenBank AL451144.5), both yielding no signal. On the telomeric side        (p-arm) the closest probe tested positive was PAC RP11-64I1 (GenBank        AC096506.5), on the centromeric side the closest probe tested positive        was RP6-60B16 (GenBank AL050305.9).&nbsp;<em><br>          <b>NOTE:</b> a description like <b>(AC096506.5_AL109609.5)del</b>          indicates that the breakpoint of the deletion is somewhere between          these two sequences (here PAC-derived) and gives a direct link to the          human genome sequence.<br>        </em>When the genomic positions of these sequences are known, the        deletion can also be described directly <b>in relation to the genome          sequence as hg19 </b><em><strong>chrX:g.(32218983_32238146)_(32984039_33252615)del</strong></em>,        i.e.        (genomic-end-position-last-positive-clone_genomic-start-position-first-negative-clone)_
        (genomic-end-position-last-negative-clone_genomic-start-position-first-positive-clone).
        Although this description is more precise in relation to the genome        sequence, the information regarding the probe sequences used can not be        derived from the description and needs to be reported separately. Note        the addition of "<em><strong>hg19</strong></em>" to indicate the        reference genome build used for the description.<em></em></li>      <em> </em>      <li><em><b>chrX:g.(?_AL109609.5)_(AL451144.5_?)del</b> </em>describes a        genomic deletion on the X-chromosome detected using FISH. The deletion        spans from PAC probes RP4-556A22 (GenBank AL109609.5) to RP11-151J4        (GenBank AL451144.5), both yielding no signal. <b>No flanking positive          probes were tested</b>, making it unclear how far the deletion extends        (compare with previous description).<em></em></li>      <em> </em>      <li><em><b>chrX:g.(AC096506.5_AL109609.5)_AL451144.5del</b></em> describes        a genomic deletion on the X-chromosome detected using FISH. The deletion        spans from PAC probes RP4-556A22 (GenBank AL109609.5) to RP11-151J4        (GenBank AL451144.5). On the telomeric side (p-arm) the closest probe        tested positive was PAC RP11-64I1 (GenBank AC096506.5), while RP4-556A22        (GenBank AL109609.5) tested negative. On the centromeric side the probe        RP11-151J4 (GenBank AL451144.5) gave a <b>reduced signal</b>,        indicating that the breakpoint lies inside this clone (note that this        identifier is <b>not</b> between brackets).<br>        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; An alternative description is        hg19&nbsp; chrX:g.(AC096506.5_AL109609.5)_AL451144.5:g.(1_100207)del        where AL451144.5:g.(1_100207) indicates that the range where the        deletion junction lies spans nucleotides 1 to 100,207. One can argue        whether this addition is very informative. Note also the addition of "<em><strong>hg19</strong></em>"        to indicate the reference genome build used for the description.<em></em></li>      <em> </em>    </ul>    <em>      <hr>      <h3><a name="array"></a>Array-detected rearrangements</h3>    </em>    <p>Basically, chromosomal rearrangements and other DNA sequence variants      detected using array technology can, based on the array-probe sequences      used, be described as those for FISH-detected rearrangements (<a href="#FISH"><i>see
          FISH-detected rearrangements</i></a>). An advantage here is that the      array probes used are often exactly defined, being mostly relatively short      20-60-mer oligonucleotide sequences. This information can be used to      exactly describe the rearrangements at the nucleotide level. <em></em></p>    <em>      <p>For deletions the basic format is        (last-probe-present_first-probe-deleted)_(last-probe-deleted_first-probe-present).
        </p> </em>    <h4><em></em><u>Examples</u><em></em></h4>    <em> </em>    <ul>      <em> </em>      <li><em><b>hg19 chrX:g.(32218983_32238146)_(32984039_33252615)del</b> </em>describes
        a deletion on the X chromosome, based on reference genome build hg19,        starting between nucleotides 32,218,983 to 32,238,146 and ending between        nucleotides 32,984,039 to 33,252,615.<em></em></li>      <em> </em>      <li><em><b>hg19 chrX:g.(?_32238146)_(32984039_?)del</b> </em>describes a        deletion on the human X chromosome, based on reference genome build        hg19, starting upstream of nucleotide 32,238,146 and ending downstream        of nucleotide 32,984,039.<em><br>          <b>NOTE:</b> the description leaves it thus unclear how far the          deletion might extend, suggesting no up- or downstream probes were          tested (and scored positive).</em></li>      <em> </em>      <li><em><b>g.(rs2342234_rs3929856)_(rs10507342_rs947283)del</b>          (alternatively hg19          chr13:g.(18,858,133_18,867,056)_(24,517,730_24,531,502)del)</em>        describes a genomic deletion on chromosome 13 detected using a        SNP-array. The deletion spans from dbSNP entries rs3929856 to        rs10507342, both yielding no signal. On the centromeric side (q-arm) the        closest probe tested positive was rs2342234, on the telomeric side the        closest probe tested positive was rs947283.<em><b><br>            NOTE:</b> although the alternative description based on the human          genome nucleotide numbering is more precise, the information regarding          the probe sequences used is lost.&nbsp;</em></li>      <em> </em>      <li><em><b>g.(?_rs3929856)_(rs10507342_?)del</b> (alternatively hg19          chr13:g.(?_18,867,056)_(24,517,730_?)del) </em>describes a deletion        detected using an array spanning from dbSNP entries rs3929856 to        rs10507342. The description assumes no flanking positive probes were        tested, making it unclear how far the deletion extends.<em></em></li>      <em> </em>    </ul>    <em>      <hr> </em>    <h3><em><a name="cyto"></a></em>Cytogenetic rearrangements</h3>    <p>A nomenclature system to describe cytogenetically detectable      rearrangements has been suggested early on (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=4041569&amp;dopt=Abstract">see
ISCN
        1985</a>). Current recommendations in this areas are made by the      "Standing Committee on Human Cytogenetic Nomenclature" and were published      recently as <a href="http://content.karger.com/ProdukteDB/produkte.asp?Aktion=showproducts&amp;searchWhat=books&amp;ProduktNr=257302">ISCN
        2013</a>.<em></em></p>    <em> </em>    <hr>    <h3><a name="2in1"></a>Two changes in one individual</h3>    <p>When <i><b>two sequence changes</b></i> are found in one gene of an      individual but it is <i><b>unknown</b></i> whether they are <b><i>located
          on the same or on different chromosomes</i></b>, the change is      described using the format c.[76A&gt;C(;)483G&gt;C] (<i><a href="recs.html#2in1">see
          Recommendations</a></i>).</p>    <hr>    <p align="center">| <a href="#top">Top of page</a> | <a href="index.html">MutNomen
        homepage</a> | <a href="checklist.html">Check-list</a> |<br>      | <b>Recommendations:</b> <a href="recs.html">general</a>,&nbsp; <a href="recs-RNA.html">RNA</a>,&nbsp;
      <a href="recs-prot.html">protein</a> |<br>      | <a href="disc.html">Discussions</a> | <a href="FAQ.html">FAQ's</a> | <a        href="standards.html">Symbols,
        codons, etc.</a> | <a href="history.html">History</a> |<br>      | <b>Example descriptions:&nbsp;</b> <a href="quickref.html">QuickRef /        symbols</a>,&nbsp; <a href="examplesDNA.html">DNA</a>,&nbsp; <a href="examplesRNA.html">RNA</a>,&nbsp;
      <a href="examplesAA.html">protein</a> | </p>    <table bgcolor="#ff9900" border="0" cellpadding="15" cellspacing="0" width="100%">      <tbody>        <tr>          <td>            <p align="center"><i><font color="#FFFFFF" face="Verdana, Arial, Helvetica, sans-serif"                  size="-5"><a                    href="http://www.HGVS.org/copyright.html">Copyright
                    ©</a> </font><font size="1">HGVS 2007 All Rights Reserved<br>                  Website Created by Rania Horaitis, Nomenclature by J.T. Den                  Dunnen - <a href="http://www.HGVS.org/disclaimer.html">Disclaimer</a>                </font></i></p>          </td>        </tr>      </tbody>    </table>  </body></html>